<div class="ngisweden-publications mb-5"><h5>User Publications</h5><div class="list-group">
            <a data-toggle="modal" data-target="#pub_20af1880766c43ec92b67618ebf96e75" href="https://publications.scilifelab.se/publication/20af1880766c43ec92b67618ebf96e75" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Single-Cell Omics Analysis of Human Basophils Reveals Two Transcriptionally Distinct Populations.<br>
                <small class="text-muted"><em>Allergy</em> (2026)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_eaf9b454889b42e98966f350d0a69e84" href="https://publications.scilifelab.se/publication/eaf9b454889b42e98966f350d0a69e84" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                A mosaic of modular variation at a single gene underpins convergent plumage coloration.<br>
                <small class="text-muted"><em>Science</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_04e9d7eb6fd041648cef2ac003bdaae7" href="https://publications.scilifelab.se/publication/04e9d7eb6fd041648cef2ac003bdaae7" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Chromosome-level assembly of the club-legged grasshopper (Gomphocerus sibiricus) genome.<br>
                <small class="text-muted"><em>G3 (Bethesda)</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_ebe4355205e54bc3b50d10a1a62fb6e8" href="https://publications.scilifelab.se/publication/ebe4355205e54bc3b50d10a1a62fb6e8" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Genetic Adaptation to Brackish Water and Spawning Season in European Cisco.<br>
                <small class="text-muted"><em>Mol. Ecol.</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_19a4115c6ad14acaaf738a62c51e4f69" href="https://publications.scilifelab.se/publication/19a4115c6ad14acaaf738a62c51e4f69" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Genome analyses suggest recent speciation and postglacial isolation in the Norwegian lemming.<br>
                <small class="text-muted"><em>Proc. Natl. Acad. Sci. U.S.A.</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_784df73e8dc84ac78534a0c33e5aec03" href="https://publications.scilifelab.se/publication/784df73e8dc84ac78534a0c33e5aec03" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Cytoskeletal alterations in neuronal cells implicate Toxoplasma gondii secretory machinery and host microRNA-containing extracellular vesicles.<br>
                <small class="text-muted"><em>Sci Rep</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_c42cf07b07844cabbe0ad109571234eb" href="https://publications.scilifelab.se/publication/c42cf07b07844cabbe0ad109571234eb" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Cellular heterogeneity in metabolism and associated microbiome of a non-model phytoflagellate.<br>
                <small class="text-muted"><em>ISME J</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_94a6617807b3431a961787b517bc63a0" href="https://publications.scilifelab.se/publication/94a6617807b3431a961787b517bc63a0" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Orthopteran Neo-Sex Chromosomes Reveal Dynamics of Recombination Suppression and Evolution of Supergenes.<br>
                <small class="text-muted"><em>Mol. Ecol.</em> (2024)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_06092f1c02924d8da8dd7a4929fd5ac3" href="https://publications.scilifelab.se/publication/06092f1c02924d8da8dd7a4929fd5ac3" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Origin, structure, and composition of the spider major ampullate silk fiber revealed by genomics, proteomics, and single-cell and spatial transcriptomics.<br>
                <small class="text-muted"><em>Sci Adv</em> (2024)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_7d3c4336130e450099e81cfa741064ec" href="https://publications.scilifelab.se/publication/7d3c4336130e450099e81cfa741064ec" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Adaptive introgression reveals the genetic basis of a sexually selected syndrome in wall lizards.<br>
                <small class="text-muted"><em>Sci Adv</em> (2024)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_a1a66eb9378f48bcbb9c32923b5503e3" href="https://publications.scilifelab.se/publication/a1a66eb9378f48bcbb9c32923b5503e3" 
                target="_blank" class="list-group-item list-group-item-action">
                Genome Variation in Three Anthophora Bee Species Reflects Divergent Demographic Histories<br>
                <small class="text-muted"><em>Mol Ecol</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_fdb2a58a717e4bbca202566f67f5792b" href="https://publications.scilifelab.se/publication/fdb2a58a717e4bbca202566f67f5792b" 
                target="_blank" class="list-group-item list-group-item-action">
                A co-speciation dilemma and a lifestyle transition with genomic consequences in Wolbachia of Neotropical Drosophila<br>
                <small class="text-muted"><em>BMC Genomics</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_b8ca2a2dba2049ec8de7192c162898c4" href="https://publications.scilifelab.se/publication/b8ca2a2dba2049ec8de7192c162898c4" 
                target="_blank" class="list-group-item list-group-item-action">
                Homo sapiens-specific evolution unveiled by ancient southern African genomes.<br>
                <small class="text-muted"><em>Nature</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_bed19b398aa143d28064d622d7f4babf" href="https://publications.scilifelab.se/publication/bed19b398aa143d28064d622d7f4babf" 
                target="_blank" class="list-group-item list-group-item-action">
                Holotype genome of the lesula provides insights into demography and evolution of a threatened primate lineage.<br>
                <small class="text-muted"><em>Genome Biol.</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_823c71158dcd4b9fb085d01e55d13e48" href="https://publications.scilifelab.se/publication/823c71158dcd4b9fb085d01e55d13e48" 
                target="_blank" class="list-group-item list-group-item-action">
                scFFPE-ATAC enables high-throughput single cell chromatin accessibility profiling in formalin-fixed paraffin-embedded samples.<br>
                <small class="text-muted"><em>Nat Commun</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_5087e22117c64ebbbc4b4299bfc98b9a" href="https://publications.scilifelab.se/publication/5087e22117c64ebbbc4b4299bfc98b9a" 
                target="_blank" class="list-group-item list-group-item-action">
                A [11C]PBR28 PET study on the associations between sleep health and microglial density.<br>
                <small class="text-muted"><em>J Neuroinflammation</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_f3a1b1396edb4101b0575fbc43ec7415" href="https://publications.scilifelab.se/publication/f3a1b1396edb4101b0575fbc43ec7415" 
                target="_blank" class="list-group-item list-group-item-action">
                Comparative evaluation of Olink Explore 3072 and mass spectrometry with peptide fractionation for plasma proteomics.<br>
                <small class="text-muted"><em>Commun Chem</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_1938da5a9fb54644b9fd03953d33a22d" href="https://publications.scilifelab.se/publication/1938da5a9fb54644b9fd03953d33a22d" 
                target="_blank" class="list-group-item list-group-item-action">
                Autoreactive T cells identified in patients with anti-Jo1+ antisynthetase syndrome recognise a new epitope on histidyl t-RNA synthetase.<br>
                <small class="text-muted"><em>Ann. Rheum. Dis.</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_a303b58b4fa1409b9589100fef20ee82" href="https://publications.scilifelab.se/publication/a303b58b4fa1409b9589100fef20ee82" 
                target="_blank" class="list-group-item list-group-item-action">
                Postnatal sustentacular cells as chromaffin progenitors and tumor cells of origin in VHL-related paragangliomas.<br>
                <small class="text-muted"><em>NPJ Precis Oncol</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_107c7b3b512c45f686868f13aff2dfd8" href="https://publications.scilifelab.se/publication/107c7b3b512c45f686868f13aff2dfd8" 
                target="_blank" class="list-group-item list-group-item-action">
                Innate immune cell subsets are enriched in synovial fluid of ACPA-negative rheumatoid arthritis and characterized by distinct type I IFN gene signatures.<br>
                <small class="text-muted"><em>Ann. Rheum. Dis.</em> (2025)</small>
                
                
                
                
            </a>
        </div>
                <p class="small text-muted mt-2">
                    See all publications at
                    <a href="https://publications.scilifelab.se/label/National%20Genomics%20Infrastructure" 
                        target="_blank" class="text-muted">
                        publications.scilifelab.se
                    </a>
                </p>
            </div>
        <div class="modal ngisweden-publications-modal fade" id="pub_20af1880766c43ec92b67618ebf96e75" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Single-Cell Omics Analysis of Human Basophils Reveals Two Transcriptionally Distinct Populations.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Sofia Papavasileiou">S Papavasileiou</span>, <span class="pub-author" title="Jiezhen Mo">J Mo</span>, <span class="pub-author" title="Daryl Boey">D Boey</span>, <span class="pub-author" title="Chenyan Wu">C Wu</span>, <span class="pub-author" title="Magnus Tronstad">M Tronstad</span>, <span class="pub-author" title="Lucille Margerie">L Margerie</span>, <span class="pub-author" title="Remi-André Olsen">RA Olsen</span>, <span class="pub-author" title="Jörg A Bachmann">JA Bachmann</span>, <span class="pub-author" title="Jing Hui Low">JH Low</span>, <span class="pub-author" title="Jocelyn Ong">J Ong</span>, <span class="pub-author" title="Lars Heede Blom">LH Blom</span>, <span class="pub-author" title="Anand Kumar Andiappan">AK Andiappan</span>, <span class="pub-author" title="Gunnar Nilsson">G Nilsson</span>, <span class="pub-author" title="Joakim S Dahlin">JS Dahlin</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Basophils are implicated in various diseases including allergies, but a comprehensive single-cell characterization of human basophils has yet to be performed. Here, we aimed to generate a single-cell omics-based reference resource of circulating human basophils, integrating transcriptomic and large-scale immunoprofiling data. We also sought to investigate basophil heterogeneity at the molecular level.

Circulating basophils were analyzed using cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq). Both short- and long-read single-cell RNA-sequencing platforms were used to capture the transcriptomic data.

CITE-seq enabled accurate identification and profiling of side scatterlow lineage- CCR3+ FcεRI+ basophils. Short-read single-cell RNA-sequencing data revealed two previously unresolved basophil populations, defined by 66 differentially expressed genes and reproducibly identified across donors. Despite the transcriptional differences, the populations displayed similar immunophenotypes based on more than 100 investigated cell surface markers. Long-read single-cell RNA-sequencing analysis confirmed the existence of the two populations and provided further insights into their gene expression profiles.

We present a multimodal single-cell resource that defines two novel transcriptionally distinct basophil populations. This resource, accessible through a user-friendly web interface, constitutes a cellular and molecular reference map for future studies of basophils in health and disease.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41498390" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/all.70209" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/20af1880766c43ec92b67618ebf96e75" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_eaf9b454889b42e98966f350d0a69e84" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A mosaic of modular variation at a single gene underpins convergent plumage coloration.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Dave Lutgen">D Lutgen</span>, <span class="pub-author" title="Valentina Peona">V Peona</span>, <span class="pub-author" title="Madeline A Chase">MA Chase</span>, <span class="pub-author" title="Niloofar Alaei Kakhki">NA Kakhki</span>, <span class="pub-author" title="Fritjof Lammers">F Lammers</span>, <span class="pub-author" title="Stacey G de Souza">SG de Souza</span>, <span class="pub-author" title="Anne-Lyse Ducrest">AL Ducrest</span>, <span class="pub-author" title="Marta Burri">M Burri</span>, <span class="pub-author" title="Pavlos Andriopoulos">P Andriopoulos</span>, <span class="pub-author" title="Sifiso M Lukhele">SM Lukhele</span>, <span class="pub-author" title="Michaella Moysi">M Moysi</span>, <span class="pub-author" title="Elizabeth Yohannes">E Yohannes</span>, <span class="pub-author" title="Abdin Abbasov">A Abbasov</span>, <span class="pub-author" title="Tamer Albayrak">T Albayrak</span>, <span class="pub-author" title="Mansour Aliabadian">M Aliabadian</span>, <span class="pub-author" title="Nicolas Auchli">N Auchli</span>, <span class="pub-author" title="Vasileios Bontzorlos">V Bontzorlos</span>, <span class="pub-author" title="Ioulios Christoforou">I Christoforou</span>, <span class="pub-author" title="José Luis Copete">JL Copete</span>, <span class="pub-author" title="Egidio Fulco">E Fulco</span>, <span class="pub-author" title="Jesus T Garcia">JT Garcia</span>, <span class="pub-author" title="Zura Javakhishvili">Z Javakhishvili</span>, <span class="pub-author" title="Anna Kazazou">A Kazazou</span>, <span class="pub-author" title="Fumin Lei">F Lei</span>, <span class="pub-author" title="Yang Liu">Y Liu</span>, <span class="pub-author" title="Nika Paposhvili">N Paposhvili</span>, <span class="pub-author" title="Robert Patchett">R Patchett</span>, <span class="pub-author" title="Áron Péter">Á Péter</span>, <span class="pub-author" title="Raphael Ritter">R Ritter</span>, <span class="pub-author" title="Attila D Sándor">AD Sándor</span>, <span class="pub-author" title="Fabian Schneider">F Schneider</span>, <span class="pub-author" title="Petar Shurulinkov">P Shurulinkov</span>, <span class="pub-author" title="Sergey Sklyarenko">S Sklyarenko</span>, <span class="pub-author" title="Borut Stumberger">B Stumberger</span>, <span class="pub-author" title="Abulfaz Tagiyev">A Tagiyev</span>, <span class="pub-author" title="Alessia Uboldi">A Uboldi</span>, <span class="pub-author" title="Nikitas Vogiatzis">N Vogiatzis</span>, <span class="pub-author" title="Fanny Taborsak-Lines">F Taborsak-Lines</span>, <span class="pub-author" title="Joel Gruselius">J Gruselius</span>, <span class="pub-author" title="Liqun Yao">L Yao</span>, <span class="pub-author" title="Catherine L Peichel">CL Peichel</span>, <span class="pub-author" title="Alexander Suh">A Suh</span>, <span class="pub-author" title="Pierre-Alexandre Gagnaire">PA Gagnaire</span>, <span class="pub-author" title="Alexander N G Kirschel">ANG Kirschel</span>, <span class="pub-author" title="Manuel Schweizer">M Schweizer</span>, <span class="pub-author" title="Holger Schielzeth">H Schielzeth</span>, <span class="pub-author" title="Reto Burri">R Burri</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The reshuffling of genomic variation from multiple origins is an important contributor to phenotypic diversification, yet insights into the evolutionary trajectories of this combinatorial process and their interplay with genetic architecture remain scarce. We show that convergent plumage color evolution in wheatears involves a monogenic architecture with modular variation introgressed at the agouti signaling protein (ASIP) locus. Introgression of a new transposable element insertion and linked protein-coding variation underpin a transspecific throat color polymorphism, which stable isotopes suggest is associated with alternative foraging niches. Cointrogression of linked regulatory ASIP variation resulted in mantle color convergence in one species, whereas convergent color evolution at the genus level required new variation. Our results demonstrate evolutionary trajectories from introgressed variation realized within the constraints of a monogenic architecture.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41100596" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/science.ado8005" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/eaf9b454889b42e98966f350d0a69e84" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_04e9d7eb6fd041648cef2ac003bdaae7" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Chromosome-level assembly of the club-legged grasshopper (Gomphocerus sibiricus) genome.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Octavio M Palacios-Gimenez">OM Palacios-Gimenez</span>, <span class="pub-author" title="Mahendra Varma">M Varma</span>, <span class="pub-author" title="Xinyi Cheng">X Cheng</span>, <span class="pub-author" title="Mai-Britt Mosbech">MB Mosbech</span>, <span class="pub-author" title="Alexander Suh">A Suh</span>, <span class="pub-author" title="Holger Schielzeth">H Schielzeth</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Grasshoppers represent true outliers in genome sizes, both within insects and within animals in general. Their genomes are large and generally variable in sizes and feature a high abundance of repetitive DNA sequences. This has hampered the assembly of grasshopper genomes to chromosome level. Here we present a chromosome-level reference genome for the club-legged grasshopper (Gomphocerus sibiricus, Acrididae: Gomphocerinae) using PacBio HiFi long-read and Hi-C sequencing technologies. In male haploid cells, the species has a chromosome set of n = 9 with an X0 sex-determination system, characterized by an absence of a Y chromosome. Our assembly spans 9.57 Gb in total, with 8.87 Gb organized into nine chromosomes-eight autosomes and the X chromosome. The final assembly has a scaffold N50 value of 1.58 Gb, covers 96.7% single copy Insecta orthologs, and contains 42,665 predicted protein-coding genes and 43,385 mRNA transcripts. We compiled a curated, non-redundant, species-specific repeat library and used it to annotate repetitive DNA, covering 81.69% of the genome, mostly DNA transposons, LINE and LTR retrotransposons. The genome of the club-legged grasshopper shows high degree of synteny with the locusts Schistocerca gregaria and Locusta migratoria, and the analysis strongly indicates three autosome-autosome centric fusions in Gomphocerinae. The genome offers a valuable resource for grasshopper genomics and for exploring the genetic basis of a transspecies color polymorphism.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41029998" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1093/g3journal/jkaf231" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/04e9d7eb6fd041648cef2ac003bdaae7" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_ebe4355205e54bc3b50d10a1a62fb6e8" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genetic Adaptation to Brackish Water and Spawning Season in European Cisco.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Qiaoling Deng">Q Deng</span>, <span class="pub-author" title="Jake Goodall">J Goodall</span>, <span class="pub-author" title="Mikaela Bergenius Nord">M Bergenius Nord</span>, <span class="pub-author" title="Ignas Bunikis">I Bunikis</span>, <span class="pub-author" title="Arianna Cocco">A Cocco</span>, <span class="pub-author" title="Bo Delling">B Delling</span>, <span class="pub-author" title="Elisabet Einarsdottir">E Einarsdottir</span>, <span class="pub-author" title="Julia Heintz">J Heintz</span>, <span class="pub-author" title="Henrik Lantz">H Lantz</span>, <span class="pub-author" title="Kerstin Lindblad-Toh">K Lindblad-Toh</span>, <span class="pub-author" title="Mai-Britt Mosbech">M Mosbech</span>, <span class="pub-author" title="Remi-Andre Olsen">R Olsen</span>, <span class="pub-author" title="Stefan Palm">S Palm</span>, <span class="pub-author" title="Mats E Pettersson">ME Pettersson</span>, <span class="pub-author" title="Martin Pippel">M Pippel</span>, <span class="pub-author" title="Lucile Soler">L Soler</span>, <span class="pub-author" title="Anti Vasemägi">A Vasemägi</span>, <span class="pub-author" title="Olga Vinnere Pettersson">OV Pettersson</span>, <span class="pub-author" title="Leif Andersson">L Andersson</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">How species adapt to diverse environmental conditions is essential for understanding evolution and the maintenance of biodiversity. The European cisco (Coregonus albula) is a salmonid that occurs in both fresh and brackish water, and this together with the presence of sympatric spring- and autumn-spawning lacustrine populations provides an opportunity for studying the genetics of adaptation in relation to salinity and timing of reproduction. Here, we present a high-quality reference genome of the European cisco based on PacBio HiFi long read sequencing and HiC-directed scaffolding. We generated low-coverage whole-genome sequencing data from 336 individuals across 12 population samples to explore population structure and genetics of ecological adaptation. We found a major subdivision between two groups of populations most likely reflecting colonisation from different glacial refugia. Within the two major groups, we detected further genetic differentiation between spring- and autumn-spawning populations and between populations from freshwater lakes, rivers and brackish water (Bothnian Bay). A genome-wide screen for genetic differentiation among populations identified a set of outlier SNPs strongly correlated with spawning timing and salinity. Several of the genes associated with spawning time, including BHLHE40, TIMELESS and CPT1A, have previously been shown to have a role in circadian rhythm biology. As many as 17 loci were associated with genetic differentiation between populations reproducing in fresh and brackish water. This study provides insights into the genomic basis of ecological adaptation in European cisco with implications for sustainable fishery management.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40903929" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/mec.70094" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/ebe4355205e54bc3b50d10a1a62fb6e8" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_19a4115c6ad14acaaf738a62c51e4f69" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genome analyses suggest recent speciation and postglacial isolation in the Norwegian lemming.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Edana Lord">E Lord</span>, <span class="pub-author" title="Isabelle S Feinauer">IS Feinauer</span>, <span class="pub-author" title="André E R Soares">AER Soares</span>, <span class="pub-author" title="Vendela K Lagerholm">VK Lagerholm</span>, <span class="pub-author" title="Karin Näsvall">K Näsvall</span>, <span class="pub-author" title="Erik Ersmark">E Ersmark</span>, <span class="pub-author" title="Remi-André Olsen">RA Olsen</span>, <span class="pub-author" title="Stefan Prost">S Prost</span>, <span class="pub-author" title="Elena A Kuzmina">EA Kuzmina</span>, <span class="pub-author" title="Nickolay G Smirnov">NG Smirnov</span>, <span class="pub-author" title="John R Stewart">JR Stewart</span>, <span class="pub-author" title="Monika V Knul">MV Knul</span>, <span class="pub-author" title="Pierre Noiret">P Noiret</span>, <span class="pub-author" title="Mietje Germonpré">M Germonpré</span>, <span class="pub-author" title="Dorothee Ehrich">D Ehrich</span>, <span class="pub-author" title="Ivan Pokrovsky">I Pokrovsky</span>, <span class="pub-author" title="Vadim B Fedorov">VB Fedorov</span>, <span class="pub-author" title="Anna V Goropashnaya">AV Goropashnaya</span>, <span class="pub-author" title="Love Dalén">L Dalén</span>, <span class="pub-author" title="David Díez-Del-Molino">D Díez-Del-Molino</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The Norwegian lemming (Lemmus lemmus) is a small rodent distributed across the Fennoscandian mountain tundra and the Kola Peninsula. The Norwegian lemming likely evolved during the Late Pleistocene and inhabited Fennoscandia shortly prior to the Last Glacial Maximum. However, the exact timing and origins of the species, and its phylogenetic position relative to the closely related Siberian lemming (Lemmus sibiricus) remain disputed. Moreover, the presence of ancient or contemporary gene flow between both species is largely untested. The Norwegian lemming displays characteristic phenotypic and behavioral adaptations (e.g., coat color, aggression) that are not present in other Lemmus species. We generated a de novo genome assembly for the Norwegian lemming and resequenced nine modern and two ancient Lemmus spp. genomes. We show that all Lemmus species form distinct monophyletic clades, with concordant topology between the mitochondrial and nuclear genome phylogenies. The Siberian lemming is divided into two distinct but paraphyletic clades, one in the east and one in the west, where the western clade represents a sister taxon to the Norwegian lemming. We estimate that the Norwegian and western Siberian lemming diverged shortly before the Last Glacial Maximum, making the Norwegian lemming one of the youngest known mammalian species. We did not find any indication of gene flow between L. lemmus and L. sibiricus, suggesting postglacial isolation of L. lemmus. Furthermore, we identify species-specific genomic differences in genes related to coat color and fat transport, which are likely associated with the distinctive coloration and overwintering behavior observed in the Norwegian lemming.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40587810" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1073/pnas.2424333122" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/19a4115c6ad14acaaf738a62c51e4f69" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_784df73e8dc84ac78534a0c33e5aec03" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Cytoskeletal alterations in neuronal cells implicate Toxoplasma gondii secretory machinery and host microRNA-containing extracellular vesicles.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Thomas Mazza">T Mazza</span>, <span class="pub-author" title="Morteza Aslanzadeh">M Aslanzadeh</span>, <span class="pub-author" title="Lïse Berentsen">L Berentsen</span>, <span class="pub-author" title="Franziska Bonath">F Bonath</span>, <span class="pub-author" title="Marc R Friedländer">MR Friedländer</span>, <span class="pub-author" title="Antonio Barragan">A Barragan</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The widespread protozoan Toxoplasma gondii chronically infects neural tissue in vertebrates and is linked to various neurological and neuropsychiatric disorders in humans. However, its effects on sparsely infected neurons and on broader neural circuits remain elusive. Our study reveals that T. gondii infection disrupts cytoskeletal dynamics in SH-SY5Y neuronal cells and primary cortical neurons. Infected neuronal cells undergo significant cytomorphological changes, including retraction of dendritic extensions and alterations in microtubule and F-actin networks, across both parasite genotypes I and II. These cytoskeletal alterations were notably diminished in cells exposed to T. gondii mutants with impaired secretion via the MYR translocon, and were independent of intraneuronal parasite replication. Moreover, a bystander effect was observed, with supernatants from T. gondii-challenged cells inducing similar cytoskeletal changes in uninfected cells. Analyses of extracellular vesicles (EVs) in supernatants revealed differential expression of host microRNAs in response to infection, most notably the upregulation of miR-221-3p, a microRNA not previously associated with T. gondii. The data indicate that unidentified parasite-derived effector(s) secreted via the MYR translocon, in conjunction with MYR-independently induced EV-associated host microRNAs, mediate cytoskeletal alterations in both infected and bystander neuronal cells. The findings provide new insights into molecular mechanisms by which T. gondii infection may disrupt neural networks, shedding light on its potential role in neuronal dysregulation.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40221584" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41598-025-96298-8" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/784df73e8dc84ac78534a0c33e5aec03" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_c42cf07b07844cabbe0ad109571234eb" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Cellular heterogeneity in metabolism and associated microbiome of a non-model phytoflagellate.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Aditya Jeevannavar">A Jeevannavar</span>, <span class="pub-author" title="Javier Florenza">J Florenza</span>, <span class="pub-author" title="Anna-Maria Divne">A Divne</span>, <span class="pub-author" title="Manu Tamminen">M Tamminen</span>, <span class="pub-author" title="Stefan Bertilsson">S Bertilsson</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Single-cell transcriptomics is a key tool for unravelling metabolism and tissue diversity in model organisms. Its potential for elucidating the ecological roles of microeukaryotes, especially non-model ones, remains largely unexplored. This study employed the Smart-seq2 protocol on Ochromonas triangulata, a microeukaryote lacking a reference genome, showcasing how transcriptional states align with two distinct growth phases: a fast-growing phase and a slow-growing phase. Besides the two expected expression clusters, each corresponding to either growth phase, a third transcriptional state was identified across both growth phases. Metabolic mapping revealed a boost of photosynthetic activity in the fast growth over the slow growth stage, as well as downregulation trend in pathways associated with ribosome functioning, CO2 fixation, and carbohydrate catabolism characteristic of the third transcriptional state. In addition, carry-over rRNA reads recapitulated the taxonomic identity of the target while revealing distinct bacterial communities, in co-culture with the eukaryote, each associated with distinct transcriptional states. This study underscores single-cell transcriptomics as a powerful tool for characterizing metabolic states in microeukaryotes without a reference genome, offering insights into unknown physiological states and individual-level interactions with different bacterial taxa. This approach holds broad applicability to describe the ecological roles of environmental microeukaryotes, culture-free, and reference-free, surpassing alternative methods like metagenomics or metatranscriptomics.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40057978" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1093/ismejo/wraf046" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/c42cf07b07844cabbe0ad109571234eb" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_94a6617807b3431a961787b517bc63a0" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Orthopteran Neo-Sex Chromosomes Reveal Dynamics of Recombination Suppression and Evolution of Supergenes.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Suvratha Jayaprasad">S Jayaprasad</span>, <span class="pub-author" title="Valentina Peona">V Peona</span>, <span class="pub-author" title="Simon J Ellerstrand">SJ Ellerstrand</span>, <span class="pub-author" title="Roberto Rossini">R Rossini</span>, <span class="pub-author" title="Ignas Bunikis">I Bunikis</span>, <span class="pub-author" title="Olga V Pettersson">OV Pettersson</span>, <span class="pub-author" title="Remi-André Olsen">RA Olsen</span>, <span class="pub-author" title="Carl-Johan Rubin">CJ Rubin</span>, <span class="pub-author" title="Elisabet Einarsdottir">E Einarsdottir</span>, <span class="pub-author" title="Franziska Bonath">F Bonath</span>, <span class="pub-author" title="Tessa M Bradford">TM Bradford</span>, <span class="pub-author" title="Steven J B Cooper">SJB Cooper</span>, <span class="pub-author" title="Bengt Hansson">B Hansson</span>, <span class="pub-author" title="Alexander Suh">A Suh</span>, <span class="pub-author" title="Takeshi Kawakami">T Kawakami</span>, <span class="pub-author" title="Holger Schielzeth">H Schielzeth</span>, <span class="pub-author" title="Octavio M Palacios-Gimenez">OM Palacios-Gimenez</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The early evolution of sex chromosomes has remained obscure for more than a century. The Vandiemenella viatica species group of morabine grasshoppers is highly suited for studying the early stages of sex chromosome divergence and degeneration of the Y chromosome. This stems from the fact that neo-XY sex chromosomes have independently evolved multiple times by X-autosome fusions with different autosomes. Here, we generated new chromosome-level assemblies for two chromosomal races representing karyotypes with and without neo-sex chromosomes (P24XY and P24X0), and sequence data of a third chromosomal race with a different neo-XY chromosome system (P25XY). Interestingly, these two neo-XY chromosomal races are formed by different X-autosome fusions (involving chr1 and chrB, respectively), and we found that both neo-Y chromosomes have partly ceased to recombine with their neo-X counterpart. We show that the neo-XY chromosomes have diverged through accumulation of SNPs and structural mutations, and that many neo-Y-linked genes have degenerated since recombination ceased. However, the non-recombining regions of neo-Y chromosomes host non-degenerated genes crucial for sex determination, such as sex-lethal and transformer, alongside genes associated with spermatogenesis, fertility, and reproduction, illustrating their integrative role as a masculinizing supergene. Contrary to expectations, the neo-Y chromosomes showed (slightly) lower density of transposable elements (TEs) compared to other genomic regions. The study reveals the unique dynamics of young sex chromosomes, with evolution of recombination suppression and pronounced decay of (some) neo-sex chromosome genes, and provides a compelling case illustrating how chromosomal fusions and post-fusion mutational processes contribute to the evolution of supergenes.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/39475093" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/mec.17567" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/94a6617807b3431a961787b517bc63a0" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_06092f1c02924d8da8dd7a4929fd5ac3" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Origin, structure, and composition of the spider major ampullate silk fiber revealed by genomics, proteomics, and single-cell and spatial transcriptomics.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Sumalata Sonavane">S Sonavane</span>, <span class="pub-author" title="Sameer Hassan">S Hassan</span>, <span class="pub-author" title="Urmimala Chatterjee">U Chatterjee</span>, <span class="pub-author" title="Lucile Soler">L Soler</span>, <span class="pub-author" title="Lena Holm">L Holm</span>, <span class="pub-author" title="Annelie Mollbrink">A Mollbrink</span>, <span class="pub-author" title="Gabriele Greco">G Greco</span>, <span class="pub-author" title="Noah Fereydouni">N Fereydouni</span>, <span class="pub-author" title="Olga Vinnere Pettersson">O Vinnere Pettersson</span>, <span class="pub-author" title="Ignas Bunikis">I Bunikis</span>, <span class="pub-author" title="Allison Churcher">A Churcher</span>, <span class="pub-author" title="Henrik Lantz">H Lantz</span>, <span class="pub-author" title="Jan Johansson">J Johansson</span>, <span class="pub-author" title="Johan Reimegård">J Reimegård</span>, <span class="pub-author" title="Anna Rising">A Rising</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Spiders produce nature's toughest fiber using renewable components at ambient temperatures and with water as solvent, making it highly interesting to replicate for the materials industry. Despite this, much remains to be understood about the bioprocessing and composition of spider silk fibers. Here, we identify 18 proteins that make up the spiders' strongest silk type, the major ampullate fiber. Single-cell RNA sequencing and spatial transcriptomics revealed that the secretory epithelium of the gland harbors six cell types. These cell types are confined to three distinct glandular zones that produce specific combinations of silk proteins. Image analysis of histological sections showed that the secretions from the three zones do not mix, and proteomics analysis revealed that these secretions form layers in the final fiber. Using a multi-omics approach, we provide substantial advancements in the understanding of the structure and function of the major ampullate silk gland as well as of the architecture and composition of the fiber it produces.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/39141739" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/sciadv.adn0597" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/06092f1c02924d8da8dd7a4929fd5ac3" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_7d3c4336130e450099e81cfa741064ec" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Adaptive introgression reveals the genetic basis of a sexually selected syndrome in wall lizards.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Nathalie Feiner">N Feiner</span>, <span class="pub-author" title="Weizhao Yang">W Yang</span>, <span class="pub-author" title="Ignas Bunikis">I Bunikis</span>, <span class="pub-author" title="Geoffrey M While">GM While</span>, <span class="pub-author" title="Tobias Uller">T Uller</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The joint expression of particular colors, morphologies, and behaviors is a common feature of adaptation, but the genetic basis for such "phenotypic syndromes" remains poorly understood. Here, we identified a complex genetic architecture associated with a sexually selected syndrome in common wall lizards, by capitalizing on the adaptive introgression of coloration and morphology into a distantly related lineage. Consistent with the hypothesis that the evolution of phenotypic syndromes in vertebrates is facilitated by developmental linkage through neural crest cells, most of the genes associated with the syndrome are involved in neural crest cell regulation. A major locus was a ~400-kb region, characterized by standing structural genetic variation and previously implied in the evolutionary innovation of coloration and beak size in birds. We conclude that features of the developmental and genetic architecture contribute to maintaining trait integration, facilitating the extensive and rapid introgressive spread of suites of sexually selected characters.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/38569035" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/sciadv.adk9315" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/7d3c4336130e450099e81cfa741064ec" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_a1a66eb9378f48bcbb9c32923b5503e3" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genome Variation in Three Anthophora Bee Species Reflects Divergent Demographic Histories</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Demetris Taliadoros">D Taliadoros</span>, <span class="pub-author" title="André E R Soares">AER Soares</span>, <span class="pub-author" title="Guilherme Dias">G Dias</span>, <span class="pub-author" title="Ignas Bunikis">I Bunikis</span>, <span class="pub-author" title="Martin Pippel">M Pippel</span>, <span class="pub-author" title="Anna Olsson">A Olsson</span>, <span class="pub-author" title="Mai‐Britt Mosbech">M Mosbech</span>, <span class="pub-author" title="Julia Heintz">J Heintz</span>, <span class="pub-author" title="Nina Lager">N Lager</span>, <span class="pub-author" title="Anna‐Sofi Strand">A Strand</span>, <span class="pub-author" title="Mats Pettersson">M Pettersson</span>, <span class="pub-author" title="Olga Vinnere Pettersson">OV Pettersson</span>, <span class="pub-author" title="Henrik Lantz">H Lantz</span>, <span class="pub-author" title="Björn Cederberg">B Cederberg</span>, <span class="pub-author" title="Kerstin Lindblad‐Toh">K Lindblad‐Toh</span>, <span class="pub-author" title="Matthew T Webster">MT Webster</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Population genomics can reveal trends and drivers of biodiversity loss, but it is still unclear how best to use measures of genome variation to understand population vulnerability in insects. Here we study genome variation in three species of Anthophora bees that show contrasting population trends in northern Europe. Two species, Anthophora plagiata and Anthophora retusa , have experienced declines and recoveries of different magnitudes in the last 50 years, whereas a third species, Anthophora quadrimaculata , has relative population stability. We generate highly contiguous genome assemblies and use them to study genome variation in 136 samples of these species collected throughout Sweden. We find exceedingly low genetic variation in A. plagiata , which has experienced a severe recent bottleneck, but high genetic variation in A. retusa , despite a similar recent population trajectory. Fragmented populations of the threatened species A. plagiata appear isolated from each other, but in A. retusa, there is a lack of deep population structure among geographically separated subpopulations. We infer population size in the distant past using MSMC2 and recent past using GONE. These methods are remarkably concordant and indicate ancient fluctuations in population size dating back to the Pleistocene, with moderate expansions in the past century in all three species. These results are comparable to some other studies of endangered insects, which have experienced population declines that predate the modern era. We detect long blocks of identity-by-state in A. plagiata , indicative of severe recent inbreeding. Translocations between isolated populations of this species could have a positive effect on their resilience.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41387163" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/mec.70204" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/a1a66eb9378f48bcbb9c32923b5503e3" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_fdb2a58a717e4bbca202566f67f5792b" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A co-speciation dilemma and a lifestyle transition with genomic consequences in Wolbachia of Neotropical Drosophila</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Konstantinos Papachristos">K Papachristos</span>, <span class="pub-author" title="Wolfgang J Miller">WJ Miller</span>, <span class="pub-author" title="Lisa Klasson">L Klasson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Long-term persistent symbiotic associations may result in co-speciation and can be inferred if species trees of hosts and symbionts are congruent in topology and divergence times. Co-speciation has been seen to occur relatively frequently in obligate associations, but is less common in parasitic or facultative ones, mainly due to the difference in horizontal transmission rates. The long-term vertical inheritance and close host association of obligate endosymbionts also generally result in smaller genomes than in facultative endosymbionts. Here, we investigate co-speciation and genome reduction using highly similar strains of the endosymbiont Wolbachia infecting Drosophila species from the willistoni and saltans groups, where only one strain, wPau, infecting D. paulistorum, is obligate.

We sequenced the Wolbachia genomes from five species of the willistoni and saltans groups and constructed phylogenies. Topological congruence was found between these Wolbachia strains and the nuclear DNA of their hosts, except for wPau and D. paulistorum, but full topological congruence was observed between Wolbachia and the host mitochondrial DNA. However, assuming temporal congruence, we estimated extremely low evolutionary rates in Wolbachia of 10- 10-10- 11 changes/site/year. Additionally, the obligate wPau strain was found to have a larger genome than closely related facultative strains, mainly due to an ongoing expansion of an IS4 element. Furthermore, wPau has lost a large proportion of its prophage WO genes, but the cif genes, known to be involved in the CI phenotype, are intact. Finally, nine of the eleven genes from the prophage WO-associated Undecim cluster are uniquely duplicated.

The congruent topologies between Wolbachia and their willistoni and saltans group hosts indicate co-speciation. However, the high similarity between Wolbachia strains, which results in low mutation rate estimates, challenges this interpretation. Contrary to the expectations of the genome reduction theory, we observed an increase in genome size in the obligate wPau strain, potentially driven by a decreased population size. Finally, the duplication of the Undecim cluster, despite a major loss of other prophage-associated genes, suggests that the genes in the Undecim cluster are under strong selection and potentially play a role in the obligate association between wPau and their D. paulistorum hosts.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41366724" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12864-025-12340-z" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/fdb2a58a717e4bbca202566f67f5792b" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_b8ca2a2dba2049ec8de7192c162898c4" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Homo sapiens-specific evolution unveiled by ancient southern African genomes.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Mattias Jakobsson">M Jakobsson</span>, <span class="pub-author" title="Carolina Bernhardsson">C Bernhardsson</span>, <span class="pub-author" title="James McKenna">J McKenna</span>, <span class="pub-author" title="Nina Hollfelder">N Hollfelder</span>, <span class="pub-author" title="Mario Vicente">M Vicente</span>, <span class="pub-author" title="Hanna Edlund">H Edlund</span>, <span class="pub-author" title="Alexandra Coutinho">A Coutinho</span>, <span class="pub-author" title="Per Sjödin">P Sjödin</span>, <span class="pub-author" title="James Brink">J Brink</span>, <span class="pub-author" title="Bernhard Zipfel">B Zipfel</span>, <span class="pub-author" title="Helena Malmström">H Malmström</span>, <span class="pub-author" title="Marlize Lombard">M Lombard</span>, <span class="pub-author" title="Carina M Schlebusch">CM Schlebusch</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Homo sapiens evolved hundreds of thousands of years ago in Africa, later spreading across the globe1, but the early evolutionary process is debated2-6. Here we present whole-genome sequencing data for 28 ancient southern African individuals, including six individuals with 25× to 7.2× genome coverage, dated to between 10,200 and 150 calibrated years before present (cal. BP). All ancient southern Africans dated to more than 1,400 cal. BP show a genetic make-up that is outside the range of genetic variation in modern-day humans (including southern African Khoe-San people, although some retain up to 80% ancient southern African ancestry), manifesting in a large fraction of Homo sapiens-specific variants that are unique to ancient southern Africans. Homo sapiens-specific variants at amino acid-altering sites fixed for all humans-which are likely to have evolved rapidly on the Homo sapiens branch-were enriched for genes associated with kidney function. Some Homo sapiens-specific variants fixed in ancient southern Africans-which are likely to have adapted rapidly on the southern African branch-were enriched for genes associated with protection against ultraviolet light. The ancient southern Africans show little spatiotemporal stratification for 9,000 years, consistent with a large, stable Holocene population transcending archaeological phases. While southern Africa served as a long-standing geographical refugium, there is outward gene flow over 8,000 years ago; however, inward gene flow manifests only after around 1,400 years ago. The ancient genomes reported here are therefore key to the evolution of Homo sapiens, and are important for advancing our understanding of human genomic variation.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41339558" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41586-025-09811-4" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/b8ca2a2dba2049ec8de7192c162898c4" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_bed19b398aa143d28064d622d7f4babf" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Holotype genome of the lesula provides insights into demography and evolution of a threatened primate lineage.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Axel Jensen">A Jensen</span>, <span class="pub-author" title="Emma R Horton">ER Horton</span>, <span class="pub-author" title="Mardoché B Koko">MB Koko</span>, <span class="pub-author" title="Kate M Detwiler">KM Detwiler</span>, <span class="pub-author" title="Katerina Guschanski">K Guschanski</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The development of genome sequencing techniques has revolutionized evolutionary biology, facilitating the study of adaptation and speciation at the genome level. Genomic data has also become a cornerstone in conservation management, allowing inferences of population demography and genetic diversity.

We sequence the genome of the holotype specimen of the elusive lesula (Cercopithecus lomamiensis), a recently described member of the guenons (tribe Cercopithecini), endemic to the Democratic Republic of the Congo. Using published and novel genomic data, we explore the evolutionary and demographic history of C. lomamiensis and its sister species C. hamlyni. We estimate that the two species split ca. 3–4 million years ago, and find that they both show high genetic diversity despite being listed as Vulnerable on the IUCN Red List. We identify signatures of positive selection in genes involved in pelage coloration and immune functions, as well as skeletal morphology and locomotor behavior, potentially related to the terrestrial lifestyle of C. lomamiensis and C. hamlyni, which stand out among the otherwise arboreal Cercopithecus genus. We specifically explore whether introgression from more distantly related terrestrial guenons was involved in the evolution of terrestriality in the hamlyni group, but found low molecular convergence suggesting that putative terrestrial adaptations occurred largely independently.

This study provides insights into the demography and evolutionary history in a poorly known, threatened primate lineage. Furthermore, our results suggest that genomic erosion is not an imminent threat to these species, and that conservation management should prioritize actions to prevent further population decline.

The online version contains supplementary material available at 10.1186/s13059-025-03877-z.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41327385" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s13059-025-03877-z" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/bed19b398aa143d28064d622d7f4babf" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_823c71158dcd4b9fb085d01e55d13e48" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">scFFPE-ATAC enables high-throughput single cell chromatin accessibility profiling in formalin-fixed paraffin-embedded samples.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ram Prakash Yadav">RP Yadav</span>, <span class="pub-author" title="Pengwei Xing">P Xing</span>, <span class="pub-author" title="Miao Zhao">M Zhao</span>, <span class="pub-author" title="Peter Hollander">P Hollander</span>, <span class="pub-author" title="Carina Strell">C Strell</span>, <span class="pub-author" title="Minglu Xie">M Xie</span>, <span class="pub-author" title="Maede Salehi">M Salehi</span>, <span class="pub-author" title="Emma Torell">E Torell</span>, <span class="pub-author" title="Tobias Sjöblom">T Sjöblom</span>, <span class="pub-author" title="Gunilla Enblad">G Enblad</span>, <span class="pub-author" title="Rose-Marie Amini">RM Amini</span>, <span class="pub-author" title="Fredrik Johansson Swartling">FJ Swartling</span>, <span class="pub-author" title="Ingrid Glimelius">I Glimelius</span>, <span class="pub-author" title="Patrick Micke">P Micke</span>, <span class="pub-author" title="Mats Hellström">M Hellström</span>, <span class="pub-author" title="Xingqi Chen">X Chen</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Formalin-fixed paraffin-embedded (FFPE) samples are the gold standard for tissue preservation in clinical and research settings. Current single-cell chromatin accessibility technologies cannot resolve cell-type-specific epigenetic profiles in FFPE tissues due to extensive DNA damage. We present scFFPE-ATAC, a high-throughput single-cell chromatin accessibility assay for FFPE samples that integrates an FFPE-adapted Tn5 transposase, ultra-high-throughput DNA barcoding (>56 million barcodes per run), T7 promoter-mediated DNA damage repair, and in vitro transcription. We benchmark scFFPE-ATAC on FFPE mouse spleen and validate its performance against fresh tissue. We apply it to human lymph node samples archived for 8-12 years and to lung cancer FFPE tissues, revealing distinct regulatory trajectories between tumor center and invasive edge. Analysis of archived follicular lymphoma and transformed diffuse large B-cell lymphoma samples identifies relapse- and transformation-associated epigenetic dynamics. scFFPE-ATAC enables retrospective, spatial, and mechanistic epigenetic studies in long-term archived specimens.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41238550" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41467-025-66170-4" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/823c71158dcd4b9fb085d01e55d13e48" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_5087e22117c64ebbbc4b4299bfc98b9a" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A [11C]PBR28 PET study on the associations between sleep health and microglial density.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Leonie Jt Balter">LJ Balter</span>, <span class="pub-author" title="Jonatan Malmros">J Malmros</span>, <span class="pub-author" title="Per Stenkrona">P Stenkrona</span>, <span class="pub-author" title="Andrea Varrone">A Varrone</span>, <span class="pub-author" title="Anton Forsberg">A Forsberg</span>, <span class="pub-author" title="Erik Gustavsson">E Gustavsson</span>, <span class="pub-author" title="Cedrique E Mouyobo">CE Mouyobo</span>, <span class="pub-author" title="Grégoria Kalpouzos">G Kalpouzos</span>, <span class="pub-author" title="Goran Papenberg">G Papenberg</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Sleep disturbances and inflammation are interconnected through shared regulatory mechanisms and are both implicated in age-related diseases. However, their connection at the level of brain-specific inflammation remains underexamined in humans. This study investigated whether specific dimensions of sleep are associated with microglial density, as measured by translocator protein (TSPO) levels, a biomarker of neuroinflammation. TSPO levels were measured using a single [11C]PBR28 positron emission tomography (PET) scan in 39 healthy adults aged 50-81 years (Mage = 66.7, SD = 8.9; 19 females, 20 males). Sleep dimensions were assessed using the Karolinska Sleep Questionnaire on three occasions over five years, twice before and once around the time of PET imaging. Shorter sleep, more frequent napping, daytime fatigue, and sleep insufficiency were associated with higher TSPO levels in the middle frontal cortex (MFC). Conversely, longer sleep was associated with higher TSPO levels in the hippocampus and putamen. Exploratory factor analysis and bootstrapping confirmed a negative association between a factor representing shorter sleep and MFC TSPO levels. Additionally, a greater deviation from optimal sleep duration over the five years, in either direction from eight hours, was associated with higher current TSPO levels in all but one examined brain regions. Peripheral C-reactive protein levels did not significantly correlate with the sleep variables or TSPO levels in any of the brain regions. Analyses were adjusted for age and sex. These findings suggest that insufficient and prolonged sleep durations are associated with elevated microglial density in frontostriatal and limbic systems, respectively, among healthy middle-aged and older adults, without any associations with peripheral inflammation. Further longitudinal studies are needed to clarify directionality and whether changes in sleep duration over time may serve as early indicators of brain health.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41239401" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12974-025-03613-1" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/5087e22117c64ebbbc4b4299bfc98b9a" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_f3a1b1396edb4101b0575fbc43ec7415" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Comparative evaluation of Olink Explore 3072 and mass spectrometry with peptide fractionation for plasma proteomics.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Noora Sissala">N Sissala</span>, <span class="pub-author" title="Haris Babačić">H Babačić</span>, <span class="pub-author" title="Isabelle R Leo">IR Leo</span>, <span class="pub-author" title="Xiaofang Cao">X Cao</span>, <span class="pub-author" title="Jenny Forshed">J Forshed</span>, <span class="pub-author" title="Lars E Eriksson">LE Eriksson</span>, <span class="pub-author" title="Janne Lehtiö">J Lehtiö</span>, <span class="pub-author" title="Claudia Fredolini">C Fredolini</span>, <span class="pub-author" title="Mikael Åberg">M Åberg</span>, <span class="pub-author" title="Maria Pernemalm">M Pernemalm</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Plasma proteomics technologies are advancing rapidly, offering new opportunities for biomarker discovery and precision medicine. Direct comparisons of available technologies are needed to understand how platform selection affects downstream findings. We compared the performance of a peptide fractionation-based mass spectrometry method (HiRIEF LC-MS/MS) and the Olink Explore 3072 proximity extension assays on 88 plasma samples, analyzing 1129 proteins with both methods. The platforms exhibited complementary proteome coverage, high precision, and concordance in estimating sex differences in protein levels. Quantitative agreement between platforms was moderate (median correlation 0.59, interquartile range 0.33-0.75), mainly influenced by technical factors. Finally, we present a publicly available tool for peptide-level analysis of platform agreement and demonstrate its utility in clarifying cross-platform discrepancies in protein and proteoform measurements. Our findings provide insights for platform selection and study design, and highlight the value of combining mass spectrometry and affinity-based approaches for more comprehensive and reliable plasma proteome profiling.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41188494" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s42004-025-01753-2" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/f3a1b1396edb4101b0575fbc43ec7415" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_1938da5a9fb54644b9fd03953d33a22d" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Autoreactive T cells identified in patients with anti-Jo1+ antisynthetase syndrome recognise a new epitope on histidyl t-RNA synthetase.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Angeles S Galindo-Feria">AS Galindo-Feria</span>, <span class="pub-author" title="Ravi Kumar Sharma">RK Sharma</span>, <span class="pub-author" title="Anatoly Dubnovitsky">A Dubnovitsky</span>, <span class="pub-author" title="Christina Gerstner">C Gerstner</span>, <span class="pub-author" title="Genadiy Kozhukh">G Kozhukh</span>, <span class="pub-author" title="Annika Van Vollenhoven">A Van Vollenhoven</span>, <span class="pub-author" title="Juan Sebastian Diaz Boada">JSD Boada</span>, <span class="pub-author" title="Daniel Ramsköld">D Ramsköld</span>, <span class="pub-author" title="Adnane Achour">A Achour</span>, <span class="pub-author" title="Maryam Dastmalchi">M Dastmalchi</span>, <span class="pub-author" title="Hugh H Reid">HH Reid</span>, <span class="pub-author" title="Tatyana Sandalova">T Sandalova</span>, <span class="pub-author" title="Jamie Rossjohn">J Rossjohn</span>, <span class="pub-author" title="Karine Chemin">K Chemin</span>, <span class="pub-author" title="Vivianne Malmström">V Malmström</span>, <span class="pub-author" title="Ingrid E Lundberg">IE Lundberg</span>, <span class="pub-author" title="Begum Horuluoglu">B Horuluoglu</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Anti-Jo1+ antisynthetase syndrome (ASyS) is characterised by autoantibodies targeting histidyl t-RNA synthetase (HisRS), association with HLA-DRB1*03:01 and a distinct clinical phenotype including interstitial lung disease, myositis, arthritis, and mechanic's hands. Previous studies of autoreactive HisRS-specific CD4+T cells point to yet undiscovered T cell epitopes. We aimed to identify new epitopes on HisRS to investigate the presence of autoreactive T cells and their corresponding T-cell receptor (TCR) repertoire from patients with ASyS.

Peptides from HisRS N-terminal region with appropriate major histocompatibility complex (MHC) anchor residues were selected for in vitro binding assays. The peptide (HisRS41-55) with the highest HLA-DRB1*03:01 binding affinity was selected for studies with HLA-class II tetramers. Peripheral blood mononuclear cells (PBMCs) from patients with ASyS with HLA-DRB1*03:01 (n = 12) were stimulated in vitro with peptide and peptide-HLA-DRB1*03:01 tetramers were used to detect HisRS+CD4+T cells. Single TCR sequencing of captured T cells allowed analyses of the underlying TCR repertoire.

We identified a new T cell epitope on HisRS with high affinity for HLA-DRB1*03:01. Autoreactive HisRS+CD4+T cells were detected in PBMCs of patients (n = 6/12). TCR repertoire analysis of HisRS+CD4+T cells revealed shared gene V-alpha and beta usages. Moreover, HisRS+CD4+T cells persisted after treatment in 2 patients (P2 and P4) and 2 identical T cell clones were detected between the initial and follow-up time points in 1 patient (P2).

Autoreactive T-cells targeting a new HisRS epitope were identified indicating T cell reactivity to diverse epitopes of the HisRS protein in patients with anti-Jo1 autoantibodies. Furthermore, we demonstrated the TCR repertoire of autoreactive HisRS+CD4+T cells in patients. Persistence of these T-cells and specific clones may be contributing to disease.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41139557" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ard.2025.09.015" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/1938da5a9fb54644b9fd03953d33a22d" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_a303b58b4fa1409b9589100fef20ee82" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Postnatal sustentacular cells as chromaffin progenitors and tumor cells of origin in VHL-related paragangliomas.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Petra Bullova">P Bullova</span>, <span class="pub-author" title="Peng Cui">P Cui</span>, <span class="pub-author" title="Maria Arceo">M Arceo</span>, <span class="pub-author" title="Jiacheng Zhu">J Zhu</span>, <span class="pub-author" title="Wenyu Li">W Li</span>, <span class="pub-author" title="Monika Plescher">M Plescher</span>, <span class="pub-author" title="Valentin Poltorachenko">V Poltorachenko</span>, <span class="pub-author" title="Katerina Stripling">K Stripling</span>, <span class="pub-author" title="Christian Santangeli">C Santangeli</span>, <span class="pub-author" title="Lidiya Mykhaylechko">L Mykhaylechko</span>, <span class="pub-author" title="Maria Eleni Kastriti">ME Kastriti</span>, <span class="pub-author" title="Catharina Larsson">C Larsson</span>, <span class="pub-author" title="C Christofer Juhlin">CC Juhlin</span>, <span class="pub-author" title="Michael Mints">M Mints</span>, <span class="pub-author" title="Susanne Schlisio">S Schlisio</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The cellular source of chromaffin cell regeneration after birth and its relationship to paraganglioma tumorigenesis remains incompletely defined. Here, we identify a postnatal population of SOX2/SOX10-expressing sustentacular glia-like cells in the organ of Zuckerkandl (OZ) and adrenal gland that give rise to chromaffin cells in vivo. These cells differ transcriptionally from embryonic chromaffin progenitors known as Schwann cell precursors and exhibit a unique progenitor signature. Genetic lineage tracing confirms their postnatal contribution to chromaffin cells, and SOX2+PHOX2B+ transitional cells were observed in both human and mouse OZ and adrenal tissues. Single-nuclei RNA-seq and inferCNA analysis of pheochromocytoma and paraganglioma (PPGL) revealed that while most sustentacular cells exhibit a stromal profile, a subset in VHL-mutated PPGLs harbor the hallmark 3p chromosomal loss shared with chief tumor cells, suggesting a clonal origin. In an additional PPGL, widespread SOX2 expression in PHOX2B+ tumor cells supports this hypothesis. Finally, DLK1-NOTCH signaling was predicted as a central regulator of chromaffin-sustentacular communication, suggesting DLK1 fine-tunes chromaffin regeneration via NOTCH inhibition and may represent a therapeutic target in PPGL.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41093965" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41698-025-01145-8" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/a303b58b4fa1409b9589100fef20ee82" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_107c7b3b512c45f686868f13aff2dfd8" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Innate immune cell subsets are enriched in synovial fluid of ACPA-negative rheumatoid arthritis and characterized by distinct type I IFN gene signatures.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Alexandra Argyriou">A Argyriou</span>, <span class="pub-author" title="Marc H Wadsworth">MH Wadsworth</span>, <span class="pub-author" title="Joshua Fienman">J Fienman</span>, <span class="pub-author" title="Ana Cristina Gonzalez-Sanchez">AC Gonzalez-Sanchez</span>, <span class="pub-author" title="Salim Ghannoum">S Ghannoum</span>, <span class="pub-author" title="Chirag Krishna">C Krishna</span>, <span class="pub-author" title="Christina Gerstner">C Gerstner</span>, <span class="pub-author" title="Begum Horuluoglu">B Horuluoglu</span>, <span class="pub-author" title="Merel Sijbranda">M Sijbranda</span>, <span class="pub-author" title="Lars Rönnblom">L Rönnblom</span>, <span class="pub-author" title="Maija-Leena Eloranta">ML Eloranta</span>, <span class="pub-author" title="Marie Wahren-Herlenius">M Wahren-Herlenius</span>, <span class="pub-author" title="Aase Hensvold">A Hensvold</span>, <span class="pub-author" title="Sara Turcinov">S Turcinov</span>, <span class="pub-author" title="Aaron Winkler">A Winkler</span>, <span class="pub-author" title="Vivianne Malmström">V Malmström</span>, <span class="pub-author" title="Karine Chemin">K Chemin</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Around 30% of patients with rheumatoid arthritis (RA) lack rheumatoid factor and anti-citrullinated protein antibodies (ACPA) complicating diagnosis and potentially delaying treatment. We hypothesised that innate immune mechanisms might be more prominent in ACPA- RA.

We performed single-cell RNA sequencing of mononuclear cells from peripheral blood (PBMC) and synovial fluid (SFMC) of patients with ACPA- and ACPA+ RA (n = 4 per group: discovery cohort; n = 8 per group: validation cohort). Dendritic cells and proinflammatory cytokine production were analysed by flow cytometry on SFMC from patients with ACPA- RA, ACPA+ RA, and psoriatic arthritis. Interferon (IFN) levels in synovial fluid (SF) and serum were measured in these groups.

Several macrophage subsets and cDC2 were enriched in ACPA- RA SF whereas the frequency of Tph and B cells was increased in ACPA+ RA SF. Type I IFN-stimulated genes were detected in SFMC, but not PBMC, of patients with ACPA- RA. A type I IFN signature was also observed in synovial tissue from two patients with ACPA- RA in an independent dataset. IFN levels were higher in SF than serum but IFN-α/β production did not differ between ACPA+ and ACPA- RA.

This study identifies a distinct innate cell composition and type I IFN gene response in synovial joints, but not in peripheral blood, of patients with ACPA- RA. Similar IFN levels across groups suggest the IFN signature may have been primed before the cells entered the joints. These findings provide a foundation for future research on type I IFN responses in ACPA- RA.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41046204" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ard.2025.07.029" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/107c7b3b512c45f686868f13aff2dfd8" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    