<div class="ngisweden-publications mb-5"><h5>User Publications</h5><div class="list-group">
            <a data-toggle="modal" data-target="#pub_d43e6d14232747caa43602a9dbe3b086" href="https://publications.scilifelab.se/publication/d43e6d14232747caa43602a9dbe3b086" 
                target="_blank" class="list-group-item list-group-item-action">
                Inflammation and neutrophil extracellular traps in cerebral cavernous malformation.<br>
                <small class="text-muted"><em>Cell. Mol. Life Sci.</em> (2022)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_8f169041de1f4c60ad274706cf0739bc" href="https://publications.scilifelab.se/publication/8f169041de1f4c60ad274706cf0739bc" 
                target="_blank" class="list-group-item list-group-item-action">
                Near Chromosome-Level Genome Assembly and Annotation of Rhodotorula babjevae Strains Reveals High Intraspecific Divergence.<br>
                <small class="text-muted"><em>JoF</em> (2022)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_c5cf7c75465a480c8ba26778bac156d6" href="https://publications.scilifelab.se/publication/c5cf7c75465a480c8ba26778bac156d6" 
                target="_blank" class="list-group-item list-group-item-action">
                Endophilin A2 deficiency protects rodents from autoimmune arthritis by modulating T cell activation.<br>
                <small class="text-muted"><em>Nat Commun</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_107c7b3b512c45f686868f13aff2dfd8" href="https://publications.scilifelab.se/publication/107c7b3b512c45f686868f13aff2dfd8" 
                target="_blank" class="list-group-item list-group-item-action">
                Innate immune cell subsets are enriched in synovial fluid of ACPA-negative rheumatoid arthritis and characterized by distinct type I IFN gene signatures.<br>
                <small class="text-muted"><em>Ann. Rheum. Dis.</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_41ddca0947d04600adb7ac6263f21f5c" href="https://publications.scilifelab.se/publication/41ddca0947d04600adb7ac6263f21f5c" 
                target="_blank" class="list-group-item list-group-item-action">
                Species-genetic diversity correlation in a metacommunity of urban pond invertebrates<br>
                <small class="text-muted"><em>Basic and Applied Ecology</em> (2024)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_c42cf07b07844cabbe0ad109571234eb" href="https://publications.scilifelab.se/publication/c42cf07b07844cabbe0ad109571234eb" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Cellular heterogeneity in metabolism and associated microbiome of a non-model phytoflagellate.<br>
                <small class="text-muted"><em>ISME J</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_7da881e6bd1c41e59a6db035fcd40b87" href="https://publications.scilifelab.se/publication/7da881e6bd1c41e59a6db035fcd40b87" 
                target="_blank" class="list-group-item list-group-item-action">
                Chromosome-level genome assembly and transcriptome-based annotation of the oleaginous yeast Rhodotorula toruloides CBS 14.<br>
                <small class="text-muted"><em>Genomics</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_d2a5def06c4c4e11b87c62b594c4363a" href="https://publications.scilifelab.se/publication/d2a5def06c4c4e11b87c62b594c4363a" 
                target="_blank" class="list-group-item list-group-item-action">
                A comprehensive map coupling histone modifications with gene regulation in adult dopaminergic and serotonergic neurons<br>
                <small class="text-muted"><em>Nat Commun</em> (2018)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_6295ee8d911c4628865b4228aa2fcc16" href="https://publications.scilifelab.se/publication/6295ee8d911c4628865b4228aa2fcc16" 
                target="_blank" class="list-group-item list-group-item-action">
                Bioinformatic tools for using whole genome sequencing as a rapid high resolution diagnostic typing tool when tracing bioterror organisms in the food and feed chain.<br>
                <small class="text-muted"><em>Int. J. Food Microbiol.</em> (2011)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_ed26a59be0924ddea2f556caaa4136c5" href="https://publications.scilifelab.se/publication/ed26a59be0924ddea2f556caaa4136c5" 
                target="_blank" class="list-group-item list-group-item-action">
                Whole genome sequencing reveals high differentiation, low levels of genetic diversity and short runs of homozygosity among Swedish wels catfish.<br>
                <small class="text-muted"><em>Heredity (Edinb)</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_19a4115c6ad14acaaf738a62c51e4f69" href="https://publications.scilifelab.se/publication/19a4115c6ad14acaaf738a62c51e4f69" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Genome analyses suggest recent speciation and postglacial isolation in the Norwegian lemming.<br>
                <small class="text-muted"><em>Proc. Natl. Acad. Sci. U.S.A.</em> (2025)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_c29aadb8392f487487302b27fddcc407" href="https://publications.scilifelab.se/publication/c29aadb8392f487487302b27fddcc407" 
                target="_blank" class="list-group-item list-group-item-action">
                Large scale library generation for high throughput sequencing.<br>
                <small class="text-muted"><em>PLoS ONE</em> (2011)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_7a923a1fcefd4484a6d57e3390730cd7" href="https://publications.scilifelab.se/publication/7a923a1fcefd4484a6d57e3390730cd7" 
                target="_blank" class="list-group-item list-group-item-action">
                BtsCI and BseGI display sequence preference in the nucleotides flanking the recognition sequence<br>
                <small class="text-muted"><em>PLoS ONE</em> (2018)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_84b19b61f97c4602a643b805bcca2c89" href="https://publications.scilifelab.se/publication/84b19b61f97c4602a643b805bcca2c89" 
                target="_blank" class="list-group-item list-group-item-action">
                Transcriptional and metabolic rewiring of colorectal cancer cells expressing the oncogenic KRASG13D mutation.<br>
                <small class="text-muted"><em>Br. J. Cancer</em> (2019)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_17a8c61af04d40a19d6176e1cda30cef" href="https://publications.scilifelab.se/publication/17a8c61af04d40a19d6176e1cda30cef" 
                target="_blank" class="list-group-item list-group-item-action">
                PRR14L mutations are associated with chromosome 22 acquired uniparental disomy, age-related clonal hematopoiesis and myeloid neoplasia.<br>
                <small class="text-muted"><em>Leukemia</em> (2018)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_5736722e01b74b29baf8fe43d4ce6819" href="https://publications.scilifelab.se/publication/5736722e01b74b29baf8fe43d4ce6819" 
                target="_blank" class="list-group-item list-group-item-action">
                RNA-seq analysis identifies key genes enhancing hoof strength to withstand barefoot racing in Standardbred trotters.<br>
                <small class="text-muted"><em>BMC Genomics</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_dfba8e4537274d90abb9a4c98eec18ae" href="https://publications.scilifelab.se/publication/dfba8e4537274d90abb9a4c98eec18ae" 
                target="_blank" class="list-group-item list-group-item-action">
                Meiofauna improve oxygenation and accelerate sulfide removal in the seasonally hypoxic seabed.<br>
                <small class="text-muted"><em>Mar Environ Res</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_d05046c85a1e4f1ca7defccbb1420945" href="https://publications.scilifelab.se/publication/d05046c85a1e4f1ca7defccbb1420945" 
                target="_blank" class="list-group-item list-group-item-action">
                Direct effects of organic pollutants on the growth and gene expression of the Baltic Sea model bacterium Rheinheimera sp. BAL341.<br>
                <small class="text-muted"><em>Microb Biotechnol</em> (2019)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_e389be71822d45d6ac04d848bfb48df6" href="https://publications.scilifelab.se/publication/e389be71822d45d6ac04d848bfb48df6" 
                target="_blank" class="list-group-item list-group-item-action">
                The power of genetic diversity in genome-wide association studies of lipids.<br>
                <small class="text-muted"><em>Nature</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_cae2c19d93fe471eac4b766e6643b20a" href="https://publications.scilifelab.se/publication/cae2c19d93fe471eac4b766e6643b20a" 
                target="_blank" class="list-group-item list-group-item-action">
                Development and characterization of an EMS-mutagenized wheat population and identification of salt-tolerant wheat lines.<br>
                <small class="text-muted"><em>BMC Plant Biol.</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_59b8dae6dc8d41c6885ed37cf0f856a3" href="https://publications.scilifelab.se/publication/59b8dae6dc8d41c6885ed37cf0f856a3" 
                target="_blank" class="list-group-item list-group-item-action">
                Improved phylogenomic sampling of free-living nematodes enhances resolution of higher-level nematode phylogeny.<br>
                <small class="text-muted"><em>BMC Evol. Biol.</em> (2019)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_f892ff63669141bc8f46e7fd8d8fde48" href="https://publications.scilifelab.se/publication/f892ff63669141bc8f46e7fd8d8fde48" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                SweGen: a whole-genome data resource of genetic variability in a cross-section of the Swedish population.<br>
                <small class="text-muted"><em>Eur. J. Hum. Genet.</em> (2017)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_f4f2d21b7c8b4a61a312b996c70f1d51" href="https://publications.scilifelab.se/publication/f4f2d21b7c8b4a61a312b996c70f1d51" 
                target="_blank" class="list-group-item list-group-item-action">
                Consistent B Cell Receptor Immunoglobulin Features Between Siblings in Familial Chronic Lymphocytic Leukemia.<br>
                <small class="text-muted"><em>Front Oncol</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_58825a68ec3c48ff88d9c006248ff996" href="https://publications.scilifelab.se/publication/58825a68ec3c48ff88d9c006248ff996" 
                target="_blank" class="list-group-item list-group-item-action">
                Full Mitogenomes in the Critically Endangered Kākāpō Reveal Major Post-Glacial and Anthropogenic Effects on Neutral Genetic Diversity.<br>
                <small class="text-muted"><em>Genes</em> (2018)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_06ca672ce48846a38877306910ec4cd1" href="https://publications.scilifelab.se/publication/06ca672ce48846a38877306910ec4cd1" 
                target="_blank" class="list-group-item list-group-item-action">
                Small‐scale thermal habitat variability may not determine seagrass resilience to climate change<br>
                <small class="text-muted"><em>Limnol Oceanogr</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_beb46404be78423ebb108a8aed3fceee" href="https://publications.scilifelab.se/publication/beb46404be78423ebb108a8aed3fceee" 
                target="_blank" class="list-group-item list-group-item-action">
                Combined prenatal Lactobacillus reuteri and ω-3 supplementation synergistically modulates DNA methylation in neonatal T helper cells.<br>
                <small class="text-muted"><em>Clin Epigenetics</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_2561d68da0bd4709abff07531e1c6b60" href="https://publications.scilifelab.se/publication/2561d68da0bd4709abff07531e1c6b60" 
                target="_blank" class="list-group-item list-group-item-action">
                Gene count from target sequence capture places three whole genome duplication events in Hibiscus L. (Malvaceae).<br>
                <small class="text-muted"><em>BMC Ecol Evol</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_06bdda0e02974b29a11b5fc22fa91b3b" href="https://publications.scilifelab.se/publication/06bdda0e02974b29a11b5fc22fa91b3b" 
                target="_blank" class="list-group-item list-group-item-action">
                Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.<br>
                <small class="text-muted"><em>Circ Genom Precis Med</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_a97fb528f0834255b1e9c8103f8ff2bc" href="https://publications.scilifelab.se/publication/a97fb528f0834255b1e9c8103f8ff2bc" 
                target="_blank" class="list-group-item list-group-item-action">
                Genome Sequence Resource for the Oomycete Taro Pathogen Phytophthora colocasiae.<br>
                <small class="text-muted"><em>Mol. Plant Microbe Interact.</em> (2018)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_d033a5c5278c4463ad6a2a211e4b46ea" href="https://publications.scilifelab.se/publication/d033a5c5278c4463ad6a2a211e4b46ea" 
                target="_blank" class="list-group-item list-group-item-action">
                Ultralow amounts of DNA from long-term archived serum samples produce high-quality methylomes.<br>
                <small class="text-muted"><em>Clin Epigenetics</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_19379a893e0b48ff9f84c9acbc2f7509" href="https://publications.scilifelab.se/publication/19379a893e0b48ff9f84c9acbc2f7509" 
                target="_blank" class="list-group-item list-group-item-action">
                Microbiological Surveillance of Biogas Plants: Targeting Acetogenic Community.<br>
                <small class="text-muted"><em>Front Microbiol</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_bd53cdd9312a4daeb054082f3ef37158" href="https://publications.scilifelab.se/publication/bd53cdd9312a4daeb054082f3ef37158" 
                target="_blank" class="list-group-item list-group-item-action">
                The genomics of heart failure: design and rationale of the HERMES consortium.<br>
                <small class="text-muted"><em>ESC Heart Failure</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_be4200f3e82b44bf886c89c0e8c2f362" href="https://publications.scilifelab.se/publication/be4200f3e82b44bf886c89c0e8c2f362" 
                target="_blank" class="list-group-item list-group-item-action">
                Temporal profiling of cytokine-induced genes in pancreatic β-cells by meta-analysis and network inference.<br>
                <small class="text-muted"><em>Genomics</em> (2014)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_4e29f3d9ea9a44c1bc81f0adee660b0b" href="https://publications.scilifelab.se/publication/4e29f3d9ea9a44c1bc81f0adee660b0b" 
                target="_blank" class="list-group-item list-group-item-action">
                Deregulation of the A-to-I RNA editing mechanism in psychiatric disorders.<br>
                <small class="text-muted"><em>Hum. Mol. Genet.</em> (2012)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_8cbc18de608b493d87efccfa5a1ae454" href="https://publications.scilifelab.se/publication/8cbc18de608b493d87efccfa5a1ae454" 
                target="_blank" class="list-group-item list-group-item-action">
                Genomic signatures of rapid adaptive divergence in a tropical montane species.<br>
                <small class="text-muted"><em>Biol. Lett.</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_2488c6cb0e74465fb22eafbfe10e83fc" href="https://publications.scilifelab.se/publication/2488c6cb0e74465fb22eafbfe10e83fc" 
                target="_blank" class="list-group-item list-group-item-action">
                Populus SVL Acts in Leaves to Modulate the Timing of Growth Cessation and Bud Set.<br>
                <small class="text-muted"><em>Front Plant Sci</em> (2022)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_6ba3a0a503434b1d9709bdead1716425" href="https://publications.scilifelab.se/publication/6ba3a0a503434b1d9709bdead1716425" 
                target="_blank" class="list-group-item list-group-item-action">
                Expansions of adaptive-like NK cells with a tissue-resident phenotype in human lung and blood.<br>
                <small class="text-muted"><em>Proc. Natl. Acad. Sci. U.S.A.</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_49398b063f5d4500a3dd373382da546a" href="https://publications.scilifelab.se/publication/49398b063f5d4500a3dd373382da546a" 
                target="_blank" class="list-group-item list-group-item-action">
                A genome-wide association study of imaging-defined atherosclerosis.<br>
                <small class="text-muted"><em>Nat Commun</em> (2025)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_3777a5cd8954424fb426f5163d4d01bf" href="https://publications.scilifelab.se/publication/3777a5cd8954424fb426f5163d4d01bf" 
                target="_blank" class="list-group-item list-group-item-action">
                Proteomics. Tissue-based map of the human proteome.<br>
                <small class="text-muted"><em>Science</em> (2015)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_c415e1ad145e4e3980e7a96e3c492895" href="https://publications.scilifelab.se/publication/c415e1ad145e4e3980e7a96e3c492895" 
                target="_blank" class="list-group-item list-group-item-action">
                Extensive Clonal Branching Shapes the Evolutionary History of High-Risk Pediatric Cancers.<br>
                <small class="text-muted"><em>Cancer Res.</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_83d87735b99145889c0e1ebe02901133" href="https://publications.scilifelab.se/publication/83d87735b99145889c0e1ebe02901133" 
                target="_blank" class="list-group-item list-group-item-action">
                Bifidobacteria-mediated immune system imprinting early in life<br>
                <small class="text-muted"><em>Cell</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_a4c3fac414964931b8a6301041dbf23d" href="https://publications.scilifelab.se/publication/a4c3fac414964931b8a6301041dbf23d" 
                target="_blank" class="list-group-item list-group-item-action">
                A comparative genomics multitool for scientific discovery and conservation.<br>
                <small class="text-muted"><em>Nature</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_5f2e4baa8bb24a13a4e27bafb78c7f35" href="https://publications.scilifelab.se/publication/5f2e4baa8bb24a13a4e27bafb78c7f35" 
                target="_blank" class="list-group-item list-group-item-action">
                Association between Aquatic Micropollutant Dissipation and River Sediment Bacterial Communities.<br>
                <small class="text-muted"><em>Environ. Sci. Technol.</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_9e5622f51ee24776be28a7e9a266fb7e" href="https://publications.scilifelab.se/publication/9e5622f51ee24776be28a7e9a266fb7e" 
                target="_blank" class="list-group-item list-group-item-action">
                Replicating associations between DNA methylation and body mass index in a longitudinal sample of older twins.<br>
                <small class="text-muted"><em>Int J Obes (Lond)</em> (2019)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_b586007ebb4542e18c827db7e108b219" href="https://publications.scilifelab.se/publication/b586007ebb4542e18c827db7e108b219" 
                target="_blank" class="list-group-item list-group-item-action">
                Experimental evidence of microbial inheritance in plants and transmission routes from seed to phyllosphere and root.<br>
                <small class="text-muted"><em>Environ. Microbiol.</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_be6dfc07d42b41648f42432ddcda8787" href="https://publications.scilifelab.se/publication/be6dfc07d42b41648f42432ddcda8787" 
                target="_blank" class="list-group-item list-group-item-action">
                Whole-genome sequencing and gene network modules predict gemcitabine/carboplatin-induced myelosuppression in non-small cell lung cancer patients.<br>
                <small class="text-muted"><em>npj Syst Biol Appl</em> (2020)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_24e0c101ab3f49ac830c669e75aa8dfa" href="https://publications.scilifelab.se/publication/24e0c101ab3f49ac830c669e75aa8dfa" 
                target="_blank" class="list-group-item list-group-item-action list-pub-collab">
                Evolutionary History, Genomic Adaptation to Toxic Diet, and Extinction of the Carolina Parakeet.<br>
                <small class="text-muted"><em>Curr. Biol.</em> (2020)</small>
                <span class="float-right" style="display:inline-block">
                <span class="badge badge-primary mt-3">NGI Collaboration</span>
                
                </span>
            </a>
        

            <a data-toggle="modal" data-target="#pub_deff2c719d7a42779cbc1bc349308d01" href="https://publications.scilifelab.se/publication/deff2c719d7a42779cbc1bc349308d01" 
                target="_blank" class="list-group-item list-group-item-action">
                How to Make a Rodent Giant: Genomic Basis and Tradeoffs of Gigantism in the Capybara, the World's Largest Rodent.<br>
                <small class="text-muted"><em>Mol. Biol. Evol.</em> (2021)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_ae4ff1411a31429f8d1c53a6fc16ed11" href="https://publications.scilifelab.se/publication/ae4ff1411a31429f8d1c53a6fc16ed11" 
                target="_blank" class="list-group-item list-group-item-action">
                PRC2-mediated repression is essential to maintain identity and function of differentiated dopaminergic and serotonergic neurons.<br>
                <small class="text-muted"><em>Sci Adv</em> (2022)</small>
                
                
                
                
            </a>
        

            <a data-toggle="modal" data-target="#pub_2a387a31c3c54f2f9d3abce4e7efdfe2" href="https://publications.scilifelab.se/publication/2a387a31c3c54f2f9d3abce4e7efdfe2" 
                target="_blank" class="list-group-item list-group-item-action">
                A Shh/Gli-driven three-node timer motif controls temporal identity and fate of neural stem cells.<br>
                <small class="text-muted"><em>Sci Adv</em> (2020)</small>
                
                
                
                
            </a>
        </div>
                <p class="small text-muted mt-2">
                    See all publications at
                    <a href="https://publications.scilifelab.se/label/National%20Genomics%20Infrastructure" 
                        target="_blank" class="text-muted">
                        publications.scilifelab.se
                    </a>
                </p>
            </div>
        <div class="modal ngisweden-publications-modal fade" id="pub_d43e6d14232747caa43602a9dbe3b086" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Inflammation and neutrophil extracellular traps in cerebral cavernous malformation.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Anthony C Y Yau">ACY Yau</span>, <span class="pub-author" title="Maria Ascencion Globisch">MA Globisch</span>, <span class="pub-author" title="Favour Chinyere Onyeogaziri">FC Onyeogaziri</span>, <span class="pub-author" title="Lei L Conze">LL Conze</span>, <span class="pub-author" title="Ross Smith">R Smith</span>, <span class="pub-author" title="Suvi Jauhiainen">S Jauhiainen</span>, <span class="pub-author" title="Monica Corada">M Corada</span>, <span class="pub-author" title="Fabrizio Orsenigo">F Orsenigo</span>, <span class="pub-author" title="Hua Huang">H Huang</span>, <span class="pub-author" title="Melanie Herre">M Herre</span>, <span class="pub-author" title="Anna-Karin Olsson">AK Olsson</span>, <span class="pub-author" title="Matteo Malinverno">M Malinverno</span>, <span class="pub-author" title="Veronica Sundell">V Sundell</span>, <span class="pub-author" title="Behnam Rezai Jahromi">B Rezai Jahromi</span>, <span class="pub-author" title="Mika Niemelä">M Niemelä</span>, <span class="pub-author" title="Aki Laakso">A Laakso</span>, <span class="pub-author" title="Cecilia Garlanda">C Garlanda</span>, <span class="pub-author" title="Alberto Mantovani">A Mantovani</span>, <span class="pub-author" title="Maria Grazia Lampugnani">MG Lampugnani</span>, <span class="pub-author" title="Elisabetta Dejana">E Dejana</span>, <span class="pub-author" title="Peetra U Magnusson">PU Magnusson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Cerebral Cavernous Malformation (CCM) is a brain vascular disease with various neurological symptoms. In this study, we describe the inflammatory profile in CCM and show for the first time the formation of neutrophil extracellular traps (NETs) in rodents and humans with CCM. Through RNA-seq analysis of cerebellum endothelial cells from wild-type mice and mice with an endothelial cell-specific ablation of the Ccm3 gene (Ccm3iECKO), we show that endothelial cells from Ccm3iECKO mice have an increased expression of inflammation-related genes. These genes encode proinflammatory cytokines and chemokines, as well as adhesion molecules, which promote recruitment of inflammatory and immune cells. Similarly, immunoassays showed elevated levels of these cytokines and chemokines in the cerebellum of the Ccm3iECKO mice. Consistently, both flow cytometry and immunofluorescence analysis showed infiltration of different subsets of leukocytes into the CCM lesions. Neutrophils, which are known to fight against infection through different strategies, including the formation of NETs, represented the leukocyte subset within the most pronounced increase in CCM. Here, we detected elevated levels of NETs in the blood and the deposition of NETs in the cerebral cavernomas of Ccm3iECKO mice. Degradation of NETs by DNase I treatment improved the vascular barrier. The deposition of NETs in the cavernomas of patients with CCM confirms the clinical relevance of NETs in CCM.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/35333979" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1007/s00018-022-04224-2" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/d43e6d14232747caa43602a9dbe3b086" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_8f169041de1f4c60ad274706cf0739bc" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Near Chromosome-Level Genome Assembly and Annotation of Rhodotorula babjevae Strains Reveals High Intraspecific Divergence.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Giselle C Martín-Hernández">GC Martín-Hernández</span>, <span class="pub-author" title="Bettina Müller">B Müller</span>, <span class="pub-author" title="Christian Brandt">C Brandt</span>, <span class="pub-author" title="Martin Hölzer">M Hölzer</span>, <span class="pub-author" title="Adrian Viehweger">A Viehweger</span>, <span class="pub-author" title="Volkmar Passoth">V Passoth</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The genus Rhodotorula includes basidiomycetous oleaginous yeast species. Rhodotorula babjevae can produce compounds of biotechnological interest such as lipids, carotenoids, and biosurfactants from low value substrates such as lignocellulose hydrolysate. High-quality genome assemblies are needed to develop genetic tools and to understand fungal evolution and genetics. Here, we combined short- and long-read sequencing to resolve the genomes of two R. babjevae strains, CBS 7808 (type strain) and DBVPG 8058, at chromosomal level. Both genomes are 21 Mbp in size and have a GC content of 68.2%. Allele frequency analysis indicates that both strains are tetraploid. The genomes consist of a maximum of 21 chromosomes with a size of 0.4 to 2.4 Mbp. In both assemblies, the mitochondrial genome was recovered in a single contig, that shared 97% pairwise identity. Pairwise identity between most chromosomes ranges from 82 to 87%. We also found indications for strain-specific extrachromosomal endogenous DNA. A total of 7591 and 7481 protein-coding genes were annotated in CBS 7808 and DBVPG 8058, respectively. CBS 7808 accumulated a higher number of tandem duplications than DBVPG 8058. We identified large translocation events between putative chromosomes. Genome divergence values between the two strains indicate that they may belong to different species.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/35448555" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.3390/jof8040323" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/8f169041de1f4c60ad274706cf0739bc" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_c5cf7c75465a480c8ba26778bac156d6" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Endophilin A2 deficiency protects rodents from autoimmune arthritis by modulating T cell activation.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ulrika Norin">U Norin</span>, <span class="pub-author" title="Carola Rintisch">C Rintisch</span>, <span class="pub-author" title="Liesu Meng">L Meng</span>, <span class="pub-author" title="Florian Forster">F Forster</span>, <span class="pub-author" title="Diana Ekman">D Ekman</span>, <span class="pub-author" title="Jonatan Tuncel">J Tuncel</span>, <span class="pub-author" title="Katrin Klocke">K Klocke</span>, <span class="pub-author" title="Johan Bäcklund">J Bäcklund</span>, <span class="pub-author" title="Min Yang">M Yang</span>, <span class="pub-author" title="Michael Y Bonner">MY Bonner</span>, <span class="pub-author" title="Gonzalo Fernandez Lahore">GF Lahore</span>, <span class="pub-author" title="Jaime James">J James</span>, <span class="pub-author" title="Klementy Shchetynsky">K Shchetynsky</span>, <span class="pub-author" title="Maria Bergquist">M Bergquist</span>, <span class="pub-author" title="Inger Gjertsson">I Gjertsson</span>, <span class="pub-author" title="Norbert Hubner">N Hubner</span>, <span class="pub-author" title="Liselotte Bäckdahl">L Bäckdahl</span>, <span class="pub-author" title="Rikard Holmdahl">R Holmdahl</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The introduction of the CTLA-4 recombinant fusion protein has demonstrated therapeutic effects by selectively modulating T-cell activation in rheumatoid arthritis. Here we show, using a forward genetic approach, that a mutation in the SH3gl1 gene encoding the endocytic protein Endophilin A2 is associated with the development of arthritis in rodents. Defective expression of SH3gl1 affects T cell effector functions and alters the activation threshold of autoreactive T cells, thereby leading to complete protection from chronic autoimmune inflammatory disease in both mice and rats. We further show that SH3GL1 regulates human T cell signaling and T cell receptor internalization, and its expression is upregulated in rheumatoid arthritis patients. Collectively our data identify SH3GL1 as a key regulator of T cell activation, and as a potential target for treatment of autoimmune diseases.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33504785" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41467-020-20586-2" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/c5cf7c75465a480c8ba26778bac156d6" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_107c7b3b512c45f686868f13aff2dfd8" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Innate immune cell subsets are enriched in synovial fluid of ACPA-negative rheumatoid arthritis and characterized by distinct type I IFN gene signatures.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Alexandra Argyriou">A Argyriou</span>, <span class="pub-author" title="Marc H Wadsworth">MH Wadsworth</span>, <span class="pub-author" title="Joshua Fienman">J Fienman</span>, <span class="pub-author" title="Ana Cristina Gonzalez-Sanchez">AC Gonzalez-Sanchez</span>, <span class="pub-author" title="Salim Ghannoum">S Ghannoum</span>, <span class="pub-author" title="Chirag Krishna">C Krishna</span>, <span class="pub-author" title="Christina Gerstner">C Gerstner</span>, <span class="pub-author" title="Begum Horuluoglu">B Horuluoglu</span>, <span class="pub-author" title="Merel Sijbranda">M Sijbranda</span>, <span class="pub-author" title="Lars Rönnblom">L Rönnblom</span>, <span class="pub-author" title="Maija-Leena Eloranta">ML Eloranta</span>, <span class="pub-author" title="Marie Wahren-Herlenius">M Wahren-Herlenius</span>, <span class="pub-author" title="Aase Hensvold">A Hensvold</span>, <span class="pub-author" title="Sara Turcinov">S Turcinov</span>, <span class="pub-author" title="Aaron Winkler">A Winkler</span>, <span class="pub-author" title="Vivianne Malmström">V Malmström</span>, <span class="pub-author" title="Karine Chemin">K Chemin</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Around 30% of patients with rheumatoid arthritis (RA) lack rheumatoid factor and anti-citrullinated protein antibodies (ACPA) complicating diagnosis and potentially delaying treatment. We hypothesised that innate immune mechanisms might be more prominent in ACPA- RA.

We performed single-cell RNA sequencing of mononuclear cells from peripheral blood (PBMC) and synovial fluid (SFMC) of patients with ACPA- and ACPA+ RA (n = 4 per group: discovery cohort; n = 8 per group: validation cohort). Dendritic cells and proinflammatory cytokine production were analysed by flow cytometry on SFMC from patients with ACPA- RA, ACPA+ RA, and psoriatic arthritis. Interferon (IFN) levels in synovial fluid (SF) and serum were measured in these groups.

Several macrophage subsets and cDC2 were enriched in ACPA- RA SF whereas the frequency of Tph and B cells was increased in ACPA+ RA SF. Type I IFN-stimulated genes were detected in SFMC, but not PBMC, of patients with ACPA- RA. A type I IFN signature was also observed in synovial tissue from two patients with ACPA- RA in an independent dataset. IFN levels were higher in SF than serum but IFN-α/β production did not differ between ACPA+ and ACPA- RA.

This study identifies a distinct innate cell composition and type I IFN gene response in synovial joints, but not in peripheral blood, of patients with ACPA- RA. Similar IFN levels across groups suggest the IFN signature may have been primed before the cells entered the joints. These findings provide a foundation for future research on type I IFN responses in ACPA- RA.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/41046204" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ard.2025.07.029" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/107c7b3b512c45f686868f13aff2dfd8" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_41ddca0947d04600adb7ac6263f21f5c" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Species-genetic diversity correlation in a metacommunity of urban pond invertebrates</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Frank Johansson">F Johansson</span>, <span class="pub-author" title="Yeserin Yildirim">Y Yildirim</span>, <span class="pub-author" title="Chaz Hyseni">C Hyseni</span>, <span class="pub-author" title="Jani Heino">J Heino</span>, <span class="pub-author" title="Jacob Höglund">J Höglund</span>, <span class="pub-author" title="Luis Mauricio Bini">LM Bini</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body d-none"></div>
                    <div class="modal-footer border-0">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/null" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.baae.2024.07.002" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/41ddca0947d04600adb7ac6263f21f5c" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_c42cf07b07844cabbe0ad109571234eb" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Cellular heterogeneity in metabolism and associated microbiome of a non-model phytoflagellate.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Aditya Jeevannavar">A Jeevannavar</span>, <span class="pub-author" title="Javier Florenza">J Florenza</span>, <span class="pub-author" title="Anna-Maria Divne">A Divne</span>, <span class="pub-author" title="Manu Tamminen">M Tamminen</span>, <span class="pub-author" title="Stefan Bertilsson">S Bertilsson</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Single-cell transcriptomics is a key tool for unravelling metabolism and tissue diversity in model organisms. Its potential for elucidating the ecological roles of microeukaryotes, especially non-model ones, remains largely unexplored. This study employed the Smart-seq2 protocol on Ochromonas triangulata, a microeukaryote lacking a reference genome, showcasing how transcriptional states align with two distinct growth phases: a fast-growing phase and a slow-growing phase. Besides the two expected expression clusters, each corresponding to either growth phase, a third transcriptional state was identified across both growth phases. Metabolic mapping revealed a boost of photosynthetic activity in the fast growth over the slow growth stage, as well as downregulation trend in pathways associated with ribosome functioning, CO2 fixation, and carbohydrate catabolism characteristic of the third transcriptional state. In addition, carry-over rRNA reads recapitulated the taxonomic identity of the target while revealing distinct bacterial communities, in co-culture with the eukaryote, each associated with distinct transcriptional states. This study underscores single-cell transcriptomics as a powerful tool for characterizing metabolic states in microeukaryotes without a reference genome, offering insights into unknown physiological states and individual-level interactions with different bacterial taxa. This approach holds broad applicability to describe the ecological roles of environmental microeukaryotes, culture-free, and reference-free, surpassing alternative methods like metagenomics or metatranscriptomics.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40057978" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1093/ismejo/wraf046" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/c42cf07b07844cabbe0ad109571234eb" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_7da881e6bd1c41e59a6db035fcd40b87" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Chromosome-level genome assembly and transcriptome-based annotation of the oleaginous yeast Rhodotorula toruloides CBS 14.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Giselle C Martín-Hernández">GC Martín-Hernández</span>, <span class="pub-author" title="Bettina Müller">B Müller</span>, <span class="pub-author" title="Mikołaj Chmielarz">M Chmielarz</span>, <span class="pub-author" title="Christian Brandt">C Brandt</span>, <span class="pub-author" title="Martin Hölzer">M Hölzer</span>, <span class="pub-author" title="Adrian Viehweger">A Viehweger</span>, <span class="pub-author" title="Volkmar Passoth">V Passoth</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Rhodotorula toruloides is an oleaginous yeast with high biotechnological potential. In order to understand the molecular physiology of lipid synthesis in R. toruloides and to advance metabolic engineering, a high-resolution genome is required. We constructed a genome draft of R. toruloides CBS 14, using a hybrid assembly approach, consisting of short and long reads generated by Illumina and Nanopore sequencing, respectively. The genome draft consists of 23 contigs and 3 scaffolds, with a N50 length of 1,529,952 bp, thus largely representing chromosomal organization. The total size of the genome is 20,534,857 bp and the overall GC content is 61.83%. Transcriptomic data from different growth conditions was used to aid species-specific gene annotation. We annotated 9464 genes and identified 11,691 transcripts. Furthermore, we demonstrated the presence of a potential plasmid, an extrachromosomal circular structure of about 11 kb with a copy number about three times as high as the other chromosomes.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34648882" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ygeno.2021.10.006" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/7da881e6bd1c41e59a6db035fcd40b87" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_d2a5def06c4c4e11b87c62b594c4363a" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A comprehensive map coupling histone modifications with gene regulation in adult dopaminergic and serotonergic neurons</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Erik Södersten">E Södersten</span>, <span class="pub-author" title="Konstantinos Toskas">K Toskas</span>, <span class="pub-author" title="Vilma Rraklli">V Rraklli</span>, <span class="pub-author" title="Katarina Tiklova">K Tiklova</span>, <span class="pub-author" title="Åsa K Björklund">ÅK Björklund</span>, <span class="pub-author" title="Markus Ringnér">M Ringnér</span>, <span class="pub-author" title="Thomas Perlmann">T Perlmann</span>, <span class="pub-author" title="Johan Holmberg">J Holmberg</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The brain is composed of hundreds of different neuronal subtypes, which largely retain their identity throughout the lifespan of the organism. The mechanisms governing this stability are not fully understood, partly due to the diversity and limited size of clinically relevant neuronal populations, which constitute a technical challenge for analysis. Here, using a strategy that allows for ChIP-seq combined with RNA-seq in small neuronal populations in vivo, we present a comparative analysis of permissive and repressive histone modifications in adult midbrain dopaminergic neurons, raphe nuclei serotonergic neurons, and embryonic neural progenitors. Furthermore, we utilize the map generated by our analysis to show that the transcriptional response of midbrain dopaminergic neurons following 6-OHDA or methamphetamine injection is characterized by increased expression of genes with promoters dually marked by H3K4me3/H3K27me3. Our study provides an in vivo genome-wide analysis of permissive/repressive histone modifications coupled to gene expression in these rare neuronal subtypes.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/29581424" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41467-018-03538-9" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/d2a5def06c4c4e11b87c62b594c4363a" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_6295ee8d911c4628865b4228aa2fcc16" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Bioinformatic tools for using whole genome sequencing as a rapid high resolution diagnostic typing tool when tracing bioterror organisms in the food and feed chain.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Bo Segerman">B Segerman</span>, <span class="pub-author" title="Dario De Medici">D De Medici</span>, <span class="pub-author" title="Monika Ehling Schulz">M Ehling Schulz</span>, <span class="pub-author" title="Patrick Fach">P Fach</span>, <span class="pub-author" title="Lucia Fenicia">L Fenicia</span>, <span class="pub-author" title="Martina Fricker">M Fricker</span>, <span class="pub-author" title="Peter Wielinga">P Wielinga</span>, <span class="pub-author" title="Bart Van Rotterdam">B Van Rotterdam</span>, <span class="pub-author" title="Rickard Knutsson">R Knutsson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The rapid technological development in the field of parallel sequencing offers new opportunities when tracing and tracking microorganisms in the food and feed chain. If a bioterror organism is deliberately spread it is of crucial importance to get as much information as possible regarding the strain as fast as possible to aid the decision process and select suitable controls, tracing and tracking tools. A lot of efforts have been made to sequence multiple strains of potential bioterror organisms so there is a relatively large set of reference genomes available. This study is focused on how to use parallel sequencing for rapid phylogenomic analysis and screen for genetic modifications. A bioinformatic methodology has been developed to rapidly analyze sequence data with minimal post-processing. Instead of assembling the genome, defining genes, defining orthologous relations and calculating distances, the present method can achieve a similar high resolution directly from the raw sequence data. The method defines orthologous sequence reads instead of orthologous genes and the average similarity of the core genome (ASC) is calculated. The sequence reads from the core and from the non-conserved genomic regions can also be separated for further analysis. Finally, the comparison algorithm is used to visualize the phylogenomic diversity of the bacterial bioterror organisms Bacillus anthracis and Clostridium botulinum using heat plot diagrams.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/20826036" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ijfoodmicro.2010.06.027" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/6295ee8d911c4628865b4228aa2fcc16" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_ed26a59be0924ddea2f556caaa4136c5" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Whole genome sequencing reveals high differentiation, low levels of genetic diversity and short runs of homozygosity among Swedish wels catfish.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Axel Jensen">A Jensen</span>, <span class="pub-author" title="Mette Lillie">M Lillie</span>, <span class="pub-author" title="Kristofer Bergström">K Bergström</span>, <span class="pub-author" title="Per Larsson">P Larsson</span>, <span class="pub-author" title="Jacob Höglund">J Höglund</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The use of genetic markers in the context of conservation is largely being outcompeted by whole-genome data. Comparative studies between the two are sparse, and the knowledge about potential effects of this methodology shift is limited. Here, we used whole-genome sequencing data to assess the genetic status of peripheral populations of the wels catfish (Silurus glanis), and discuss the results in light of a recent microsatellite study of the same populations. The Swedish populations of the wels catfish have suffered from severe declines during the last centuries and persists in only a few isolated water systems. Fragmented populations generally are at greater risk of extinction, for example due to loss of genetic diversity, and may thus require conservation actions. We sequenced individuals from the three remaining native populations (Båven, Emån, and Möckeln) and one reintroduced population of admixed origin (Helge å), and found that genetic diversity was highest in Emån but low overall, with strong differentiation among the populations. No signature of recent inbreeding was found, but a considerable number of short runs of homozygosity were present in all populations, likely linked to historically small population sizes and bottleneck events. Genetic substructure within any of the native populations was at best weak. Individuals from the admixed population Helge å shared most genetic ancestry with the Båven population (72%). Our results are largely in agreement with the microsatellite study, and stresses the need to protect these isolated populations at the northern edge of the distribution of the species.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33963302" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41437-021-00438-5" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/ed26a59be0924ddea2f556caaa4136c5" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_19a4115c6ad14acaaf738a62c51e4f69" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genome analyses suggest recent speciation and postglacial isolation in the Norwegian lemming.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Edana Lord">E Lord</span>, <span class="pub-author" title="Isabelle S Feinauer">IS Feinauer</span>, <span class="pub-author" title="André E R Soares">AER Soares</span>, <span class="pub-author" title="Vendela K Lagerholm">VK Lagerholm</span>, <span class="pub-author" title="Karin Näsvall">K Näsvall</span>, <span class="pub-author" title="Erik Ersmark">E Ersmark</span>, <span class="pub-author" title="Remi-André Olsen">RA Olsen</span>, <span class="pub-author" title="Stefan Prost">S Prost</span>, <span class="pub-author" title="Elena A Kuzmina">EA Kuzmina</span>, <span class="pub-author" title="Nickolay G Smirnov">NG Smirnov</span>, <span class="pub-author" title="John R Stewart">JR Stewart</span>, <span class="pub-author" title="Monika V Knul">MV Knul</span>, <span class="pub-author" title="Pierre Noiret">P Noiret</span>, <span class="pub-author" title="Mietje Germonpré">M Germonpré</span>, <span class="pub-author" title="Dorothee Ehrich">D Ehrich</span>, <span class="pub-author" title="Ivan Pokrovsky">I Pokrovsky</span>, <span class="pub-author" title="Vadim B Fedorov">VB Fedorov</span>, <span class="pub-author" title="Anna V Goropashnaya">AV Goropashnaya</span>, <span class="pub-author" title="Love Dalén">L Dalén</span>, <span class="pub-author" title="David Díez-Del-Molino">D Díez-Del-Molino</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">The Norwegian lemming (Lemmus lemmus) is a small rodent distributed across the Fennoscandian mountain tundra and the Kola Peninsula. The Norwegian lemming likely evolved during the Late Pleistocene and inhabited Fennoscandia shortly prior to the Last Glacial Maximum. However, the exact timing and origins of the species, and its phylogenetic position relative to the closely related Siberian lemming (Lemmus sibiricus) remain disputed. Moreover, the presence of ancient or contemporary gene flow between both species is largely untested. The Norwegian lemming displays characteristic phenotypic and behavioral adaptations (e.g., coat color, aggression) that are not present in other Lemmus species. We generated a de novo genome assembly for the Norwegian lemming and resequenced nine modern and two ancient Lemmus spp. genomes. We show that all Lemmus species form distinct monophyletic clades, with concordant topology between the mitochondrial and nuclear genome phylogenies. The Siberian lemming is divided into two distinct but paraphyletic clades, one in the east and one in the west, where the western clade represents a sister taxon to the Norwegian lemming. We estimate that the Norwegian and western Siberian lemming diverged shortly before the Last Glacial Maximum, making the Norwegian lemming one of the youngest known mammalian species. We did not find any indication of gene flow between L. lemmus and L. sibiricus, suggesting postglacial isolation of L. lemmus. Furthermore, we identify species-specific genomic differences in genes related to coat color and fat transport, which are likely associated with the distinctive coloration and overwintering behavior observed in the Norwegian lemming.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40587810" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1073/pnas.2424333122" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/19a4115c6ad14acaaf738a62c51e4f69" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_c29aadb8392f487487302b27fddcc407" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Large scale library generation for high throughput sequencing.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Erik Borgström">E Borgström</span>, <span class="pub-author" title="Sverker Lundin">S Lundin</span>, <span class="pub-author" title="Joakim Lundeberg">J Lundeberg</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Large efforts have recently been made to automate the sample preparation protocols for massively parallel sequencing in order to match the increasing instrument throughput. Still, the size selection through agarose gel electrophoresis separation is a labor-intensive bottleneck of these protocols.

In this study a method for automatic library preparation and size selection on a liquid handling robot is presented. The method utilizes selective precipitation of certain sizes of DNA molecules on to paramagnetic beads for cleanup and selection after standard enzymatic reactions.

The method is used to generate libraries for de novo and re-sequencing on the Illumina HiSeq 2000 instrument with a throughput of 12 samples per instrument in approximately 4 hours. The resulting output data show quality scores and pass filter rates comparable to manually prepared samples. The sample size distribution can be adjusted for each application, and are suitable for all high throughput DNA processing protocols seeking to control size intervals.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21589638" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1371/journal.pone.0019119" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/c29aadb8392f487487302b27fddcc407" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_7a923a1fcefd4484a6d57e3390730cd7" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">BtsCI and BseGI display sequence preference in the nucleotides flanking the recognition sequence</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="João Rosa">J Rosa</span>, <span class="pub-author" title="Esther Fernandez-Gonzalez">E Fernandez-Gonzalez</span>, <span class="pub-author" title="Cosimo Ducani">C Ducani</span>, <span class="pub-author" title="Björn Högberg">B Högberg</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Restriction enzymes are the bread and butter of Molecular Biology. Nonetheless, how restriction enzymes recognize and cleave their target is not always clear. When developing a method for the enzymatic production of oligonucleotides, we noticed that type II endonucleases BtsCI and BseGI, which recognize the sequence GGATGNN^, perform incomplete digestions of DNA hairpins, with the top strand nick not always occurring correctly. We tested the cutting of synthetic hairpins containing all possible combinations of dinucleotides following the recognition site and our results show that all sequences containing one adenine following GGATG were digested more efficiently. We further show that the same sequence preference is also observable in double stranded DNA at higher Mg2+ concentrations and even in optimal conditions. Kinetic results show that BtsCI has a noteworthy difference in the first-rate constants between different sequences and between the two catalytic domains. An increase in Mg2+ resulted in a drastic decrease in the catalytic activity of the top (sense) strand that wasn't always accompanied by a nick in the bottom strand (antisense).</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/30118487" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1371/journal.pone.0202057" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/7a923a1fcefd4484a6d57e3390730cd7" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_84b19b61f97c4602a643b805bcca2c89" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Transcriptional and metabolic rewiring of colorectal cancer cells expressing the oncogenic KRASG13D mutation.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Theodosia Charitou">T Charitou</span>, <span class="pub-author" title="Sriganesh Srihari">S Srihari</span>, <span class="pub-author" title="Miriam A Lynn">MA Lynn</span>, <span class="pub-author" title="Mohamed-Ali Jarboui">MA Jarboui</span>, <span class="pub-author" title="Erik Fasterius">E Fasterius</span>, <span class="pub-author" title="Max Moldovan">M Moldovan</span>, <span class="pub-author" title="Senji Shirasawa">S Shirasawa</span>, <span class="pub-author" title="Toshiyuki Tsunoda">T Tsunoda</span>, <span class="pub-author" title="Marius Ueffing">M Ueffing</span>, <span class="pub-author" title="Jianling Xie">J Xie</span>, <span class="pub-author" title="Jin Xin">J Xin</span>, <span class="pub-author" title="Xuemin Wang">X Wang</span>, <span class="pub-author" title="Christopher G Proud">CG Proud</span>, <span class="pub-author" title="Karsten Boldt">K Boldt</span>, <span class="pub-author" title="Cristina Al-Khalili Szigyarto">C Al-Khalili Szigyarto</span>, <span class="pub-author" title="Walter Kolch">W Kolch</span>, <span class="pub-author" title="David J Lynn">DJ Lynn</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Activating mutations in KRAS frequently occur in colorectal cancer (CRC) patients, leading to resistance to EGFR-targeted therapies.

To better understand the cellular reprogramming which occurs in mutant KRAS cells, we have undertaken a systems-level analysis of four CRC cell lines which express either wild type (wt) KRAS or the oncogenic KRASG13D allele (mtKRAS).

RNAseq revealed that genes involved in ribosome biogenesis, mRNA translation and metabolism were significantly upregulated in mtKRAS cells. Consistent with the transcriptional data, protein synthesis and cell proliferation were significantly higher in the mtKRAS cells. Targeted metabolomics analysis also confirmed the metabolic reprogramming in mtKRAS cells. Interestingly, mtKRAS cells were highly transcriptionally responsive to EGFR activation by TGFα stimulation, which was associated with an unexpected downregulation of genes involved in a range of anabolic processes. While TGFα treatment strongly activated protein synthesis in wtKRAS cells, protein synthesis was not activated above basal levels in the TGFα-treated mtKRAS cells. This was likely due to the defective activation of the mTORC1 and other pathways by TGFα in mtKRAS cells, which was associated with impaired activation of PKB signalling and a transient induction of AMPK signalling.

We have found that mtKRAS cells are substantially rewired at the transcriptional, translational and metabolic levels and that this rewiring may reveal new vulnerabilities in oncogenic KRAS CRC cells that could be exploited in future.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31133691" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41416-019-0477-7" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/84b19b61f97c4602a643b805bcca2c89" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_17a8c61af04d40a19d6176e1cda30cef" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">PRR14L mutations are associated with chromosome 22 acquired uniparental disomy, age-related clonal hematopoiesis and myeloid neoplasia.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Andrew Chase">A Chase</span>, <span class="pub-author" title="Andrea Pellagatti">A Pellagatti</span>, <span class="pub-author" title="Shalini Singh">S Singh</span>, <span class="pub-author" title="Joannah Score">J Score</span>, <span class="pub-author" title="William J Tapper">WJ Tapper</span>, <span class="pub-author" title="Feng Lin">F Lin</span>, <span class="pub-author" title="Yvette Hoade">Y Hoade</span>, <span class="pub-author" title="Catherine Bryant">C Bryant</span>, <span class="pub-author" title="Nicola Trim">N Trim</span>, <span class="pub-author" title="Bon Ham Yip">BH Yip</span>, <span class="pub-author" title="Katerina Zoi">K Zoi</span>, <span class="pub-author" title="Chiara Rasi">C Rasi</span>, <span class="pub-author" title="Lars A Forsberg">LA Forsberg</span>, <span class="pub-author" title="Jan P Dumanski">JP Dumanski</span>, <span class="pub-author" title="Jacqueline Boultwood">J Boultwood</span>, <span class="pub-author" title="Nicholas C P Cross">NCP Cross</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Acquired uniparental disomy (aUPD, also known as copy-neutral loss of heterozygosity) is a common feature of cancer cells and characterized by extended tracts of somatically-acquired homozygosity without any concurrent loss or gain of genetic material. The presumed genetic targets of many regions of aUPD remain unknown. Here we describe the association of chromosome 22 aUPD with mutations that delete the C-terminus of PRR14L in patients with chronic myelomonocytic leukemia (CMML), related myeloid neoplasms and age-related clonal hematopoiesis (ARCH). Myeloid panel analysis identified a median of three additional mutated genes (range 1-6) in cases with a myeloid neoplasm (n = 8), but no additional mutations in cases with ARCH (n = 2) suggesting that mutated PRR14L alone may be sufficient to drive clonality. PRR14L has very limited homology to other proteins and its function is unknown. ShRNA knockdown of PRR14L in human CD34+ cells followed by in vitro growth and differentiation assays showed an increase in monocytes and decrease in neutrophils, decrease in neutrophils, consistent with a CMML-like phenotype. RNA-Seq and cellular localization studies suggest a role for PRR14L in cell division. PRR14L is thus a novel, biallelically mutated gene and potential founding abnormality in myeloid neoplasms.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/30573780" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41375-018-0340-5" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/17a8c61af04d40a19d6176e1cda30cef" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_5736722e01b74b29baf8fe43d4ce6819" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">RNA-seq analysis identifies key genes enhancing hoof strength to withstand barefoot racing in Standardbred trotters.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Doreen Schwochow">D Schwochow</span>, <span class="pub-author" title="Asmaa Alameddine">A Alameddine</span>, <span class="pub-author" title="Ellinor Spörndly-Nees">E Spörndly-Nees</span>, <span class="pub-author" title="Mathilde Montigny">M Montigny</span>, <span class="pub-author" title="Rakan Naboulsi">R Naboulsi</span>, <span class="pub-author" title="Anna Jansson">A Jansson</span>, <span class="pub-author" title="Adnan Niazi">A Niazi</span>, <span class="pub-author" title="Gabriella Lindgren">G Lindgren</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Racing without protective shoes is common in the Swedish harness racing industry, as it can enhance horses' performance on the track. Trainers typically decide whether a horse will race barefoot based on practical experience rather than objective measures. However, this practice can sometimes lead to excessive hoof wear, posing potential welfare concerns for racing horses. Gene expression differences may help reveal the underlying genetic mechanisms associated with different phenotypic traits. To explore an objective measure for assessing which horses are best suited for barefoot racing, we conducted a polyA-selected RNA-seq experiment on tissue from the growth zone at the coronary band of the hoof. This experiment compared tissues from Standardbred trotters capable of repeatedly racing barefoot without injury (n = 11) to those that could not (n = 7). By combining stringent phenotyping with racing records and trainer interviews, we aimed to elucidate the biological factors related to hoof strength in barefoot racing, focusing on differential abundant genes.

The RNA-seq analysis identified five significantly downregulated genes in horses capable of competing barefoot across consecutive races. These genes are associated with various biological processes relevant for hoof strength: ACCS, IRX2 and TRAPPAC6A contribute to enhancing the structural integrity of the hoof; MT2A regulates its metal homeostasis and SLC35F3 likely influences local vasoconstriction in the hoof. These gene findings suggest a coordinated genetic basis for structural reinforcement and physiological support of the hoof, which may be critical for sustaining performance under barefoot conditions.

Our findings suggest that the ability of Standardbred trotters to race barefoot in consecutive events is reflected in distinct gene expression patterns, underscoring a genetic basis for hoof strength. This supports further genome-wide scans aimed at identifying genetic markers for hoof durability in these horses. The focused design of our study- comparing horses that could consistently race barefoot with those that could not- enabled us to isolate a select group of genes involved in diverse aspects of hoof biology essential for quality and resilience of horse hooves. This insight could ultimately be applied to augment both the performance and wellbeing of equine athletes across disciplines.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40826322" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12864-025-11814-4" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/5736722e01b74b29baf8fe43d4ce6819" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_dfba8e4537274d90abb9a4c98eec18ae" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Meiofauna improve oxygenation and accelerate sulfide removal in the seasonally hypoxic seabed.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Stefano Bonaglia">S Bonaglia</span>, <span class="pub-author" title="Johanna Hedberg">J Hedberg</span>, <span class="pub-author" title="Ugo Marzocchi">U Marzocchi</span>, <span class="pub-author" title="Sven Iburg">S Iburg</span>, <span class="pub-author" title="Ronnie N Glud">RN Glud</span>, <span class="pub-author" title="Francisco J A Nascimento">FJA Nascimento</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Oxygen depleted areas are widespread in the marine realm. Unlike macrofauna, meiofauna are abundant in hypoxic sediments. We studied to what extent meiofauna affect oxygen availability, sulfide removal and microbial communities. Meiofauna were extracted alive and added to intact sediments simulating abundance gradients previously reported in the area. A total of 324 porewater microprofiles were recorded over a 3-week incubation period and microbial community structure and cable bacteria densities were determined at the end of the experiment. At high abundances meiofauna activity deepened oxygen penetration by 85%, 59%, and 62% after 5, 14, and 22 days, respectively, compared to control sediment with scarce meiofauna. After 6 days, meiofauna increased the volume of oxidized, sulfide-free sediment by 68% and reduced sulfide fluxes from 8.8 to 0.4 mmol m-2 d-1. After 15 days, the difference with the control attenuated due to the presence of a cable bacteria population, which facilitated sulfides oxidation in all treatments. 16S rRNA gene analysis revealed that meiofauna affected microbial community structure (beta diversity). Thus, meiofauna bioturbation plays an important role in deepening oxygen penetration, counteracting euxinia and in structuring microbial diversity of hypoxic sediments. Co-existence with cable bacteria demonstrates neutralism interaction between these two ecosystem engineers.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/32662428" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.marenvres.2020.104968" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/dfba8e4537274d90abb9a4c98eec18ae" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_d05046c85a1e4f1ca7defccbb1420945" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Direct effects of organic pollutants on the growth and gene expression of the Baltic Sea model bacterium Rheinheimera sp. BAL341.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Christofer M G Karlsson">CMG Karlsson</span>, <span class="pub-author" title="Elena Cerro-Gálvez">E Cerro-Gálvez</span>, <span class="pub-author" title="Daniel Lundin">D Lundin</span>, <span class="pub-author" title="Camilla Karlsson">C Karlsson</span>, <span class="pub-author" title="Maria Vila-Costa">M Vila-Costa</span>, <span class="pub-author" title="Jarone Pinhassi">J Pinhassi</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Organic pollutants (OPs) are critically toxic, bioaccumulative and globally widespread. Moreover, several OPs negatively influence aquatic wildlife. Although bacteria are major drivers of the ocean carbon cycle and the turnover of vital elements, there is limited knowledge of OP effects on heterotrophic bacterioplankton. We therefore investigated growth and gene expression responses of the Baltic Sea model bacterium Rheinheimera sp. BAL341 to environmentally relevant concentrations of distinct classes of OPs in 2-h incubation experiments. During exponential growth, exposure to a mix of polycyclic aromatic hydrocarbons, alkanes and organophosphate esters (denoted MIX) resulted in a significant decrease (between 9% and 18%) in bacterial abundance and production compared with controls. In contrast, combined exposure to perfluorooctanesulfonic acids and perfluorooctanoic acids (denoted PFAS) had no significant effect on growth. Nevertheless, MIX and PFAS exposures both induced significant shifts in gene expression profiles compared with controls in exponential growth. This involved several functional metabolism categories (e.g. stress response and fatty acids metabolism), some of which were pollutant-specific (e.g. phosphate acquisition and alkane-1 monooxygenase genes). In stationary phase, only two genes in the MIX treatment were significantly differentially expressed. The substantial direct influence of OPs on metabolism during bacterial growth suggests that widespread OPs could severely alter biogeochemical processes governed by bacterioplankton.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31270938" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/1751-7915.13441" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/d05046c85a1e4f1ca7defccbb1420945" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_e389be71822d45d6ac04d848bfb48df6" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">The power of genetic diversity in genome-wide association studies of lipids.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Sarah E Graham">SE Graham</span>, <span class="pub-author" title="Shoa L Clarke">SL Clarke</span>, <span class="pub-author" title="Kuan-Han H Wu">KH Wu</span>, <span class="pub-author" title="Stavroula Kanoni">S Kanoni</span>, <span class="pub-author" title="Greg J M Zajac">GJM Zajac</span>, <span class="pub-author" title="Shweta Ramdas">S Ramdas</span>, <span class="pub-author" title="Ida Surakka">I Surakka</span>, <span class="pub-author" title="Ioanna Ntalla">I Ntalla</span>, <span class="pub-author" title="Sailaja Vedantam">S Vedantam</span>, <span class="pub-author" title="Thomas W Winkler">TW Winkler</span>, <span class="pub-author" title="Adam E Locke">AE Locke</span>, <span class="pub-author" title="Eirini Marouli">E Marouli</span>, <span class="pub-author" title="Mi Yeong Hwang">MY Hwang</span>, <span class="pub-author" title="Sohee Han">S Han</span>, <span class="pub-author" title="Akira Narita">A Narita</span>, <span class="pub-author" title="Ananyo Choudhury">A Choudhury</span>, <span class="pub-author" title="Amy R Bentley">AR Bentley</span>, <span class="pub-author" title="Kenneth Ekoru">K Ekoru</span>, <span class="pub-author" title="Anurag Verma">A Verma</span>, <span class="pub-author" title="Bhavi Trivedi">B Trivedi</span>, <span class="pub-author" title="Hilary C Martin">HC Martin</span>, <span class="pub-author" title="Karen A Hunt">KA Hunt</span>, <span class="pub-author" title="Qin Hui">Q Hui</span>, <span class="pub-author" title="Derek Klarin">D Klarin</span>, <span class="pub-author" title="Xiang Zhu">X Zhu</span>, <span class="pub-author" title="Gudmar Thorleifsson">G Thorleifsson</span>, <span class="pub-author" title="Anna Helgadottir">A Helgadottir</span>, <span class="pub-author" title="Daniel F Gudbjartsson">DF Gudbjartsson</span>, <span class="pub-author" title="Hilma Holm">H Holm</span>, <span class="pub-author" title="Isleifur Olafsson">I Olafsson</span>, <span class="pub-author" title="Masato Akiyama">M Akiyama</span>, <span class="pub-author" title="Saori Sakaue">S Sakaue</span>, <span class="pub-author" title="Chikashi Terao">C Terao</span>, <span class="pub-author" title="Masahiro Kanai">M Kanai</span>, <span class="pub-author" title="Wei Zhou">W Zhou</span>, <span class="pub-author" title="Ben M Brumpton">BM Brumpton</span>, <span class="pub-author" title="Humaira Rasheed">H Rasheed</span>, <span class="pub-author" title="Sanni E Ruotsalainen">SE Ruotsalainen</span>, <span class="pub-author" title="Aki S Havulinna">AS Havulinna</span>, <span class="pub-author" title="Yogasudha Veturi">Y Veturi</span>, <span class="pub-author" title="QiPing Feng">Q Feng</span>, <span class="pub-author" title="Elisabeth A Rosenthal">EA Rosenthal</span>, <span class="pub-author" title="Todd Lingren">T Lingren</span>, <span class="pub-author" title="Jennifer Allen Pacheco">JA Pacheco</span>, <span class="pub-author" title="Sarah A Pendergrass">SA Pendergrass</span>, <span class="pub-author" title="Jeffrey Haessler">J Haessler</span>, <span class="pub-author" title="Franco Giulianini">F Giulianini</span>, <span class="pub-author" title="Yuki Bradford">Y Bradford</span>, <span class="pub-author" title="Jason E Miller">JE Miller</span>, <span class="pub-author" title="Archie Campbell">A Campbell</span>, <span class="pub-author" title="Kuang Lin">K Lin</span>, <span class="pub-author" title="Iona Y Millwood">IY Millwood</span>, <span class="pub-author" title="George Hindy">G Hindy</span>, <span class="pub-author" title="Asif Rasheed">A Rasheed</span>, <span class="pub-author" title="Jessica D Faul">JD Faul</span>, <span class="pub-author" title="Wei Zhao">W Zhao</span>, <span class="pub-author" title="David R Weir">DR Weir</span>, <span class="pub-author" title="Constance Turman">C Turman</span>, <span class="pub-author" title="Hongyan Huang">H Huang</span>, <span class="pub-author" title="Mariaelisa Graff">M Graff</span>, <span class="pub-author" title="Anubha Mahajan">A Mahajan</span>, <span class="pub-author" title="Michael R Brown">MR Brown</span>, <span class="pub-author" title="Weihua Zhang">W Zhang</span>, <span class="pub-author" title="Ketian Yu">K Yu</span>, <span class="pub-author" title="Ellen M Schmidt">EM Schmidt</span>, <span class="pub-author" title="Anita Pandit">A Pandit</span>, <span class="pub-author" title="Stefan Gustafsson">S Gustafsson</span>, <span class="pub-author" title="Xianyong Yin">X Yin</span>, <span class="pub-author" title="Jian'an Luan">J Luan</span>, <span class="pub-author" title="Jing-Hua Zhao">JH Zhao</span>, <span class="pub-author" title="Fumihiko Matsuda">F Matsuda</span>, <span class="pub-author" title="Hye-Mi Jang">HM Jang</span>, <span class="pub-author" title="Kyungheon Yoon">K Yoon</span>, <span class="pub-author" title="Carolina Medina-Gomez">C Medina-Gomez</span>, <span class="pub-author" title="Achilleas Pitsillides">A Pitsillides</span>, <span class="pub-author" title="Jouke Jan Hottenga">JJ Hottenga</span>, <span class="pub-author" title="Gonneke Willemsen">G Willemsen</span>, <span class="pub-author" title="Andrew R Wood">AR Wood</span>, <span class="pub-author" title="Yingji Ji">Y Ji</span>, <span class="pub-author" title="Zishan Gao">Z Gao</span>, <span class="pub-author" title="Simon Haworth">S Haworth</span>, <span class="pub-author" title="Ruth E Mitchell">RE Mitchell</span>, <span class="pub-author" title="Jin Fang Chai">JF Chai</span>, <span class="pub-author" title="Mette Aadahl">M Aadahl</span>, <span class="pub-author" title="Jie Yao">J Yao</span>, <span class="pub-author" title="Ani Manichaikul">A Manichaikul</span>, <span class="pub-author" title="Helen R Warren">HR Warren</span>, <span class="pub-author" title="Julia Ramirez">J Ramirez</span>, <span class="pub-author" title="Jette Bork-Jensen">J Bork-Jensen</span>, <span class="pub-author" title="Line L Kårhus">LL Kårhus</span>, <span class="pub-author" title="Anuj Goel">A Goel</span>, <span class="pub-author" title="Maria Sabater-Lleal">M Sabater-Lleal</span>, <span class="pub-author" title="Raymond Noordam">R Noordam</span>, <span class="pub-author" title="Carlo Sidore">C Sidore</span>, <span class="pub-author" title="Edoardo Fiorillo">E Fiorillo</span>, <span class="pub-author" title="Aaron F McDaid">AF McDaid</span>, <span class="pub-author" title="Pedro Marques-Vidal">P Marques-Vidal</span>, <span class="pub-author" title="Matthias Wielscher">M Wielscher</span>, <span class="pub-author" title="Stella Trompet">S Trompet</span>, <span class="pub-author" title="Naveed Sattar">N Sattar</span>, <span class="pub-author" title="Line T Møllehave">LT Møllehave</span>, <span class="pub-author" title="Betina H Thuesen">BH Thuesen</span>, <span class="pub-author" title="Matthias Munz">M Munz</span>, <span class="pub-author" title="Lingyao Zeng">L Zeng</span>, <span class="pub-author" title="Jianfeng Huang">J Huang</span>, <span class="pub-author" title="Bin Yang">B Yang</span>, <span class="pub-author" title="Alaitz Poveda">A Poveda</span>, <span class="pub-author" title="Azra Kurbasic">A Kurbasic</span>, <span class="pub-author" title="Claudia Lamina">C Lamina</span>, <span class="pub-author" title="Lukas Forer">L Forer</span>, <span class="pub-author" title="Markus Scholz">M Scholz</span>, <span class="pub-author" title="Tessel E Galesloot">TE Galesloot</span>, <span class="pub-author" title="Jonathan P Bradfield">JP Bradfield</span>, <span class="pub-author" title="E Warwick Daw">EW Daw</span>, <span class="pub-author" title="Joseph M Zmuda">JM Zmuda</span>, <span class="pub-author" title="Jonathan S Mitchell">JS Mitchell</span>, <span class="pub-author" title="Christian Fuchsberger">C Fuchsberger</span>, <span class="pub-author" title="Henry Christensen">H Christensen</span>, <span class="pub-author" title="Jennifer A Brody">JA Brody</span>, <span class="pub-author" title="Mary F Feitosa">MF Feitosa</span>, <span class="pub-author" title="Mary K Wojczynski">MK Wojczynski</span>, <span class="pub-author" title="Michael Preuss">M Preuss</span>, <span class="pub-author" title="Massimo Mangino">M Mangino</span>, <span class="pub-author" title="Paraskevi Christofidou">P Christofidou</span>, <span class="pub-author" title="Niek Verweij">N Verweij</span>, <span class="pub-author" title="Jan W Benjamins">JW Benjamins</span>, <span class="pub-author" title="Jorgen Engmann">J Engmann</span>, <span class="pub-author" title="Rachel L Kember">RL Kember</span>, <span class="pub-author" title="Roderick C Slieker">RC Slieker</span>, <span class="pub-author" title="Ken Sin Lo">KS Lo</span>, <span class="pub-author" title="Nuno R Zilhao">NR Zilhao</span>, <span class="pub-author" title="Phuong Le">P Le</span>, <span class="pub-author" title="Marcus E Kleber">ME Kleber</span>, <span class="pub-author" title="Graciela E Delgado">GE Delgado</span>, <span class="pub-author" title="Shaofeng Huo">S Huo</span>, <span class="pub-author" title="Daisuke D Ikeda">DD Ikeda</span>, <span class="pub-author" title="Hiroyuki Iha">H Iha</span>, <span class="pub-author" title="Jian Yang">J Yang</span>, <span class="pub-author" title="Jun Liu">J Liu</span>, <span class="pub-author" title="Hampton L Leonard">HL Leonard</span>, <span class="pub-author" title="Jonathan Marten">J Marten</span>, <span class="pub-author" title="Börge Schmidt">B Schmidt</span>, <span class="pub-author" title="Marina Arendt">M Arendt</span>, <span class="pub-author" title="Laura J Smyth">LJ Smyth</span>, <span class="pub-author" title="Marisa Cañadas-Garre">M Cañadas-Garre</span>, <span class="pub-author" title="Chaolong Wang">C Wang</span>, <span class="pub-author" title="Masahiro Nakatochi">M Nakatochi</span>, <span class="pub-author" title="Andrew Wong">A Wong</span>, <span class="pub-author" title="Nina Hutri-Kähönen">N Hutri-Kähönen</span>, <span class="pub-author" title="Xueling Sim">X Sim</span>, <span class="pub-author" title="Rui Xia">R Xia</span>, <span class="pub-author" title="Alicia Huerta-Chagoya">A Huerta-Chagoya</span>, <span class="pub-author" title="Juan Carlos Fernandez-Lopez">JC Fernandez-Lopez</span>, <span class="pub-author" title="Valeriya Lyssenko">V Lyssenko</span>, <span class="pub-author" title="Meraj Ahmed">M Ahmed</span>, <span class="pub-author" title="Anne U Jackson">AU Jackson</span>, <span class="pub-author" title="Noha A Yousri">NA Yousri</span>, <span class="pub-author" title="Marguerite R Irvin">MR Irvin</span>, <span class="pub-author" title="Christopher Oldmeadow">C Oldmeadow</span>, <span class="pub-author" title="Han-Na Kim">HN Kim</span>, <span class="pub-author" title="Seungho Ryu">S Ryu</span>, <span class="pub-author" title="Paul R H J Timmers">PRHJ Timmers</span>, <span class="pub-author" title="Liubov Arbeeva">L Arbeeva</span>, <span class="pub-author" title="Rajkumar Dorajoo">R Dorajoo</span>, <span class="pub-author" title="Leslie A Lange">LA Lange</span>, <span class="pub-author" title="Xiaoran Chai">X Chai</span>, <span class="pub-author" title="Gauri Prasad">G Prasad</span>, <span class="pub-author" title="Laura Lorés-Motta">L Lorés-Motta</span>, <span class="pub-author" title="Marc Pauper">M Pauper</span>, <span class="pub-author" title="Jirong Long">J Long</span>, <span class="pub-author" title="Xiaohui Li">X Li</span>, <span class="pub-author" title="Elizabeth Theusch">E Theusch</span>, <span class="pub-author" title="Fumihiko Takeuchi">F Takeuchi</span>, <span class="pub-author" title="Cassandra N Spracklen">CN Spracklen</span>, <span class="pub-author" title="Anu Loukola">A Loukola</span>, <span class="pub-author" title="Sailalitha Bollepalli">S Bollepalli</span>, <span class="pub-author" title="Sophie C Warner">SC Warner</span>, <span class="pub-author" title="Ya Xing Wang">YX Wang</span>, <span class="pub-author" title="Wen B Wei">WB Wei</span>, <span class="pub-author" title="Teresa Nutile">T Nutile</span>, <span class="pub-author" title="Daniela Ruggiero">D Ruggiero</span>, <span class="pub-author" title="Yun Ju Sung">YJ Sung</span>, <span class="pub-author" title="Yi-Jen Hung">YJ Hung</span>, <span class="pub-author" title="Shufeng Chen">S Chen</span>, <span class="pub-author" title="Fangchao Liu">F Liu</span>, <span class="pub-author" title="Jingyun Yang">J Yang</span>, <span class="pub-author" title="Katherine A Kentistou">KA Kentistou</span>, <span class="pub-author" title="Mathias Gorski">M Gorski</span>, <span class="pub-author" title="Marco Brumat">M Brumat</span>, <span class="pub-author" title="Karina Meidtner">K Meidtner</span>, <span class="pub-author" title="Lawrence F Bielak">LF Bielak</span>, <span class="pub-author" title="Jennifer A Smith">JA Smith</span>, <span class="pub-author" title="Prashantha Hebbar">P Hebbar</span>, <span class="pub-author" title="Aliki-Eleni Farmaki">AE Farmaki</span>, <span class="pub-author" title="Edith Hofer">E Hofer</span>, <span class="pub-author" title="Maoxuan Lin">M Lin</span>, <span class="pub-author" title="Chao Xue">C Xue</span>, <span class="pub-author" title="Jifeng Zhang">J Zhang</span>, <span class="pub-author" title="Maria Pina Concas">MP Concas</span>, <span class="pub-author" title="Simona Vaccargiu">S Vaccargiu</span>, <span class="pub-author" title="Peter J van der Most">PJ van der Most</span>, <span class="pub-author" title="Niina Pitkänen">N Pitkänen</span>, <span class="pub-author" title="Brian E Cade">BE Cade</span>, <span class="pub-author" title="Jiwon Lee">J Lee</span>, <span class="pub-author" title="Sander W van der Laan">SW van der Laan</span>, <span class="pub-author" title="Kumaraswamy Naidu Chitrala">KN Chitrala</span>, <span class="pub-author" title="Stefan Weiss">S Weiss</span>, <span class="pub-author" title="Martina E Zimmermann">ME Zimmermann</span>, <span class="pub-author" title="Jong Young Lee">JY Lee</span>, <span class="pub-author" title="Hyeok Sun Choi">HS Choi</span>, <span class="pub-author" title="Maria Nethander">M Nethander</span>, <span class="pub-author" title="Sandra Freitag-Wolf">S Freitag-Wolf</span>, <span class="pub-author" title="Lorraine Southam">L Southam</span>, <span class="pub-author" title="Nigel W Rayner">NW Rayner</span>, <span class="pub-author" title="Carol A Wang">CA Wang</span>, <span class="pub-author" title="Shih-Yi Lin">SY Lin</span>, <span class="pub-author" title="Jun-Sing Wang">JS Wang</span>, <span class="pub-author" title="Christian Couture">C Couture</span>, <span class="pub-author" title="Leo-Pekka Lyytikäinen">LP Lyytikäinen</span>, <span class="pub-author" title="Kjell Nikus">K Nikus</span>, <span class="pub-author" title="Gabriel Cuellar-Partida">G Cuellar-Partida</span>, <span class="pub-author" title="Henrik Vestergaard">H Vestergaard</span>, <span class="pub-author" title="Bertha Hildalgo">B Hildalgo</span>, <span class="pub-author" title="Olga Giannakopoulou">O Giannakopoulou</span>, <span class="pub-author" title="Qiuyin Cai">Q Cai</span>, <span class="pub-author" title="Morgan O Obura">MO Obura</span>, <span class="pub-author" title="Jessica van Setten">J van Setten</span>, <span class="pub-author" title="Xiaoyin Li">X Li</span>, <span class="pub-author" title="Karen Schwander">K Schwander</span>, <span class="pub-author" title="Natalie Terzikhan">N Terzikhan</span>, <span class="pub-author" title="Jae Hun Shin">JH Shin</span>, <span class="pub-author" title="Rebecca D Jackson">RD Jackson</span>, <span class="pub-author" title="Alexander P Reiner">AP Reiner</span>, <span class="pub-author" title="Lisa Warsinger Martin">LW Martin</span>, <span class="pub-author" title="Zhengming Chen">Z Chen</span>, <span class="pub-author" title="Liming Li">L Li</span>, <span class="pub-author" title="Heather M Highland">HM Highland</span>, <span class="pub-author" title="Kristin L Young">KL Young</span>, <span class="pub-author" title="Takahisa Kawaguchi">T Kawaguchi</span>, <span class="pub-author" title="Joachim Thiery">J Thiery</span>, <span class="pub-author" title="Joshua C Bis">JC Bis</span>, <span class="pub-author" title="Girish N Nadkarni">GN Nadkarni</span>, <span class="pub-author" title="Lenore J Launer">LJ Launer</span>, <span class="pub-author" title="Huaixing Li">H Li</span>, <span class="pub-author" title="Mike A Nalls">MA Nalls</span>, <span class="pub-author" title="Olli T Raitakari">OT Raitakari</span>, <span class="pub-author" title="Sahoko Ichihara">S Ichihara</span>, <span class="pub-author" title="Sarah H Wild">SH Wild</span>, <span class="pub-author" title="Christopher P Nelson">CP Nelson</span>, <span class="pub-author" title="Harry Campbell">H Campbell</span>, <span class="pub-author" title="Susanne Jäger">S Jäger</span>, <span class="pub-author" title="Toru Nabika">T Nabika</span>, <span class="pub-author" title="Fahd Al-Mulla">F Al-Mulla</span>, <span class="pub-author" title="Harri Niinikoski">H Niinikoski</span>, <span class="pub-author" title="Peter S Braund">PS Braund</span>, <span class="pub-author" title="Ivana Kolcic">I Kolcic</span>, <span class="pub-author" title="Peter Kovacs">P Kovacs</span>, <span class="pub-author" title="Tota Giardoglou">T Giardoglou</span>, <span class="pub-author" title="Tomohiro Katsuya">T Katsuya</span>, <span class="pub-author" title="Konain Fatima Bhatti">KF Bhatti</span>, <span class="pub-author" title="Dominique de Kleijn">D de Kleijn</span>, <span class="pub-author" title="Gert J de Borst">GJ de Borst</span>, <span class="pub-author" title="Eung Kweon Kim">EK Kim</span>, <span class="pub-author" title="Hieab H H Adams">HHH Adams</span>, <span class="pub-author" title="M Arfan Ikram">MA Ikram</span>, <span class="pub-author" title="Xiaofeng Zhu">X Zhu</span>, <span class="pub-author" title="Folkert W Asselbergs">FW Asselbergs</span>, <span class="pub-author" title="Adriaan O Kraaijeveld">AO Kraaijeveld</span>, <span class="pub-author" title="Joline W J Beulens">JWJ Beulens</span>, <span class="pub-author" title="Xiao-Ou Shu">XO Shu</span>, <span class="pub-author" title="Loukianos S Rallidis">LS Rallidis</span>, <span class="pub-author" title="Oluf Pedersen">O Pedersen</span>, <span class="pub-author" title="Torben Hansen">T Hansen</span>, <span class="pub-author" title="Paul Mitchell">P Mitchell</span>, <span class="pub-author" title="Alex W Hewitt">AW Hewitt</span>, <span class="pub-author" title="Mika Kähönen">M Kähönen</span>, <span class="pub-author" title="Louis Pérusse">L Pérusse</span>, <span class="pub-author" title="Claude Bouchard">C Bouchard</span>, <span class="pub-author" title="Anke Tönjes">A Tönjes</span>, <span class="pub-author" title="Yii-Der Ida Chen">YI Chen</span>, <span class="pub-author" title="Craig E Pennell">CE Pennell</span>, <span class="pub-author" title="Trevor A Mori">TA Mori</span>, <span class="pub-author" title="Wolfgang Lieb">W Lieb</span>, <span class="pub-author" title="Andre Franke">A Franke</span>, <span class="pub-author" title="Claes Ohlsson">C Ohlsson</span>, <span class="pub-author" title="Dan Mellström">D Mellström</span>, <span class="pub-author" title="Yoon Shin Cho">YS Cho</span>, <span class="pub-author" title="Hyejin Lee">H Lee</span>, <span class="pub-author" title="Jian-Min Yuan">JM Yuan</span>, <span class="pub-author" title="Woon-Puay Koh">WP Koh</span>, <span class="pub-author" title="Sang Youl Rhee">SY Rhee</span>, <span class="pub-author" title="Jeong-Taek Woo">JT Woo</span>, <span class="pub-author" title="Iris M Heid">IM Heid</span>, <span class="pub-author" title="Klaus J Stark">KJ Stark</span>, <span class="pub-author" title="Henry Völzke">H Völzke</span>, <span class="pub-author" title="Georg Homuth">G Homuth</span>, <span class="pub-author" title="Michele K Evans">MK Evans</span>, <span class="pub-author" title="Alan B Zonderman">AB Zonderman</span>, <span class="pub-author" title="Ozren Polasek">O Polasek</span>, <span class="pub-author" title="Gerard Pasterkamp">G Pasterkamp</span>, <span class="pub-author" title="Imo E Hoefer">IE Hoefer</span>, <span class="pub-author" title="Susan Redline">S Redline</span>, <span class="pub-author" title="Katja Pahkala">K Pahkala</span>, <span class="pub-author" title="Albertine J Oldehinkel">AJ Oldehinkel</span>, <span class="pub-author" title="Harold Snieder">H Snieder</span>, <span class="pub-author" title="Ginevra Biino">G Biino</span>, <span class="pub-author" title="Reinhold Schmidt">R Schmidt</span>, <span class="pub-author" title="Helena Schmidt">H Schmidt</span>, <span class="pub-author" title="Y Eugene Chen">YE Chen</span>, <span class="pub-author" title="Stefania Bandinelli">S Bandinelli</span>, <span class="pub-author" title="George Dedoussis">G Dedoussis</span>, <span class="pub-author" title="Thangavel Alphonse Thanaraj">TA Thanaraj</span>, <span class="pub-author" title="Sharon L R Kardia">SLR Kardia</span>, <span class="pub-author" title="Norihiro Kato">N Kato</span>, <span class="pub-author" title="Matthias B Schulze">MB Schulze</span>, <span class="pub-author" title="Giorgia Girotto">G Girotto</span>, <span class="pub-author" title="Bettina Jung">B Jung</span>, <span class="pub-author" title="Carsten A Böger">CA Böger</span>, <span class="pub-author" title="Peter K Joshi">PK Joshi</span>, <span class="pub-author" title="David A Bennett">DA Bennett</span>, <span class="pub-author" title="Philip L De Jager">PL De Jager</span>, <span class="pub-author" title="Xiangfeng Lu">X Lu</span>, <span class="pub-author" title="Vasiliki Mamakou">V Mamakou</span>, <span class="pub-author" title="Morris Brown">M Brown</span>, <span class="pub-author" title="Mark J Caulfield">MJ Caulfield</span>, <span class="pub-author" title="Patricia B Munroe">PB Munroe</span>, <span class="pub-author" title="Xiuqing Guo">X Guo</span>, <span class="pub-author" title="Marina Ciullo">M Ciullo</span>, <span class="pub-author" title="Jost B Jonas">JB Jonas</span>, <span class="pub-author" title="Nilesh J Samani">NJ Samani</span>, <span class="pub-author" title="Jaakko Kaprio">J Kaprio</span>, <span class="pub-author" title="Päivi Pajukanta">P Pajukanta</span>, <span class="pub-author" title="Linda S Adair">LS Adair</span>, <span class="pub-author" title="Sonny Augustin Bechayda">SA Bechayda</span>, <span class="pub-author" title="H Janaka de Silva">HJ de Silva</span>, <span class="pub-author" title="Ananda R Wickremasinghe">AR Wickremasinghe</span>, <span class="pub-author" title="Ronald M Krauss">RM Krauss</span>, <span class="pub-author" title="Jer-Yuarn Wu">JY Wu</span>, <span class="pub-author" title="Wei Zheng">W Zheng</span>, <span class="pub-author" title="Anneke I den Hollander">AI den Hollander</span>, <span class="pub-author" title="Dwaipayan Bharadwaj">D Bharadwaj</span>, <span class="pub-author" title="Adolfo Correa">A Correa</span>, <span class="pub-author" title="James G Wilson">JG Wilson</span>, <span class="pub-author" title="Lars Lind">L Lind</span>, <span class="pub-author" title="Chew-Kiat Heng">CK Heng</span>, <span class="pub-author" title="Amanda E Nelson">AE Nelson</span>, <span class="pub-author" title="Yvonne M Golightly">YM Golightly</span>, <span class="pub-author" title="James F Wilson">JF Wilson</span>, <span class="pub-author" title="Brenda Penninx">B Penninx</span>, <span class="pub-author" title="Hyung-Lae Kim">HL Kim</span>, <span class="pub-author" title="John Attia">J Attia</span>, <span class="pub-author" title="Rodney J Scott">RJ Scott</span>, <span class="pub-author" title="D C Rao">DC Rao</span>, <span class="pub-author" title="Donna K Arnett">DK Arnett</span>, <span class="pub-author" title="Steven C Hunt">SC Hunt</span>, <span class="pub-author" title="Mark Walker">M Walker</span>, <span class="pub-author" title="Heikki A Koistinen">HA Koistinen</span>, <span class="pub-author" title="Giriraj R Chandak">GR Chandak</span>, <span class="pub-author" title="Chittaranjan S Yajnik">CS Yajnik</span>, <span class="pub-author" title="Josep M Mercader">JM Mercader</span>, <span class="pub-author" title="Teresa Tusié-Luna">T Tusié-Luna</span>, <span class="pub-author" title="Carlos A Aguilar-Salinas">CA Aguilar-Salinas</span>, <span class="pub-author" title="Clicerio Gonzalez Villalpando">CG Villalpando</span>, <span class="pub-author" title="Lorena Orozco">L Orozco</span>, <span class="pub-author" title="Myriam Fornage">M Fornage</span>, <span class="pub-author" title="E Shyong Tai">ES Tai</span>, <span class="pub-author" title="Rob M van Dam">RM van Dam</span>, <span class="pub-author" title="Terho Lehtimäki">T Lehtimäki</span>, <span class="pub-author" title="Nish Chaturvedi">N Chaturvedi</span>, <span class="pub-author" title="Mitsuhiro Yokota">M Yokota</span>, <span class="pub-author" title="Jianjun Liu">J Liu</span>, <span class="pub-author" title="Dermot F Reilly">DF Reilly</span>, <span class="pub-author" title="Amy Jayne McKnight">AJ McKnight</span>, <span class="pub-author" title="Frank Kee">F Kee</span>, <span class="pub-author" title="Karl-Heinz Jöckel">KH Jöckel</span>, <span class="pub-author" title="Mark I McCarthy">MI McCarthy</span>, <span class="pub-author" title="Colin N A Palmer">CNA Palmer</span>, <span class="pub-author" title="Veronique Vitart">V Vitart</span>, <span class="pub-author" title="Caroline Hayward">C Hayward</span>, <span class="pub-author" title="Eleanor Simonsick">E Simonsick</span>, <span class="pub-author" title="Cornelia M van Duijn">CM van Duijn</span>, <span class="pub-author" title="Fan Lu">F Lu</span>, <span class="pub-author" title="Jia Qu">J Qu</span>, <span class="pub-author" title="Haretsugu Hishigaki">H Hishigaki</span>, <span class="pub-author" title="Xu Lin">X Lin</span>, <span class="pub-author" title="Winfried März">W März</span>, <span class="pub-author" title="Esteban J Parra">EJ Parra</span>, <span class="pub-author" title="Miguel Cruz">M Cruz</span>, <span class="pub-author" title="Vilmundur Gudnason">V Gudnason</span>, <span class="pub-author" title="Jean-Claude Tardif">JC Tardif</span>, <span class="pub-author" title="Guillaume Lettre">G Lettre</span>, <span class="pub-author" title="Leen M 't Hart">LM 't Hart</span>, <span class="pub-author" title="Petra J M Elders">PJM Elders</span>, <span class="pub-author" title="Scott M Damrauer">SM Damrauer</span>, <span class="pub-author" title="Meena Kumari">M Kumari</span>, <span class="pub-author" title="Mika Kivimaki">M Kivimaki</span>, <span class="pub-author" title="Pim van der Harst">P van der Harst</span>, <span class="pub-author" title="Tim D Spector">TD Spector</span>, <span class="pub-author" title="Ruth J F Loos">RJF Loos</span>, <span class="pub-author" title="Michael A Province">MA Province</span>, <span class="pub-author" title="Bruce M Psaty">BM Psaty</span>, <span class="pub-author" title="Ivan Brandslund">I Brandslund</span>, <span class="pub-author" title="Peter P Pramstaller">PP Pramstaller</span>, <span class="pub-author" title="Kaare Christensen">K Christensen</span>, <span class="pub-author" title="Samuli Ripatti">S Ripatti</span>, <span class="pub-author" title="Elisabeth Widén">E Widén</span>, <span class="pub-author" title="Hakon Hakonarson">H Hakonarson</span>, <span class="pub-author" title="Struan F A Grant">SFA Grant</span>, <span class="pub-author" title="Lambertus A L M Kiemeney">LALM Kiemeney</span>, <span class="pub-author" title="Jacqueline de Graaf">J de Graaf</span>, <span class="pub-author" title="Markus Loeffler">M Loeffler</span>, <span class="pub-author" title="Florian Kronenberg">F Kronenberg</span>, <span class="pub-author" title="Dongfeng Gu">D Gu</span>, <span class="pub-author" title="Jeanette Erdmann">J Erdmann</span>, <span class="pub-author" title="Heribert Schunkert">H Schunkert</span>, <span class="pub-author" title="Paul W Franks">PW Franks</span>, <span class="pub-author" title="Allan Linneberg">A Linneberg</span>, <span class="pub-author" title="J Wouter Jukema">JW Jukema</span>, <span class="pub-author" title="Amit V Khera">AV Khera</span>, <span class="pub-author" title="Minna Männikkö">M Männikkö</span>, <span class="pub-author" title="Marjo-Riitta Jarvelin">MR Jarvelin</span>, <span class="pub-author" title="Zoltan Kutalik">Z Kutalik</span>, <span class="pub-author" title="Francesco Cucca">F Cucca</span>, <span class="pub-author" title="Dennis O Mook-Kanamori">DO Mook-Kanamori</span>, <span class="pub-author" title="Ko Willems van Dijk">KW van Dijk</span>, <span class="pub-author" title="Hugh Watkins">H Watkins</span>, <span class="pub-author" title="David P Strachan">DP Strachan</span>, <span class="pub-author" title="Niels Grarup">N Grarup</span>, <span class="pub-author" title="Peter Sever">P Sever</span>, <span class="pub-author" title="Neil Poulter">N Poulter</span>, <span class="pub-author" title="Jerome I Rotter">JI Rotter</span>, <span class="pub-author" title="Thomas M Dantoft">TM Dantoft</span>, <span class="pub-author" title="Fredrik Karpe">F Karpe</span>, <span class="pub-author" title="Matt J Neville">MJ Neville</span>, <span class="pub-author" title="Nicholas J Timpson">NJ Timpson</span>, <span class="pub-author" title="Ching-Yu Cheng">CY Cheng</span>, <span class="pub-author" title="Tien-Yin Wong">TY Wong</span>, <span class="pub-author" title="Chiea Chuen Khor">CC Khor</span>, <span class="pub-author" title="Charumathi Sabanayagam">C Sabanayagam</span>, <span class="pub-author" title="Annette Peters">A Peters</span>, <span class="pub-author" title="Christian Gieger">C Gieger</span>, <span class="pub-author" title="Andrew T Hattersley">AT Hattersley</span>, <span class="pub-author" title="Nancy L Pedersen">NL Pedersen</span>, <span class="pub-author" title="Patrik K E Magnusson">PKE Magnusson</span>, <span class="pub-author" title="Dorret I Boomsma">DI Boomsma</span>, <span class="pub-author" title="Eco J C de Geus">EJC de Geus</span>, <span class="pub-author" title="L Adrienne Cupples">LA Cupples</span>, <span class="pub-author" title="Joyce B J van Meurs">JBJ van Meurs</span>, <span class="pub-author" title="Mohsen Ghanbari">M Ghanbari</span>, <span class="pub-author" title="Penny Gordon-Larsen">P Gordon-Larsen</span>, <span class="pub-author" title="Wei Huang">W Huang</span>, <span class="pub-author" title="Young Jin Kim">YJ Kim</span>, <span class="pub-author" title="Yasuharu Tabara">Y Tabara</span>, <span class="pub-author" title="Nicholas J Wareham">NJ Wareham</span>, <span class="pub-author" title="Claudia Langenberg">C Langenberg</span>, <span class="pub-author" title="Eleftheria Zeggini">E Zeggini</span>, <span class="pub-author" title="Johanna Kuusisto">J Kuusisto</span>, <span class="pub-author" title="Markku Laakso">M Laakso</span>, <span class="pub-author" title="Erik Ingelsson">E Ingelsson</span>, <span class="pub-author" title="Goncalo Abecasis">G Abecasis</span>, <span class="pub-author" title="John C Chambers">JC Chambers</span>, <span class="pub-author" title="Jaspal S Kooner">JS Kooner</span>, <span class="pub-author" title="Paul S de Vries">PS de Vries</span>, <span class="pub-author" title="Alanna C Morrison">AC Morrison</span>, <span class="pub-author" title="Kari E North">KE North</span>, <span class="pub-author" title="Martha Daviglus">M Daviglus</span>, <span class="pub-author" title="Peter Kraft">P Kraft</span>, <span class="pub-author" title="Nicholas G Martin">NG Martin</span>, <span class="pub-author" title="John B Whitfield">JB Whitfield</span>, <span class="pub-author" title="Shahid Abbas">S Abbas</span>, <span class="pub-author" title="Danish Saleheen">D Saleheen</span>, <span class="pub-author" title="Robin G Walters">RG Walters</span>, <span class="pub-author" title="Michael V Holmes">MV Holmes</span>, <span class="pub-author" title="Corri Black">C Black</span>, <span class="pub-author" title="Blair H Smith">BH Smith</span>, <span class="pub-author" title="Anne E Justice">AE Justice</span>, <span class="pub-author" title="Aris Baras">A Baras</span>, <span class="pub-author" title="Julie E Buring">JE Buring</span>, <span class="pub-author" title="Paul M Ridker">PM Ridker</span>, <span class="pub-author" title="Daniel I Chasman">DI Chasman</span>, <span class="pub-author" title="Charles Kooperberg">C Kooperberg</span>, <span class="pub-author" title="Wei-Qi Wei">WQ Wei</span>, <span class="pub-author" title="Gail P Jarvik">GP Jarvik</span>, <span class="pub-author" title="Bahram Namjou">B Namjou</span>, <span class="pub-author" title="M Geoffrey Hayes">MG Hayes</span>, <span class="pub-author" title="Marylyn D Ritchie">MD Ritchie</span>, <span class="pub-author" title="Pekka Jousilahti">P Jousilahti</span>, <span class="pub-author" title="Veikko Salomaa">V Salomaa</span>, <span class="pub-author" title="Kristian Hveem">K Hveem</span>, <span class="pub-author" title="Bjørn Olav Åsvold">BO Åsvold</span>, <span class="pub-author" title="Michiaki Kubo">M Kubo</span>, <span class="pub-author" title="Yoichiro Kamatani">Y Kamatani</span>, <span class="pub-author" title="Yukinori Okada">Y Okada</span>, <span class="pub-author" title="Yoshinori Murakami">Y Murakami</span>, <span class="pub-author" title="Unnur Thorsteinsdottir">U Thorsteinsdottir</span>, <span class="pub-author" title="Kari Stefansson">K Stefansson</span>, <span class="pub-author" title="Yuk-Lam Ho">YL Ho</span>, <span class="pub-author" title="Julie A Lynch">JA Lynch</span>, <span class="pub-author" title="Daniel J Rader">DJ Rader</span>, <span class="pub-author" title="Philip S Tsao">PS Tsao</span>, <span class="pub-author" title="Kyong-Mi Chang">KM Chang</span>, <span class="pub-author" title="Kelly Cho">K Cho</span>, <span class="pub-author" title="Christopher J O'Donnell">CJ O'Donnell</span>, <span class="pub-author" title="John M Gaziano">JM Gaziano</span>, <span class="pub-author" title="Peter Wilson">P Wilson</span>, <span class="pub-author" title="Charles N Rotimi">CN Rotimi</span>, <span class="pub-author" title="Scott Hazelhurst">S Hazelhurst</span>, <span class="pub-author" title="Michèle Ramsay">M Ramsay</span>, <span class="pub-author" title="Richard C Trembath">RC Trembath</span>, <span class="pub-author" title="David A van Heel">DA van Heel</span>, <span class="pub-author" title="Gen Tamiya">G Tamiya</span>, <span class="pub-author" title="Masayuki Yamamoto">M Yamamoto</span>, <span class="pub-author" title="Bong-Jo Kim">BJ Kim</span>, <span class="pub-author" title="Karen L Mohlke">KL Mohlke</span>, <span class="pub-author" title="Timothy M Frayling">TM Frayling</span>, <span class="pub-author" title="Joel N Hirschhorn">JN Hirschhorn</span>, <span class="pub-author" title="Sekar Kathiresan">S Kathiresan</span>, <span class="pub-author" title=" VA Million Veteran Program"> VA Million Veteran Program</span>, <span class="pub-author" title=" Global Lipids Genetics Consortium*"> Global Lipids Genetics Consortium*</span>, <span class="pub-author" title="Michael Boehnke">M Boehnke</span>, <span class="pub-author" title="Pradeep Natarajan">P Natarajan</span>, <span class="pub-author" title="Gina M Peloso">GM Peloso</span>, <span class="pub-author" title="Christopher D Brown">CD Brown</span>, <span class="pub-author" title="Andrew P Morris">AP Morris</span>, <span class="pub-author" title="Themistocles L Assimes">TL Assimes</span>, <span class="pub-author" title="Panos Deloukas">P Deloukas</span>, <span class="pub-author" title="Yan V Sun">YV Sun</span>, <span class="pub-author" title="Cristen J Willer">CJ Willer</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Increased blood lipid levels are heritable risk factors of cardiovascular disease with varied prevalence worldwide owing to different dietary patterns and medication use1. Despite advances in prevention and treatment, in particular through reducing low-density lipoprotein cholesterol levels2, heart disease remains the leading cause of death worldwide3. Genome-wideassociation studies (GWAS) of blood lipid levels have led to important biological and clinical insights, as well as new drug targets, for cardiovascular disease. However, most previous GWAS4-23 have been conducted in European ancestry populations and may have missed genetic variants that contribute to lipid-level variation in other ancestry groups. These include differences in allele frequencies, effect sizes and linkage-disequilibrium patterns24. Here we conduct a multi-ancestry, genome-wide genetic discovery meta-analysis of lipid levels in approximately 1.65 million individuals, including 350,000 of non-European ancestries. We quantify the gain in studying non-European ancestries and provide evidence to support the expansion of recruitment of additional ancestries, even with relatively small sample sizes. We find that increasing diversity rather than studying additional individuals of European ancestry results in substantial improvements in fine-mapping functional variants and portability of polygenic prediction (evaluated in approximately 295,000 individuals from 7 ancestry groupings). Modest gains in the number of discovered loci and ancestry-specific variants were also achieved. As GWAS expand emphasis beyond the identification of genes and fundamental biology towards the use of genetic variants for preventive and precision medicine25, we anticipate that increased diversity of participants will lead to more accurate and equitable26 application of polygenic scores in clinical practice.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34887591" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41586-021-04064-3" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/e389be71822d45d6ac04d848bfb48df6" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_cae2c19d93fe471eac4b766e6643b20a" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Development and characterization of an EMS-mutagenized wheat population and identification of salt-tolerant wheat lines.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Johanna Lethin">J Lethin</span>, <span class="pub-author" title="Shahriar S M Shakil">SSM Shakil</span>, <span class="pub-author" title="Sameer Hassan">S Hassan</span>, <span class="pub-author" title="Nick Sirijovski">N Sirijovski</span>, <span class="pub-author" title="Mats Töpel">M Töpel</span>, <span class="pub-author" title="Olof Olsson">O Olsson</span>, <span class="pub-author" title="Henrik Aronsson">H Aronsson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Triticum aestivum (wheat) is one of the world's oldest crops and has been used for >8000 years as a food crop in North Africa, West Asia and Europe. Today, wheat is one of the most important sources of grain for humans, and is cultivated on greater areas of land than any other crop. As the human population increases and soil salinity becomes more prevalent, there is increased pressure on wheat breeders to develop salt-tolerant varieties in order to meet growing demands for yield and grain quality. Here we developed a mutant wheat population using the moderately salt-tolerant Bangladeshi variety BARI Gom-25, with the primary goal of further increasing salt tolerance.

After titrating the optimal ethyl methanesulfonate (EMS) concentration, ca 30,000 seeds were treated with 1% EMS, and 1676 lines, all originating from single seeds, survived through the first four generations. Most mutagenized lines showed a similar phenotype to BARI Gom-25, although visual differences such as dwarfing, giant plants, early and late flowering and altered leaf morphology were seen in some lines. By developing an assay for salt tolerance, and by screening the mutagenized population, we identified 70 lines exhibiting increased salt tolerance. The selected lines typically showed a 70% germination rate on filter paper soaked in 200 mM NaCl, compared to 0-30% for BARI Gom-25. From two of the salt-tolerant OlsAro lines (OA42 and OA70), genomic DNA was sequenced to 15x times coverage. A comparative analysis against the BARI Gom-25 genomic sequence identified a total of 683,201 (OA42), and 768,954 (OA70) SNPs distributed throughout the three sub-genomes (A, B and D). The mutation frequency was determined to be approximately one per 20,000 bp. All the 70 selected salt-tolerant lines were tested for root growth in the laboratory, and under saline field conditions in Bangladesh. The results showed that all the lines selected for tolerance showed a better salt tolerance phenotype than both BARI Gom-25 and other local wheat varieties tested.

The mutant wheat population developed here will be a valuable resource in the development of novel salt-tolerant varieties for the benefit of saline farming.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31931695" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12870-019-2137-8" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/cae2c19d93fe471eac4b766e6643b20a" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_59b8dae6dc8d41c6885ed37cf0f856a3" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Improved phylogenomic sampling of free-living nematodes enhances resolution of higher-level nematode phylogeny.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ashleigh B Smythe">AB Smythe</span>, <span class="pub-author" title="Oleksandr Holovachov">O Holovachov</span>, <span class="pub-author" title="Kevin M Kocot">KM Kocot</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Nematodes are among the most diverse and abundant metazoans on Earth, but research on them has been biased toward parasitic taxa and model organisms. Free-living nematodes, particularly from the clades Enoplia and Dorylaimia, have been underrepresented in genome-scale phylogenetic analyses to date, leading to poor resolution of deep relationships within the phylum.

We supplemented publicly available data by sequencing transcriptomes of nine free-living nematodes and two important outgroups and conducted a phylum-wide phylogenomic analysis including a total of 108 nematodes. Analysis of a dataset generated using a conservative orthology inference strategy resulted in a matrix with a high proportion of missing data and moderate to weak support for branching within and placement of Enoplia. A less conservative orthology inference approach recovered more genes and resulted in higher support for the deepest splits within Nematoda, recovering Enoplia as the sister taxon to the rest of Nematoda. Relationships within major clades were similar to those found in previously published studies based on 18S rDNA.

Expanded transcriptome sequencing of free-living nematodes has contributed to better resolution among deep nematode lineages, though the dataset is still strongly biased toward parasites. Inclusion of more free-living nematodes in future phylogenomic analyses will allow a clearer understanding of many interesting aspects of nematode evolution, such as morphological and molecular adaptations to parasitism and whether nematodes originated in a marine or terrestrial environment.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31195978" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12862-019-1444-x" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/59b8dae6dc8d41c6885ed37cf0f856a3" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_f892ff63669141bc8f46e7fd8d8fde48" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">SweGen: a whole-genome data resource of genetic variability in a cross-section of the Swedish population.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Adam Ameur">A Ameur</span>, <span class="pub-author" title="Johan Dahlberg">J Dahlberg</span>, <span class="pub-author" title="Pall Olason">P Olason</span>, <span class="pub-author" title="Francesco Vezzi">F Vezzi</span>, <span class="pub-author" title="Robert Karlsson">R Karlsson</span>, <span class="pub-author" title="Marcel Martin">M Martin</span>, <span class="pub-author" title="Johan Viklund">J Viklund</span>, <span class="pub-author" title="Andreas Kusalananda Kähäri">AK Kähäri</span>, <span class="pub-author" title="Pär Lundin">P Lundin</span>, <span class="pub-author" title="Huiwen Che">H Che</span>, <span class="pub-author" title="Jessada Thutkawkorapin">J Thutkawkorapin</span>, <span class="pub-author" title="Jesper Eisfeldt">J Eisfeldt</span>, <span class="pub-author" title="Samuel Lampa">S Lampa</span>, <span class="pub-author" title="Mats Dahlberg">M Dahlberg</span>, <span class="pub-author" title="Jonas Hagberg">J Hagberg</span>, <span class="pub-author" title="Niclas Jareborg">N Jareborg</span>, <span class="pub-author" title="Ulrika Liljedahl">U Liljedahl</span>, <span class="pub-author" title="Inger Jonasson">I Jonasson</span>, <span class="pub-author" title="Åsa Johansson">Å Johansson</span>, <span class="pub-author" title="Lars Feuk">L Feuk</span>, <span class="pub-author" title="Joakim Lundeberg">J Lundeberg</span>, <span class="pub-author" title="Ann-Christine Syvänen">AC Syvänen</span>, <span class="pub-author" title="Sverker Lundin">S Lundin</span>, <span class="pub-author" title="Daniel Nilsson">D Nilsson</span>, <span class="pub-author" title="Björn Nystedt">B Nystedt</span>, <span class="pub-author" title="Patrik Ke Magnusson">PK Magnusson</span>, <span class="pub-author" title="Ulf Gyllensten">U Gyllensten</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">Here we describe the SweGen data set, a comprehensive map of genetic variation in the Swedish population. These data represent a basic resource for clinical genetics laboratories as well as for sequencing-based association studies by providing information on genetic variant frequencies in a cohort that is well matched to national patient cohorts. To select samples for this study, we first examined the genetic structure of the Swedish population using high-density SNP-array data from a nation-wide cohort of over 10 000 Swedish-born individuals included in the Swedish Twin Registry. A total of 1000 individuals, reflecting a cross-section of the population and capturing the main genetic structure, were selected for whole-genome sequencing. Analysis pipelines were developed for automated alignment, variant calling and quality control of the sequencing data. This resulted in a genome-wide collection of aggregated variant frequencies in the Swedish population that we have made available to the scientific community through the website https://swefreq.nbis.se. A total of 29.2 million single-nucleotide variants and 3.8 million indels were detected in the 1000 samples, with 9.9 million of these variants not present in current databases. Each sample contributed with an average of 7199 individual-specific variants. In addition, an average of 8645 larger structural variants (SVs) were detected per individual, and we demonstrate that the population frequencies of these SVs can be used for efficient filtering analyses. Finally, our results show that the genetic diversity within Sweden is substantial compared with the diversity among continental European populations, underscoring the relevance of establishing a local reference data set.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/28832569" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/ejhg.2017.130" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/f892ff63669141bc8f46e7fd8d8fde48" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_f4f2d21b7c8b4a61a312b996c70f1d51" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Consistent B Cell Receptor Immunoglobulin Features Between Siblings in Familial Chronic Lymphocytic Leukemia.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="P Martijn Kolijn">PM Kolijn</span>, <span class="pub-author" title="Alice F Muggen">AF Muggen</span>, <span class="pub-author" title="Viktor Ljungström">V Ljungström</span>, <span class="pub-author" title="Andreas Agathangelidis">A Agathangelidis</span>, <span class="pub-author" title="Ingrid L M Wolvers-Tettero">ILM Wolvers-Tettero</span>, <span class="pub-author" title="H Berna Beverloo">HB Beverloo</span>, <span class="pub-author" title="Karol Pál">K Pál</span>, <span class="pub-author" title="Paul J Hengeveld">PJ Hengeveld</span>, <span class="pub-author" title="Nikos Darzentas">N Darzentas</span>, <span class="pub-author" title="Rudi W Hendriks">RW Hendriks</span>, <span class="pub-author" title="Jacques J M van Dongen">JJM van Dongen</span>, <span class="pub-author" title="Richard Rosenquist">R Rosenquist</span>, <span class="pub-author" title="Anton W Langerak">AW Langerak</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Key processes in the onset and evolution of chronic lymphocytic leukemia (CLL) are thought to include chronic (antigenic) activation of mature B cells through the B cell receptor (BcR), signals from the microenvironment, and acquisition of genetic alterations. Here we describe three families in which two or more siblings were affected by CLL. We investigated whether there are immunogenetic similarities in the leukemia-specific immunoglobulin heavy (IGH) and light (IGL/IGK) chain gene rearrangements of the siblings in each family. Furthermore, we performed array analysis to study if similarities in CLL-associated chromosomal aberrations are present within each family and screened for somatic mutations using paired tumor/normal whole-genome sequencing (WGS). In two families a consistent IGHV gene mutational status (one IGHV-unmutated, one IGHV-mutated) was observed. Intriguingly, the third family with four affected siblings was characterized by usage of the lambda IGLV3-21 gene, with the hallmark R110 mutation of the recently described clinically aggressive IGLV3-21R110 subset. In this family, the CLL-specific rearrangements in two siblings could be assigned to either stereotyped subset #2 or the immunogenetically related subset #169, both of which belong to the broader IGLV3-21R110 subgroup. Consistent patterns of cytogenetic aberrations were encountered in all three families. Furthermore, the CLL clones carried somatic mutations previously associated with IGHV mutational status, cytogenetic aberrations and stereotyped subsets, respectively. From these findings, we conclude that similarities in immunogenetic characteristics in familial CLL, in combination with genetic aberrations acquired, point towards shared underlying mechanisms behind CLL development within each family.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34513715" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.3389/fonc.2021.740083" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/f4f2d21b7c8b4a61a312b996c70f1d51" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_58825a68ec3c48ff88d9c006248ff996" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Full Mitogenomes in the Critically Endangered Kākāpō Reveal Major Post-Glacial and Anthropogenic Effects on Neutral Genetic Diversity.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Nicolas Dussex">N Dussex</span>, <span class="pub-author" title="Johanna von Seth">J von Seth</span>, <span class="pub-author" title="Bruce C Robertson">BC Robertson</span>, <span class="pub-author" title="Love Dalén">L Dalén</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Understanding how species respond to population declines is a central question in conservation and evolutionary biology. Population declines are often associated with loss of genetic diversity, inbreeding and accumulation of deleterious mutations, which can lead to a reduction in fitness and subsequently contribute to extinction. Using temporal approaches can help us understand the effects of population declines on genetic diversity in real time. Sequencing pre-decline as well as post-decline mitogenomes representing all the remaining mitochondrial diversity, we estimated the loss of genetic diversity in the critically endangered kākāpō (Strigops habroptilus). We detected a signal of population expansion coinciding with the end of the Pleistocene last glacial maximum (LGM). Also, we found some evidence for northern and southern lineages, supporting the hypothesis that the species may have been restricted to isolated northern and southern refugia during the LGM. We observed an important loss of neutral genetic diversity associated with European settlement in New Zealand but we could not exclude a population decline associated with Polynesian settlement in New Zealand. However, we did not find evidence for fixation of deleterious mutations. We argue that despite high pre-decline genetic diversity, a rapid and range-wide decline combined with the lek mating system, and life-history traits of kākāpō contributed to a rapid loss of genetic diversity following severe population declines.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/29671759" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.3390/genes9040220" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/58825a68ec3c48ff88d9c006248ff996" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_06ca672ce48846a38877306910ec4cd1" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Small‐scale thermal habitat variability may not determine seagrass resilience to climate change</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Giannina S I Hattich">GSI Hattich</span>, <span class="pub-author" title="Marlene Jahnke">M Jahnke</span>, <span class="pub-author" title="Swantje Enge">S Enge</span>, <span class="pub-author" title="Niklas Niemi">N Niemi</span>, <span class="pub-author" title="Maru Bernal‐Gómez">M Bernal‐Gómez</span>, <span class="pub-author" title="Pierre De Wit">P De Wit</span>, <span class="pub-author" title="Jonathan N Havenhand">JN Havenhand</span>, <span class="pub-author" title="Christian Pansch">C Pansch</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body d-none"></div>
                    <div class="modal-footer border-0">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/null" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1002/lno.70049" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/06ca672ce48846a38877306910ec4cd1" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_beb46404be78423ebb108a8aed3fceee" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Combined prenatal Lactobacillus reuteri and ω-3 supplementation synergistically modulates DNA methylation in neonatal T helper cells.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Johanna Huoman">J Huoman</span>, <span class="pub-author" title="David Martínez-Enguita">D Martínez-Enguita</span>, <span class="pub-author" title="Elin Olsson">E Olsson</span>, <span class="pub-author" title="Jan Ernerudh">J Ernerudh</span>, <span class="pub-author" title="Lennart Nilsson">L Nilsson</span>, <span class="pub-author" title="Karel Duchén">K Duchén</span>, <span class="pub-author" title="Mika Gustafsson">M Gustafsson</span>, <span class="pub-author" title="Maria C Jenmalm">MC Jenmalm</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Environmental exposures may alter DNA methylation patterns of T helper cells. As T helper cells are instrumental for allergy development, changes in methylation patterns may constitute a mechanism of action for allergy preventive interventions. While epigenetic effects of separate perinatal probiotic or ω-3 fatty acid supplementation have been studied previously, the combined treatment has not been assessed. We aimed to investigate epigenome-wide DNA methylation patterns from a sub-group of children in an on-going randomised double-blind placebo-controlled allergy prevention trial using pre- and postnatal combined Lactobacillus reuteri and ω-3 fatty acid treatment. To this end, > 866000 CpG sites (MethylationEPIC 850K array) in cord blood CD4+ T cells were examined in samples from all four study arms (double-treatment: n = 18, single treatments: probiotics n = 16, ω-3 n = 15, and double placebo: n = 14). Statistical and bioinformatic analyses identified treatment-associated differentially methylated CpGs and genes, which were used to identify putatively treatment-induced network modules. Pathway analyses inferred biological relevance, and comparisons were made to an independent allergy data set.

Comparing the active treatments to the double placebo group, most differentially methylated CpGs and genes were hypermethylated, possibly suggesting induction of transcriptional inhibition. The double-treated group showed the largest number of differentially methylated CpGs, of which many were unique, suggesting synergy between interventions. Clusters within the double-treated network module consisted of immune-related pathways, including T cell receptor signalling, and antigen processing and presentation, with similar pathways revealed for the single-treatment modules. CpGs derived from differential methylation and network module analyses were enriched in an independent allergy data set, particularly in the double-treatment group, proposing treatment-induced DNA methylation changes as relevant for allergy development.

Prenatal L. reuteri and/or ω-3 fatty acid treatment results in hypermethylation and affects immune- and allergy-related pathways in neonatal T helper cells, with potentially synergistic effects between the interventions and relevance for allergic disease. Further studies need to address these findings on a transcriptional level, and whether the results associate to allergy development in the children. Understanding the role of DNA methylation in regulating effects of perinatal probiotic and ω-3 interventions may provide essential knowledge in the development of efficacious allergy preventive strategies. Trial registration ClinicalTrials.gov, ClinicalTrials.gov-ID: NCT01542970. Registered 27th of February 2012-Retrospectively registered, https://clinicaltrials.gov/ct2/show/NCT01542970 .</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34193262" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s13148-021-01115-4" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/beb46404be78423ebb108a8aed3fceee" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_2561d68da0bd4709abff07531e1c6b60" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Gene count from target sequence capture places three whole genome duplication events in Hibiscus L. (Malvaceae).</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="J S Eriksson">JS Eriksson</span>, <span class="pub-author" title="C D Bacon">CD Bacon</span>, <span class="pub-author" title="D J Bennett">DJ Bennett</span>, <span class="pub-author" title="B E Pfeil">BE Pfeil</span>, <span class="pub-author" title="B Oxelman">B Oxelman</span>, <span class="pub-author" title="A Antonelli">A Antonelli</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The great diversity in plant genome size and chromosome number is partly due to polyploidization (i.e. genome doubling events). The differences in genome size and chromosome number among diploid plant species can be a window into the intriguing phenomenon of past genome doubling that may be obscured through time by the process of diploidization. The genus Hibiscus L. (Malvaceae) has a wide diversity of chromosome numbers and a complex genomic history. Hibiscus is ideal for exploring past genomic events because although two ancient genome duplication events have been identified, more are likely to be found due to its diversity of chromosome numbers. To reappraise the history of whole-genome duplication events in Hibiscus, we tested three alternative scenarios describing different polyploidization events.

Using target sequence capture, we designed a new probe set for Hibiscus and generated 87 orthologous genes from four diploid species. We detected paralogues in > 54% putative single-copy genes. 34 of these genes were selected for testing three different genome duplication scenarios using gene counting. All species of Hibiscus sampled shared one genome duplication with H. syriacus, and one whole genome duplication occurred along the branch leading to H. syriacus.

Here, we corroborated the independent genome doubling previously found in the lineage leading to H. syriacus and a shared genome doubling of this lineage and the remainder of Hibiscus. Additionally, we found a previously undiscovered genome duplication shared by the /Pavonia and /Malvaviscus clades (both nested within Hibiscus) with the occurrences of two copies in what were otherwise single-copy genes. Our results highlight the complexity of genomic diversity in some plant groups, which makes orthology assessment and accurate phylogenomic inference difficult.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34078291" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s12862-021-01751-7" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/2561d68da0bd4709abff07531e1c6b60" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_06bdda0e02974b29a11b5fc22fa91b3b" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genetic Predisposition to Coronary Artery Disease in Type 2 Diabetes Mellitus.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Natalie R van Zuydam">NR van Zuydam</span>, <span class="pub-author" title="Claes Ladenvall">C Ladenvall</span>, <span class="pub-author" title="Benjamin F Voight">BF Voight</span>, <span class="pub-author" title="Rona J Strawbridge">RJ Strawbridge</span>, <span class="pub-author" title="Juan Fernandez-Tajes">J Fernandez-Tajes</span>, <span class="pub-author" title="N William Rayner">NW Rayner</span>, <span class="pub-author" title="Neil R Robertson">NR Robertson</span>, <span class="pub-author" title="Anubha Mahajan">A Mahajan</span>, <span class="pub-author" title="Efthymia Vlachopoulou">E Vlachopoulou</span>, <span class="pub-author" title="Anuj Goel">A Goel</span>, <span class="pub-author" title="Marcus E Kleber">ME Kleber</span>, <span class="pub-author" title="Christopher P Nelson">CP Nelson</span>, <span class="pub-author" title="Lydia Coulter Kwee">LC Kwee</span>, <span class="pub-author" title="Tõnu Esko">T Esko</span>, <span class="pub-author" title="Evelin Mihailov">E Mihailov</span>, <span class="pub-author" title="Reedik Mägi">R Mägi</span>, <span class="pub-author" title="Lili Milani">L Milani</span>, <span class="pub-author" title="Krista Fischer">K Fischer</span>, <span class="pub-author" title="Stavroula Kanoni">S Kanoni</span>, <span class="pub-author" title="Jitender Kumar">J Kumar</span>, <span class="pub-author" title="Ci Song">C Song</span>, <span class="pub-author" title="Jaana A Hartiala">JA Hartiala</span>, <span class="pub-author" title="Nancy L Pedersen">NL Pedersen</span>, <span class="pub-author" title="Markus Perola">M Perola</span>, <span class="pub-author" title="Christian Gieger">C Gieger</span>, <span class="pub-author" title="Annette Peters">A Peters</span>, <span class="pub-author" title="Liming Qu">L Qu</span>, <span class="pub-author" title="Sara M Willems">SM Willems</span>, <span class="pub-author" title="Alex S F Doney">ASF Doney</span>, <span class="pub-author" title="Andrew D Morris">AD Morris</span>, <span class="pub-author" title="Yan Zheng">Y Zheng</span>, <span class="pub-author" title="Giorgio Sesti">G Sesti</span>, <span class="pub-author" title="Frank B Hu">FB Hu</span>, <span class="pub-author" title="Lu Qi">L Qi</span>, <span class="pub-author" title="Markku Laakso">M Laakso</span>, <span class="pub-author" title="Unnur Thorsteinsdottir">U Thorsteinsdottir</span>, <span class="pub-author" title="Harald Grallert">H Grallert</span>, <span class="pub-author" title="Cornelia van Duijn">C van Duijn</span>, <span class="pub-author" title="Muredach P Reilly">MP Reilly</span>, <span class="pub-author" title="Erik Ingelsson">E Ingelsson</span>, <span class="pub-author" title="Panos Deloukas">P Deloukas</span>, <span class="pub-author" title="Sek Kathiresan">S Kathiresan</span>, <span class="pub-author" title="Andres Metspalu">A Metspalu</span>, <span class="pub-author" title="Svati H Shah">SH Shah</span>, <span class="pub-author" title="Juha Sinisalo">J Sinisalo</span>, <span class="pub-author" title="Veikko Salomaa">V Salomaa</span>, <span class="pub-author" title="Anders Hamsten">A Hamsten</span>, <span class="pub-author" title="Nilesh J Samani">NJ Samani</span>, <span class="pub-author" title="Winfried März">W März</span>, <span class="pub-author" title="Stanley L Hazen">SL Hazen</span>, <span class="pub-author" title="Hugh Watkins">H Watkins</span>, <span class="pub-author" title="Danish Saleheen">D Saleheen</span>, <span class="pub-author" title="Andrew P Morris">AP Morris</span>, <span class="pub-author" title="Helen M Colhoun">HM Colhoun</span>, <span class="pub-author" title="Leif Groop">L Groop</span>, <span class="pub-author" title="Mark I McCarthy">MI McCarthy</span>, <span class="pub-author" title="Colin N A Palmer">CNA Palmer</span>, <span class="pub-author" title=" SUMMIT Steering Committee; CARDIOGRAMplusC4D Steering Committee*"> SUMMIT Steering Committee; CARDIOGRAMplusC4D Steering Committee*</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Coronary artery disease (CAD) is accelerated in subjects with type 2 diabetes mellitus (T2D).

To test whether this reflects differential genetic influences on CAD risk in subjects with T2D, we performed a systematic assessment of genetic overlap between CAD and T2D in 66 643 subjects (27 708 with CAD and 24 259 with T2D). Variants showing apparent association with CAD in stratified analyses or evidence of interaction were evaluated in a further 117 787 subjects (16 694 with CAD and 11 537 with T2D).

None of the previously characterized CAD loci was found to have specific effects on CAD in T2D individuals, and a genome-wide interaction analysis found no new variants for CAD that could be considered T2D specific. When we considered the overall genetic correlations between CAD and its risk factors, we found no substantial differences in these relationships by T2D background.

This study found no evidence that the genetic architecture of CAD differs in those with T2D compared with those without T2D.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33321069" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1161/CIRCGEN.119.002769" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/06bdda0e02974b29a11b5fc22fa91b3b" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_a97fb528f0834255b1e9c8103f8ff2bc" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genome Sequence Resource for the Oomycete Taro Pathogen Phytophthora colocasiae.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ramesh R Vetukuri">RR Vetukuri</span>, <span class="pub-author" title="Sandeep K Kushwaha">SK Kushwaha</span>, <span class="pub-author" title="Diya Sen">D Sen</span>, <span class="pub-author" title="Stephen C Whisson">SC Whisson</span>, <span class="pub-author" title="Kurt H Lamour">KH Lamour</span>, <span class="pub-author" title="Laura J Grenville-Briggs">LJ Grenville-Briggs</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Phytophthora colocasiae is a phytopathogenic oomycete that causes leaf blight and corm rot on taro (Colocasia esculenta), an important staple crop in the tropics. The impact of P. colocasiae is a serious concern for food security in Asian and Oceanic regions. Vietnamese strain 7290 of P. colocasiae was sequenced (Illumina) to assemble a draft genome of 56.6 Mb, comprised of 19,853 scaffolds and 19,984 predicted protein-coding genes. As in other Phytophthora species, P. colocasiae possesses numerous pathogenicity-related genes, such as the RxLR class of effectors. This draft genome sequence of P. colocasiae provides a resource to underpin the first steps in determining the molecular mechanisms of disease development in this pathosystem.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/29558317" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1094/MPMI-12-17-0321-A" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/a97fb528f0834255b1e9c8103f8ff2bc" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_d033a5c5278c4463ad6a2a211e4b46ea" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Ultralow amounts of DNA from long-term archived serum samples produce high-quality methylomes.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Marcin W Wojewodzic">MW Wojewodzic</span>, <span class="pub-author" title="Magnus Leithaug">M Leithaug</span>, <span class="pub-author" title="Marianne Lauritzen">M Lauritzen</span>, <span class="pub-author" title="Robert Lyle">R Lyle</span>, <span class="pub-author" title="Sofia Haglund">S Haglund</span>, <span class="pub-author" title="Carl-Johan Rubin">CJ Rubin</span>, <span class="pub-author" title="Philip A Ewels">PA Ewels</span>, <span class="pub-author" title="Tom Grotmol">T Grotmol</span>, <span class="pub-author" title="Trine B Rounge">TB Rounge</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Long-term stored serum is considered challenging for epigenomic analyses: as there are no cells, circulating DNA is scarce, and amplification removes epigenetic signals. Additionally, pre-analytical treatments and storage might introduce biases and fragmentation to the DNA. In particular, starting with low-input DNA can result in low-diversity libraries. However, successful whole-genome bisulphite sequencing (WGBS) of such serum samples has the potential to open biobanks for epigenetic analyses and deliver novel prediagnostic biomarkers. Here, we perform WGBS using the Accel-NGS library preparation kit on ultralow amounts of DNA from long-term archived samples with diverse pretreatments from the Janus Serum Bank.

Ninety-four of the 96 samples produced satisfactory methylation calls; an average of 578 M reads per sample generated a mean coverage of 17× and mean duplication level of 35%. Failed samples were related to poor bisulphite conversion rather than to sequencing or library preparation. We demonstrate the feasibility of WGBS on ultralow DNA yields from serum samples stored up to 48 years.

Our results show the potential of large serum biobank collections for future epigenomic studies and biomarker discovery.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33980276" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1186/s13148-021-01097-3" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/d033a5c5278c4463ad6a2a211e4b46ea" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_19379a893e0b48ff9f84c9acbc2f7509" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Microbiological Surveillance of Biogas Plants: Targeting Acetogenic Community.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Abhijeet Singh">A Singh</span>, <span class="pub-author" title="Jan Moestedt">J Moestedt</span>, <span class="pub-author" title="Andreas Berg">A Berg</span>, <span class="pub-author" title="Anna Schnürer">A Schnürer</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Acetogens play a very important role in anaerobic digestion and are essential in ensuring process stability. Despite this, targeted studies of the acetogenic community in biogas processes remain limited. Some efforts have been made to identify and understand this community, but the lack of a reliable molecular analysis strategy makes the detection of acetogenic bacteria tedious. Recent studies suggest that screening of bacterial genetic material for formyltetrahydrofolate synthetase (FTHFS), a key marker enzyme in the Wood-Ljungdahl pathway, can give a strong indication of the presence of putative acetogens in biogas environments. In this study, we applied an acetogen-targeted analyses strategy developed previously by our research group for microbiological surveillance of commercial biogas plants. The surveillance comprised high-throughput sequencing of FTHFS gene amplicons and unsupervised data analysis with the AcetoScan pipeline. The results showed differences in the acetogenic community structure related to feed substrate and operating parameters. They also indicated that our surveillance method can be helpful in the detection of community changes before observed changes in physico-chemical profiles, and that frequent high-throughput surveillance can assist in management towards stable process operation, thus improving the economic viability of biogas plants. To our knowledge, this is the first study to apply a high-throughput microbiological surveillance approach to visualise the potential acetogenic population in commercial biogas digesters.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34484143" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.3389/fmicb.2021.700256" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/19379a893e0b48ff9f84c9acbc2f7509" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_bd53cdd9312a4daeb054082f3ef37158" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">The genomics of heart failure: design and rationale of the HERMES consortium.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="R Thomas Lumbers">RT Lumbers</span>, <span class="pub-author" title="Sonia Shah">S Shah</span>, <span class="pub-author" title="Honghuang Lin">H Lin</span>, <span class="pub-author" title="Tomasz Czuba">T Czuba</span>, <span class="pub-author" title="Albert Henry">A Henry</span>, <span class="pub-author" title="Daniel I Swerdlow">DI Swerdlow</span>, <span class="pub-author" title="Anders Mälarstig">A Mälarstig</span>, <span class="pub-author" title="Charlotte Andersson">C Andersson</span>, <span class="pub-author" title="Niek Verweij">N Verweij</span>, <span class="pub-author" title="Michael V Holmes">MV Holmes</span>, <span class="pub-author" title="Johan Ärnlöv">J Ärnlöv</span>, <span class="pub-author" title="Per Svensson">P Svensson</span>, <span class="pub-author" title="Harry Hemingway">H Hemingway</span>, <span class="pub-author" title="Neneh Sallah">N Sallah</span>, <span class="pub-author" title="Peter Almgren">P Almgren</span>, <span class="pub-author" title="Krishna G Aragam">KG Aragam</span>, <span class="pub-author" title="Geraldine Asselin">G Asselin</span>, <span class="pub-author" title="Joshua D Backman">JD Backman</span>, <span class="pub-author" title="Mary L Biggs">ML Biggs</span>, <span class="pub-author" title="Heather L Bloom">HL Bloom</span>, <span class="pub-author" title="Eric Boersma">E Boersma</span>, <span class="pub-author" title="Jeffrey Brandimarto">J Brandimarto</span>, <span class="pub-author" title="Michael R Brown">MR Brown</span>, <span class="pub-author" title="Hans-Peter Brunner-La Rocca">HP Brunner-La Rocca</span>, <span class="pub-author" title="David J Carey">DJ Carey</span>, <span class="pub-author" title="Mark D Chaffin">MD Chaffin</span>, <span class="pub-author" title="Daniel I Chasman">DI Chasman</span>, <span class="pub-author" title="Olympe Chazara">O Chazara</span>, <span class="pub-author" title="Xing Chen">X Chen</span>, <span class="pub-author" title="Xu Chen">X Chen</span>, <span class="pub-author" title="Jonathan H Chung">JH Chung</span>, <span class="pub-author" title="William Chutkow">W Chutkow</span>, <span class="pub-author" title="John G F Cleland">JGF Cleland</span>, <span class="pub-author" title="James P Cook">JP Cook</span>, <span class="pub-author" title="Simon de Denus">S de Denus</span>, <span class="pub-author" title="Abbas Dehghan">A Dehghan</span>, <span class="pub-author" title="Graciela E Delgado">GE Delgado</span>, <span class="pub-author" title="Spiros Denaxas">S Denaxas</span>, <span class="pub-author" title="Alexander S Doney">AS Doney</span>, <span class="pub-author" title="Marcus Dörr">M Dörr</span>, <span class="pub-author" title="Samuel C Dudley">SC Dudley</span>, <span class="pub-author" title="Gunnar Engström">G Engström</span>, <span class="pub-author" title="Tõnu Esko">T Esko</span>, <span class="pub-author" title="Ghazaleh Fatemifar">G Fatemifar</span>, <span class="pub-author" title="Stephan B Felix">SB Felix</span>, <span class="pub-author" title="Chris Finan">C Finan</span>, <span class="pub-author" title="Ian Ford">I Ford</span>, <span class="pub-author" title="Francoise Fougerousse">F Fougerousse</span>, <span class="pub-author" title="René Fouodjio">R Fouodjio</span>, <span class="pub-author" title="Mohsen Ghanbari">M Ghanbari</span>, <span class="pub-author" title="Sahar Ghasemi">S Ghasemi</span>, <span class="pub-author" title="Vilmantas Giedraitis">V Giedraitis</span>, <span class="pub-author" title="Franco Giulianini">F Giulianini</span>, <span class="pub-author" title="John S Gottdiener">JS Gottdiener</span>, <span class="pub-author" title="Stefan Gross">S Gross</span>, <span class="pub-author" title="Daníel F Guðbjartsson">DF Guðbjartsson</span>, <span class="pub-author" title="Hongsheng Gui">H Gui</span>, <span class="pub-author" title="Rebecca Gutmann">R Gutmann</span>, <span class="pub-author" title="Christopher M Haggerty">CM Haggerty</span>, <span class="pub-author" title="Pim van der Harst">P van der Harst</span>, <span class="pub-author" title="Åsa K Hedman">ÅK Hedman</span>, <span class="pub-author" title="Anna Helgadottir">A Helgadottir</span>, <span class="pub-author" title="Hans Hillege">H Hillege</span>, <span class="pub-author" title="Craig L Hyde">CL Hyde</span>, <span class="pub-author" title="Jaison Jacob">J Jacob</span>, <span class="pub-author" title="J Wouter Jukema">JW Jukema</span>, <span class="pub-author" title="Frederick Kamanu">F Kamanu</span>, <span class="pub-author" title="Isabella Kardys">I Kardys</span>, <span class="pub-author" title="Maryam Kavousi">M Kavousi</span>, <span class="pub-author" title="Kay-Tee Khaw">KT Khaw</span>, <span class="pub-author" title="Marcus E Kleber">ME Kleber</span>, <span class="pub-author" title="Lars Køber">L Køber</span>, <span class="pub-author" title="Andrea Koekemoer">A Koekemoer</span>, <span class="pub-author" title="Bill Kraus">B Kraus</span>, <span class="pub-author" title="Karoline Kuchenbaecker">K Kuchenbaecker</span>, <span class="pub-author" title="Claudia Langenberg">C Langenberg</span>, <span class="pub-author" title="Lars Lind">L Lind</span>, <span class="pub-author" title="Cecilia M Lindgren">CM Lindgren</span>, <span class="pub-author" title="Barry London">B London</span>, <span class="pub-author" title="Luca A Lotta">LA Lotta</span>, <span class="pub-author" title="Ruth C Lovering">RC Lovering</span>, <span class="pub-author" title="Jian'an Luan">J Luan</span>, <span class="pub-author" title="Patrik Magnusson">P Magnusson</span>, <span class="pub-author" title="Anubha Mahajan">A Mahajan</span>, <span class="pub-author" title="Douglas Mann">D Mann</span>, <span class="pub-author" title="Kenneth B Margulies">KB Margulies</span>, <span class="pub-author" title="Nicholas A Marston">NA Marston</span>, <span class="pub-author" title="Winfried März">W März</span>, <span class="pub-author" title="John J V McMurray">JJV McMurray</span>, <span class="pub-author" title="Olle Melander">O Melander</span>, <span class="pub-author" title="Giorgio Melloni">G Melloni</span>, <span class="pub-author" title="Ify R Mordi">IR Mordi</span>, <span class="pub-author" title="Michael P Morley">MP Morley</span>, <span class="pub-author" title="Andrew D Morris">AD Morris</span>, <span class="pub-author" title="Andrew P Morris">AP Morris</span>, <span class="pub-author" title="Alanna C Morrison">AC Morrison</span>, <span class="pub-author" title="Michael W Nagle">MW Nagle</span>, <span class="pub-author" title="Christopher P Nelson">CP Nelson</span>, <span class="pub-author" title="Christopher Newton-Cheh">C Newton-Cheh</span>, <span class="pub-author" title="Alexander Niessner">A Niessner</span>, <span class="pub-author" title="Teemu Niiranen">T Niiranen</span>, <span class="pub-author" title="Christoph Nowak">C Nowak</span>, <span class="pub-author" title="Michelle L O'Donoghue">ML O'Donoghue</span>, <span class="pub-author" title="Anjali T Owens">AT Owens</span>, <span class="pub-author" title="Colin N A Palmer">CNA Palmer</span>, <span class="pub-author" title="Guillaume Paré">G Paré</span>, <span class="pub-author" title="Markus Perola">M Perola</span>, <span class="pub-author" title="Louis-Philippe Lemieux Perreault">LL Perreault</span>, <span class="pub-author" title="Eliana Portilla-Fernandez">E Portilla-Fernandez</span>, <span class="pub-author" title="Bruce M Psaty">BM Psaty</span>, <span class="pub-author" title="Kenneth M Rice">KM Rice</span>, <span class="pub-author" title="Paul M Ridker">PM Ridker</span>, <span class="pub-author" title="Simon P R Romaine">SPR Romaine</span>, <span class="pub-author" title="Carolina Roselli">C Roselli</span>, <span class="pub-author" title="Jerome I Rotter">JI Rotter</span>, <span class="pub-author" title="Christian T Ruff">CT Ruff</span>, <span class="pub-author" title="Marc S Sabatine">MS Sabatine</span>, <span class="pub-author" title="Perttu Salo">P Salo</span>, <span class="pub-author" title="Veikko Salomaa">V Salomaa</span>, <span class="pub-author" title="Jessica van Setten">J van Setten</span>, <span class="pub-author" title="Alaa A Shalaby">AA Shalaby</span>, <span class="pub-author" title="Diane T Smelser">DT Smelser</span>, <span class="pub-author" title="Nicholas L Smith">NL Smith</span>, <span class="pub-author" title="Kari Stefansson">K Stefansson</span>, <span class="pub-author" title="Steen Stender">S Stender</span>, <span class="pub-author" title="David J Stott">DJ Stott</span>, <span class="pub-author" title="Garðar Sveinbjörnsson">G Sveinbjörnsson</span>, <span class="pub-author" title="Mari-Liis Tammesoo">ML Tammesoo</span>, <span class="pub-author" title="Jean-Claude Tardif">JC Tardif</span>, <span class="pub-author" title="Kent D Taylor">KD Taylor</span>, <span class="pub-author" title="Maris Teder-Laving">M Teder-Laving</span>, <span class="pub-author" title="Alexander Teumer">A Teumer</span>, <span class="pub-author" title="Guðmundur Thorgeirsson">G Thorgeirsson</span>, <span class="pub-author" title="Unnur Thorsteinsdottir">U Thorsteinsdottir</span>, <span class="pub-author" title="Christian Torp-Pedersen">C Torp-Pedersen</span>, <span class="pub-author" title="Stella Trompet">S Trompet</span>, <span class="pub-author" title="Danny Tuckwell">D Tuckwell</span>, <span class="pub-author" title="Benoit Tyl">B Tyl</span>, <span class="pub-author" title="Andre G Uitterlinden">AG Uitterlinden</span>, <span class="pub-author" title="Felix Vaura">F Vaura</span>, <span class="pub-author" title="Abirami Veluchamy">A Veluchamy</span>, <span class="pub-author" title="Peter M Visscher">PM Visscher</span>, <span class="pub-author" title="Uwe Völker">U Völker</span>, <span class="pub-author" title="Adriaan A Voors">AA Voors</span>, <span class="pub-author" title="Xiaosong Wang">X Wang</span>, <span class="pub-author" title="Nicholas J Wareham">NJ Wareham</span>, <span class="pub-author" title="Peter E Weeke">PE Weeke</span>, <span class="pub-author" title="Raul Weiss">R Weiss</span>, <span class="pub-author" title="Harvey D White">HD White</span>, <span class="pub-author" title="Kerri L Wiggins">KL Wiggins</span>, <span class="pub-author" title="Heming Xing">H Xing</span>, <span class="pub-author" title="Jian Yang">J Yang</span>, <span class="pub-author" title="Yifan Yang">Y Yang</span>, <span class="pub-author" title="Laura M Yerges-Armstrong">LM Yerges-Armstrong</span>, <span class="pub-author" title="Bing Yu">B Yu</span>, <span class="pub-author" title="Faiez Zannad">F Zannad</span>, <span class="pub-author" title="Faye Zhao">F Zhao</span>, <span class="pub-author" title=" Regeneron Genetics Center"> Regeneron Genetics Center</span>, <span class="pub-author" title="Jemma B Wilk">JB Wilk</span>, <span class="pub-author" title="Hilma Holm">H Holm</span>, <span class="pub-author" title="Naveed Sattar">N Sattar</span>, <span class="pub-author" title="Steven A Lubitz">SA Lubitz</span>, <span class="pub-author" title="David E Lanfear">DE Lanfear</span>, <span class="pub-author" title="Svati Shah">S Shah</span>, <span class="pub-author" title="Michael E Dunn">ME Dunn</span>, <span class="pub-author" title="Quinn S Wells">QS Wells</span>, <span class="pub-author" title="Folkert W Asselbergs">FW Asselbergs</span>, <span class="pub-author" title="Aroon D Hingorani">AD Hingorani</span>, <span class="pub-author" title="Marie-Pierre Dubé">MP Dubé</span>, <span class="pub-author" title="Nilesh J Samani">NJ Samani</span>, <span class="pub-author" title="Chim C Lang">CC Lang</span>, <span class="pub-author" title="Thomas P Cappola">TP Cappola</span>, <span class="pub-author" title="Patrick T Ellinor">PT Ellinor</span>, <span class="pub-author" title="Ramachandran S Vasan">RS Vasan</span>, <span class="pub-author" title="J Gustav Smith">JG Smith</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The HERMES (HEart failure Molecular Epidemiology for Therapeutic targetS) consortium aims to identify the genomic and molecular basis of heart failure.

The consortium currently includes 51 studies from 11 countries, including 68 157 heart failure cases and 949 888 controls, with data on heart failure events and prognosis. All studies collected biological samples and performed genome-wide genotyping of common genetic variants. The enrolment of subjects into participating studies ranged from 1948 to the present day, and the median follow-up following heart failure diagnosis ranged from 2 to 116 months. Forty-nine of 51 individual studies enrolled participants of both sexes; in these studies, participants with heart failure were predominantly male (34-90%). The mean age at diagnosis or ascertainment across all studies ranged from 54 to 84 years. Based on the aggregate sample, we estimated 80% power to genetic variant associations with risk of heart failure with an odds ratio of ≥1.10 for common variants (allele frequency ≥ 0.05) and ≥1.20 for low-frequency variants (allele frequency 0.01-0.05) at P < 5 × 10-8 under an additive genetic model.

HERMES is a global collaboration aiming to (i) identify the genetic determinants of heart failure; (ii) generate insights into the causal pathways leading to heart failure and enable genetic approaches to target prioritization; and (iii) develop genomic tools for disease stratification and risk prediction.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34480422" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1002/ehf2.13517" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/bd53cdd9312a4daeb054082f3ef37158" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_be4200f3e82b44bf886c89c0e8c2f362" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Temporal profiling of cytokine-induced genes in pancreatic β-cells by meta-analysis and network inference.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Miguel Lopes">M Lopes</span>, <span class="pub-author" title="Burak Kutlu">B Kutlu</span>, <span class="pub-author" title="Michela Miani">M Miani</span>, <span class="pub-author" title="Claus H Bang-Berthelsen">CH Bang-Berthelsen</span>, <span class="pub-author" title="Joachim Størling">J Størling</span>, <span class="pub-author" title="Flemming Pociot">F Pociot</span>, <span class="pub-author" title="Nathan Goodman">N Goodman</span>, <span class="pub-author" title="Lee Hood">L Hood</span>, <span class="pub-author" title="Nils Welsh">N Welsh</span>, <span class="pub-author" title="Gianluca Bontempi">G Bontempi</span>, <span class="pub-author" title="Decio L Eizirik">DL Eizirik</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Type 1 Diabetes (T1D) is an autoimmune disease where local release of cytokines such as IL-1β and IFN-γ contributes to β-cell apoptosis. To identify relevant genes regulating this process we performed a meta-analysis of 8 datasets of β-cell gene expression after exposure to IL-1β and IFN-γ. Two of these datasets are novel and contain time-series expressions in human islet cells and rat INS-1E cells. Genes were ranked according to their differential expression within and after 24 h from exposure, and characterized by function and prior knowledge in the literature. A regulatory network was then inferred from the human time expression datasets, using a time-series extension of a network inference method. The two most differentially expressed genes previously unknown in T1D literature (RIPK2 and ELF3) were found to modulate cytokine-induced apoptosis. The inferred regulatory network is thus supported by the experimental validation, providing a proof-of-concept for the proposed statistical inference approach.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/24462878" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.ygeno.2013.12.007" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/be4200f3e82b44bf886c89c0e8c2f362" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_4e29f3d9ea9a44c1bc81f0adee660b0b" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Deregulation of the A-to-I RNA editing mechanism in psychiatric disorders.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Gilad Silberberg">G Silberberg</span>, <span class="pub-author" title="Daniel Lundin">D Lundin</span>, <span class="pub-author" title="Ruth Navon">R Navon</span>, <span class="pub-author" title="Marie Öhman">M Öhman</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Schizophrenia and bipolar disorder (BPD) are common neurodevelopmental disorders, characterized by various life-crippling symptoms and high suicide rates. Multiple studies support a strong genetic involvement in the etiology of these disorders, although patterns of inheritance are variable and complex. Adenosine-to-inosine RNA editing is a cellular mechanism, which has been implicated in mental disorders and suicide. To examine the involvement of altered RNA editing in these disorders, we: (i) quantified the mRNA levels of the adenosine deaminase acting on RNA (ADAR) editing enzymes by real-time quantitative polymerase chain reaction, and (ii) measured the editing levels in transcripts of several neuroreceptors using 454 high-throughput sequencing, in dorsolateral-prefrontal cortices of schizophrenics, BPD patients and controls. Increased expression of specific ADAR2 variants with diminished catalytic activity was observed in schizophrenia. Our results also indicate that the I/V editing site in the glutamate receptor, ionotropic kainate 2 (GRIK2) transcript is under-edited in BPD (type I) patients (45.8 versus 53.9%, P= 0.023). GRIK2 has been implicated in mood disorders, and editing of its I/V site can modulate Ca(+2) permeability of the channel, consistent with numerous observations of elevated intracellular Ca(+2) levels in BPD patients. Our findings may therefore, at least partly, explain a molecular mechanism underlying the disorder. In addition, an intriguing correlation was found between editing events on separate exons of GRIK2. Finally, multiple novel editing sites were detected near previously known sites, albeit most with very low editing rates. This supports the hypothesis raised previously regarding the existence of wide-spread low-level 'background' editing as a mechanism that enhances adaptation and evolvability.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/21984433" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1093/hmg/ddr461" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/4e29f3d9ea9a44c1bc81f0adee660b0b" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_8cbc18de608b493d87efccfa5a1ae454" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Genomic signatures of rapid adaptive divergence in a tropical montane species.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Per G P Ericson">PGP Ericson</span>, <span class="pub-author" title="Martin Irestedt">M Irestedt</span>, <span class="pub-author" title="Huishang She">H She</span>, <span class="pub-author" title="Yanhua Qu">Y Qu</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Mountain regions contain extraordinary biodiversity. The environmental heterogeneity and glacial cycles often accelerate speciation and adaptation of montane species, but how these processes influence the genomic differentiation of these species is largely unknown. Using a novel chromosome-level genome and population genomic comparisons, we study allopatric divergence and selection in an iconic bird living in a tropical mountain region in New Guinea, Archbold's bowerbird (Amblyornis papuensis). Our results show that the two populations inhabiting the eastern and western Central Range became isolated ca 11 800 years ago, probably because the suitable habitats for this cold-tolerating bird decreased when the climate got warmer. Our genomic scans detect that genes in highly divergent genomic regions are over-represented in developmental processes, which is probably associated with the observed differences in body size between the populations. Overall, our results suggest that environmental differences between the eastern and western Central Range probably drive adaptive divergence between them.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34314643" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1098/rsbl.2021.0089" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/8cbc18de608b493d87efccfa5a1ae454" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_2488c6cb0e74465fb22eafbfe10e83fc" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Populus SVL Acts in Leaves to Modulate the Timing of Growth Cessation and Bud Set.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Domenique André">D André</span>, <span class="pub-author" title="José Alfredo Zambrano">JA Zambrano</span>, <span class="pub-author" title="Bo Zhang">B Zhang</span>, <span class="pub-author" title="Keh Chien Lee">KC Lee</span>, <span class="pub-author" title="Mark Rühl">M Rühl</span>, <span class="pub-author" title="Alice Marcon">A Marcon</span>, <span class="pub-author" title="Ove Nilsson">O Nilsson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">SHORT VEGETATIVE PHASE (SVP) is an important regulator of FLOWERING LOCUS T (FT) in the thermosensory pathway of Arabidopsis. It is a negative regulator of flowering and represses FT transcription. In poplar trees, FT2 is central for the photoperiodic control of growth cessation, which also requires the decrease of bioactive gibberellins (GAs). In angiosperm trees, genes similar to SVP, sometimes named DORMANCY-ASSOCIATED MADS-BOX genes, control temperature-mediated bud dormancy. Here we show that SVL, an SVP ortholog in aspen trees, besides its role in controlling dormancy through its expression in buds, is also contributing to the regulation of short day induced growth cessation and bud set through its expression in leaves. SVL is upregulated during short days in leaves and binds to the FT2 promoter to repress its transcription. It furthermore decreases the amount of active GAs, whose downregulation is essential for growth cessation, by repressing the transcription of GA20 oxidase. Finally, the SVL protein is more stable in colder temperatures, thus integrating the temperature signal into the response. We conclude that the molecular function of SVL in the photoperiodic pathway has been conserved between Arabidopsis and poplar trees, albeit the physiological process it controls has changed. SVL is thus both involved in regulating the photoperiod response in leaves, modulating the timing of growth cessation and bud set, and in the subsequent temperature regulation of dormancy in the buds.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/35251092" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.3389/fpls.2022.823019" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/2488c6cb0e74465fb22eafbfe10e83fc" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_6ba3a0a503434b1d9709bdead1716425" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Expansions of adaptive-like NK cells with a tissue-resident phenotype in human lung and blood.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Demi Brownlie">D Brownlie</span>, <span class="pub-author" title="Marlena Scharenberg">M Scharenberg</span>, <span class="pub-author" title="Jeff E Mold">JE Mold</span>, <span class="pub-author" title="Joanna Hård">J Hård</span>, <span class="pub-author" title="Eliisa Kekäläinen">E Kekäläinen</span>, <span class="pub-author" title="Marcus Buggert">M Buggert</span>, <span class="pub-author" title="Son Nguyen">S Nguyen</span>, <span class="pub-author" title="Jennifer N Wilson">JN Wilson</span>, <span class="pub-author" title="Mamdoh Al-Ameri">M Al-Ameri</span>, <span class="pub-author" title="Hans-Gustaf Ljunggren">HG Ljunggren</span>, <span class="pub-author" title="Nicole Marquardt">N Marquardt</span>, <span class="pub-author" title="Jakob Michaëlsson">J Michaëlsson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Human adaptive-like "memory" CD56dimCD16+ natural killer (NK) cells in peripheral blood from cytomegalovirus-seropositive individuals have been extensively investigated in recent years and are currently explored as a treatment strategy for hematological cancers. However, treatment of solid tumors remains limited due to insufficient NK cell tumor infiltration, and it is unknown whether large expansions of adaptive-like NK cells that are equipped for tissue residency and tumor homing exist in peripheral tissues. Here, we show that human lung and blood contains adaptive-like CD56brightCD16- NK cells with hallmarks of tissue residency, including expression of CD49a. Expansions of adaptive-like lung tissue-resident NK (trNK) cells were found to be present independently of adaptive-like CD56dimCD16+ NK cells and to be hyperresponsive toward target cells. Together, our data demonstrate that phenotypically, functionally, and developmentally distinct subsets of adaptive-like NK cells exist in human lung and blood. Given their tissue-related character and hyperresponsiveness, human lung adaptive-like trNK cells might represent a suitable alternative for therapies targeting solid tumors.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33836578" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1073/pnas.2016580118" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/6ba3a0a503434b1d9709bdead1716425" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_49398b063f5d4500a3dd373382da546a" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A genome-wide association study of imaging-defined atherosclerosis.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Anders Gummesson">A Gummesson</span>, <span class="pub-author" title="Per Lundmark">P Lundmark</span>, <span class="pub-author" title="Qiao Sen Chen">QS Chen</span>, <span class="pub-author" title="Elias Björnson">E Björnson</span>, <span class="pub-author" title="Koen F Dekkers">KF Dekkers</span>, <span class="pub-author" title="Ulf Hammar">U Hammar</span>, <span class="pub-author" title="Martin Adiels">M Adiels</span>, <span class="pub-author" title="Yunzhang Wang">Y Wang</span>, <span class="pub-author" title="Therese Andersson">T Andersson</span>, <span class="pub-author" title="Göran Bergström">G Bergström</span>, <span class="pub-author" title="Carl-Johan Carlhäll">CJ Carlhäll</span>, <span class="pub-author" title="David Erlinge">D Erlinge</span>, <span class="pub-author" title="Tomas Jernberg">T Jernberg</span>, <span class="pub-author" title="Fredrik Landfors">F Landfors</span>, <span class="pub-author" title="Lars Lind">L Lind</span>, <span class="pub-author" title="Maria Mannila">M Mannila</span>, <span class="pub-author" title="Olle Melander">O Melander</span>, <span class="pub-author" title="Carlo Pirazzi">C Pirazzi</span>, <span class="pub-author" title="Johan Sundström">J Sundström</span>, <span class="pub-author" title="Carl Johan Östgren">CJ Östgren</span>, <span class="pub-author" title="Cecilia Gunnarsson">C Gunnarsson</span>, <span class="pub-author" title="Marju Orho-Melander">M Orho-Melander</span>, <span class="pub-author" title="Stefan Söderberg">S Söderberg</span>, <span class="pub-author" title="Tove Fall">T Fall</span>, <span class="pub-author" title="Bruna Gigante">B Gigante</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Imaging-defined atherosclerosis represents an intermediate phenotype of atherosclerotic cardiovascular disease (ASCVD). Genome-wide association studies (GWAS) on directly measured coronary plaques using coronary computed tomography angiography (CCTA) are scarce. In the so far largest population-based cohort with CCTA data, we performed a GWAS on coronary plaque burden as determined by the segment involvement score (SIS) in 24,811 European individuals. We identified 20 significant independent genetic markers for SIS, three of which were found in loci not implicated in ASCVD before. Further GWAS on coronary artery calcification showed similar results to that of SIS, whereas a GWAS on ultrasound-assessed carotid plaques identified both shared and non-shared loci with SIS. In two-sample Mendelian randomization studies using SIS-associated markers in UK Biobank and CARDIoGRAMplusC4D, one extra coronary segment with atherosclerosis corresponded to 1.8-fold increased odds of myocardial infarction. This GWAS data can aid future studies of causal pathways in ASCVD.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/40164586" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41467-025-57457-7" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/49398b063f5d4500a3dd373382da546a" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_3777a5cd8954424fb426f5163d4d01bf" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Proteomics. Tissue-based map of the human proteome.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Mathias Uhlén">M Uhlén</span>, <span class="pub-author" title="Linn Fagerberg">L Fagerberg</span>, <span class="pub-author" title="Björn M Hallström">BM Hallström</span>, <span class="pub-author" title="Cecilia Lindskog">C Lindskog</span>, <span class="pub-author" title="Per Oksvold">P Oksvold</span>, <span class="pub-author" title="Adil Mardinoglu">A Mardinoglu</span>, <span class="pub-author" title="Åsa Sivertsson">Å Sivertsson</span>, <span class="pub-author" title="Caroline Kampf">C Kampf</span>, <span class="pub-author" title="Evelina Sjöstedt">E Sjöstedt</span>, <span class="pub-author" title="Anna Asplund">A Asplund</span>, <span class="pub-author" title="IngMarie Olsson">I Olsson</span>, <span class="pub-author" title="Karolina Edlund">K Edlund</span>, <span class="pub-author" title="Emma Lundberg">E Lundberg</span>, <span class="pub-author" title="Sanjay Navani">S Navani</span>, <span class="pub-author" title="Cristina Al-Khalili Szigyarto">CA Szigyarto</span>, <span class="pub-author" title="Jacob Odeberg">J Odeberg</span>, <span class="pub-author" title="Dijana Djureinovic">D Djureinovic</span>, <span class="pub-author" title="Jenny Ottosson Takanen">JO Takanen</span>, <span class="pub-author" title="Sophia Hober">S Hober</span>, <span class="pub-author" title="Tove Alm">T Alm</span>, <span class="pub-author" title="Per-Henrik Edqvist">PH Edqvist</span>, <span class="pub-author" title="Holger Berling">H Berling</span>, <span class="pub-author" title="Hanna Tegel">H Tegel</span>, <span class="pub-author" title="Jan Mulder">J Mulder</span>, <span class="pub-author" title="Johan Rockberg">J Rockberg</span>, <span class="pub-author" title="Peter Nilsson">P Nilsson</span>, <span class="pub-author" title="Jochen M Schwenk">JM Schwenk</span>, <span class="pub-author" title="Marica Hamsten">M Hamsten</span>, <span class="pub-author" title="Kalle von Feilitzen">K von Feilitzen</span>, <span class="pub-author" title="Mattias Forsberg">M Forsberg</span>, <span class="pub-author" title="Lukas Persson">L Persson</span>, <span class="pub-author" title="Fredric Johansson">F Johansson</span>, <span class="pub-author" title="Martin Zwahlen">M Zwahlen</span>, <span class="pub-author" title="Gunnar von Heijne">G von Heijne</span>, <span class="pub-author" title="Jens Nielsen">J Nielsen</span>, <span class="pub-author" title="Fredrik Pontén">F Pontén</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Resolving the molecular details of proteome variation in the different tissues and organs of the human body will greatly increase our knowledge of human biology and disease. Here, we present a map of the human tissue proteome based on an integrated omics approach that involves quantitative transcriptomics at the tissue and organ level, combined with tissue microarray-based immunohistochemistry, to achieve spatial localization of proteins down to the single-cell level. Our tissue-based analysis detected more than 90% of the putative protein-coding genes. We used this approach to explore the human secretome, the membrane proteome, the druggable proteome, the cancer proteome, and the metabolic functions in 32 different tissues and organs. All the data are integrated in an interactive Web-based database that allows exploration of individual proteins, as well as navigation of global expression patterns, in all major tissues and organs in the human body.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/25613900" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/science.1260419" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/3777a5cd8954424fb426f5163d4d01bf" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_c415e1ad145e4e3980e7a96e3c492895" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Extensive Clonal Branching Shapes the Evolutionary History of High-Risk Pediatric Cancers.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Natalie Andersson">N Andersson</span>, <span class="pub-author" title="Bjorn Bakker">B Bakker</span>, <span class="pub-author" title="Jenny Karlsson">J Karlsson</span>, <span class="pub-author" title="Anders Valind">A Valind</span>, <span class="pub-author" title="Linda Holmquist Mengelbier">L Holmquist Mengelbier</span>, <span class="pub-author" title="Diana C J Spierings">DCJ Spierings</span>, <span class="pub-author" title="Floris Foijer">F Foijer</span>, <span class="pub-author" title="David Gisselsson">D Gisselsson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Darwinian evolution of tumor cells remains underexplored in childhood cancer. We here reconstruct the evolutionary histories of 56 pediatric primary tumors, including 24 neuroblastomas, 24 Wilms tumors, and 8 rhabdomyosarcomas. Whole-genome copy-number and whole-exome mutational profiling of multiple regions per tumor were performed, followed by clonal deconvolution to reconstruct a phylogenetic tree for each tumor. Overall, 88% of the tumors exhibited genetic variation among primary tumor regions. This variability typically emerged through collateral phylogenetic branching, leading to spatial variability in the distribution of more than 50% (96/173) of detected diagnostically informative genetic aberrations. Single-cell sequencing of 547 individual cancer cells from eight solid pediatric tumors confirmed branching evolution to be a fundamental underlying principle of genetic variation in all cases. Strikingly, cell-to-cell genetic diversity was almost twice as high in aggressive compared with clinically favorable tumors (median Simpson index of diversity 0.45 vs. 0.88; P = 0.029). Similarly, a comparison of multiregional sampling data from a total of 274 tumor regions showed that new phylogenetic branches emerge at a higher frequency per sample and carry a higher mutational load in high-risk than in low-risk tumors. Timelines based on spatial genetic variation showed that the mutations most influencing relapse risk occur at initiation of clonal expansion in neuroblastoma and rhabdomyosarcoma, whereas in Wilms tumor, they are late events. Thus, from an evolutionary standpoint, some high-risk childhood cancers are born bad, whereas others grow worse over time. SIGNIFICANCE: Different pediatric cancers with a high risk of relapse share a common generic pattern of extensively branching evolution of somatic mutations. GRAPHICAL ABSTRACT: http://cancerres.aacrjournals.org/content/canres/80/7/1512/F1.large.jpg.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/32041836" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1158/0008-5472.CAN-19-3468" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/c415e1ad145e4e3980e7a96e3c492895" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_83d87735b99145889c0e1ebe02901133" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Bifidobacteria-mediated immune system imprinting early in life</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Bethany M Henrick">BM Henrick</span>, <span class="pub-author" title="Lucie Rodriguez">L Rodriguez</span>, <span class="pub-author" title="Tadepally Lakshmikanth">T Lakshmikanth</span>, <span class="pub-author" title="Christian Pou">C Pou</span>, <span class="pub-author" title="Ewa Henckel">E Henckel</span>, <span class="pub-author" title="Aron Arzoomand">A Arzoomand</span>, <span class="pub-author" title="Axel Olin">A Olin</span>, <span class="pub-author" title="Jun Wang">J Wang</span>, <span class="pub-author" title="Jaromir Mikes">J Mikes</span>, <span class="pub-author" title="Ziyang Tan">Z Tan</span>, <span class="pub-author" title="Yang Chen">Y Chen</span>, <span class="pub-author" title="Amy M Ehrlich">AM Ehrlich</span>, <span class="pub-author" title="Anna Karin Bernhardsson">AK Bernhardsson</span>, <span class="pub-author" title="Constantin Habimana Mugabo">CH Mugabo</span>, <span class="pub-author" title="Ylva Ambrosiani">Y Ambrosiani</span>, <span class="pub-author" title="Anna Gustafsson">A Gustafsson</span>, <span class="pub-author" title="Stephanie Chew">S Chew</span>, <span class="pub-author" title="Heather K Brown">HK Brown</span>, <span class="pub-author" title="Johann Prambs">J Prambs</span>, <span class="pub-author" title="Kajsa Bohlin">K Bohlin</span>, <span class="pub-author" title="Ryan D Mitchell">RD Mitchell</span>, <span class="pub-author" title="Mark A Underwood">MA Underwood</span>, <span class="pub-author" title="Jennifer T Smilowitz">JT Smilowitz</span>, <span class="pub-author" title="J Bruce German">JB German</span>, <span class="pub-author" title="Steven A Frese">SA Frese</span>, <span class="pub-author" title="Petter Brodin">P Brodin</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Immune-microbe interactions early in life influence the risk of allergies, asthma, and other inflammatory diseases. Breastfeeding guides healthier immune-microbe relationships by providing nutrients to specialized microbes that in turn benefit the host's immune system. Such bacteria have co-evolved with humans but are now increasingly rare in modern societies. Here we show that a lack of bifidobacteria, and in particular depletion of genes required for human milk oligosaccharide (HMO) utilization from the metagenome, is associated with systemic inflammation and immune dysregulation early in life. In breastfed infants given Bifidobacterium infantis EVC001, which expresses all HMO-utilization genes, intestinal T helper 2 (Th2) and Th17 cytokines were silenced and interferon β (IFNβ) was induced. Fecal water from EVC001-supplemented infants contains abundant indolelactate and B. infantis-derived indole-3-lactic acid (ILA) upregulated immunoregulatory galectin-1 in Th2 and Th17 cells during polarization, providing a functional link between beneficial microbes and immunoregulation during the first months of life.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/34143954" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.cell.2021.05.030" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/83d87735b99145889c0e1ebe02901133" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_a4c3fac414964931b8a6301041dbf23d" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A comparative genomics multitool for scientific discovery and conservation.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title=" Zoonomia Consortium"> Zoonomia Consortium</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">The Zoonomia Project is investigating the genomics of shared and specialized traits in eutherian mammals. Here we provide genome assemblies for 131 species, of which all but 9 are previously uncharacterized, and describe a whole-genome alignment of 240 species of considerable phylogenetic diversity, comprising representatives from more than 80% of mammalian families. We find that regions of reduced genetic diversity are more abundant in species at a high risk of extinction, discern signals of evolutionary selection at high resolution and provide insights from individual reference genomes. By prioritizing phylogenetic diversity and making data available quickly and without restriction, the Zoonomia Project aims to support biological discovery, medical research and the conservation of biodiversity.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33177664" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41586-020-2876-6" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/a4c3fac414964931b8a6301041dbf23d" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_5f2e4baa8bb24a13a4e27bafb78c7f35" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Association between Aquatic Micropollutant Dissipation and River Sediment Bacterial Communities.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Claudia Coll">C Coll</span>, <span class="pub-author" title="Raven Bier">R Bier</span>, <span class="pub-author" title="Zhe Li">Z Li</span>, <span class="pub-author" title="Silke Langenheder">S Langenheder</span>, <span class="pub-author" title="Elena Gorokhova">E Gorokhova</span>, <span class="pub-author" title="Anna Sobek">A Sobek</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Assessment of micropollutant biodegradation is essential to determine the persistence of potentially hazardous chemicals in aquatic ecosystems. We studied the dissipation half-lives of 10 micropollutants in sediment-water incubations (based on the OECD 308 standard) with sediment from two European rivers sampled upstream and downstream of wastewater treatment plant (WWTP) discharge. Dissipation half-lives (DT50s) were highly variable between the tested compounds, ranging from 1.5 to 772 days. Sediment from one river sampled downstream from the WWTP showed the fastest dissipation of all micropollutants after sediment RNA normalization. By characterizing sediment bacteria using 16S rRNA sequences, bacterial community composition of a sediment was associated with its capacity for dissipating micropollutants. Bacterial amplicon sequence variants of the genera Ralstonia, Pseudomonas, Hyphomicrobium, and Novosphingobium, which are known degraders of contaminants, were significantly more abundant in the sediment incubations where fast dissipation was observed. Our study illuminates the limitations of the OECD 308 standard to account for variation of dissipation rates of micropollutants due to differences in bacterial community composition. This limitation is problematic particularly for those compounds with DT50s close to regulatory persistence criteria. Thus, it is essential to consider bacterial community composition as a source of variability in regulatory biodegradation and persistence assessments.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33104348" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1021/acs.est.0c04393" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/5f2e4baa8bb24a13a4e27bafb78c7f35" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_9e5622f51ee24776be28a7e9a266fb7e" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Replicating associations between DNA methylation and body mass index in a longitudinal sample of older twins.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ida K Karlsson">IK Karlsson</span>, <span class="pub-author" title="Malin Ericsson">M Ericsson</span>, <span class="pub-author" title="Yunzhang Wang">Y Wang</span>, <span class="pub-author" title="Juulia Jylhävä">J Jylhävä</span>, <span class="pub-author" title="Sara Hägg">S Hägg</span>, <span class="pub-author" title="Nancy L Pedersen">NL Pedersen</span>, <span class="pub-author" title="Chandra A Reynolds">CA Reynolds</span>, <span class="pub-author" title="Anna K Dahl Aslan">AK Dahl Aslan</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">There is an important interplay between epigenetic factors and body weight, and previous work has identified ten sites where DNA methylation is robustly associated with body mass index (BMI) cross-sectionally. However, interpretation of the associations is complicated by the substantial changes in BMI often occurring in late-life, and the fact that methylation is often driven by genetic variation. This study therefore investigated the longitudinal association between these ten sites and BMI from midlife to late-life, and whether associations persist after controlling for genetic factors.

We used data from 535 individuals (mean age 68) in the Swedish Adoption/Twin Study of Aging (SATSA) with longitudinal measures of both DNA methylation from blood samples and BMI, spanning up to 20 years. Methylation levels were measured with the Infinium Human Methylation 450K or Infinium MethylationEpic array, with seven of the ten sites passing quality control. Latent growth curve models were applied to investigate longitudinal associations between methylation and BMI, and between-within models to study associations within twin pairs, thus adjusting for genetic factors.

Baseline DNA methylation levels at five of the seven sites were associated with BMI level at age 65 (cg00574958 [CPT1A]; cg11024682 [SREBF1]), and/or change (cg06192883 [MYO5C]; cg06946797 [RMI2]; cg08857797 [VPS25]). For four of the five sites, the associations remained comparable within twin pairs. However, the effects of cg06192883 were substantially attenuated within pairs. No change in DNA methylation was detected for any of the seven evaluated sites.

Five of the seven sites investigated were associated with late-life level and/or change in BMI. The effects for four of the sites remained similar when examined within twin pairs, indicating that the associations are mainly environmentally driven. However, the substantial attenuation in the association between cg06192883 and late-life BMI within pairs points to the importance of genetic factors in this association.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31801962" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41366-019-0498-6" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/9e5622f51ee24776be28a7e9a266fb7e" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_b586007ebb4542e18c827db7e108b219" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Experimental evidence of microbial inheritance in plants and transmission routes from seed to phyllosphere and root.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Ahmed Abdelfattah">A Abdelfattah</span>, <span class="pub-author" title="Michael Wisniewski">M Wisniewski</span>, <span class="pub-author" title="Leonardo Schena">L Schena</span>, <span class="pub-author" title="Ayco J M Tack">AJM Tack</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">While the environment is considered the primary origin of the plant microbiome, the potential role of seeds as a source of transmitting microorganisms has not received much attention. Here we tested the hypothesis that the plant microbiome is partially inherited through vertical transmission. An experimental culturing device was constructed to grow oak seedlings in a microbe-free environment while keeping belowground and aboveground tissues separated. The microbial communities associated with the acorn's embryo and pericarp and the developing seeding's phyllosphere and root systems were analysed using amplicon sequencing of fungal ITS and bacterial 16S rDNA. Results showed that the seed microbiome is diverse and non-randomly distributed within an acorn. The microbial composition of the phyllosphere was diverse and strongly resembled the composition found in the embryo, whereas the roots and pericarp each had a less diverse and distinct microbial community. Our findings demonstrate a high level of microbial diversity and spatial partitioning of the fungal and bacterial community within both seed and seedling, indicating inheritance, niche differentiation and divergent transmission routes for the establishment of root and phyllosphere communities.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33427409" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1111/1462-2920.15392" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/b586007ebb4542e18c827db7e108b219" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_be6dfc07d42b41648f42432ddcda8787" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Whole-genome sequencing and gene network modules predict gemcitabine/carboplatin-induced myelosuppression in non-small cell lung cancer patients.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Niclas Björn">N Björn</span>, <span class="pub-author" title="Tejaswi Venkata Satya Badam">TVS Badam</span>, <span class="pub-author" title="Rapolas Spalinskas">R Spalinskas</span>, <span class="pub-author" title="Eva Brandén">E Brandén</span>, <span class="pub-author" title="Hirsh Koyi">H Koyi</span>, <span class="pub-author" title="Rolf Lewensohn">R Lewensohn</span>, <span class="pub-author" title="Luigi De Petris">L De Petris</span>, <span class="pub-author" title="Zelmina Lubovac-Pilav">Z Lubovac-Pilav</span>, <span class="pub-author" title="Pelin Sahlén">P Sahlén</span>, <span class="pub-author" title="Joakim Lundeberg">J Lundeberg</span>, <span class="pub-author" title="Mika Gustafsson">M Gustafsson</span>, <span class="pub-author" title="Henrik Gréen">H Gréen</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Gemcitabine/carboplatin chemotherapy commonly induces myelosuppression, including neutropenia, leukopenia, and thrombocytopenia. Predicting patients at risk of these adverse drug reactions (ADRs) and adjusting treatments accordingly is a long-term goal of personalized medicine. This study used whole-genome sequencing (WGS) of blood samples from 96 gemcitabine/carboplatin-treated non-small cell lung cancer (NSCLC) patients and gene network modules for predicting myelosuppression. Association of genetic variants in PLINK found 4594, 5019, and 5066 autosomal SNVs/INDELs with p ≤ 1 × 10-3 for neutropenia, leukopenia, and thrombocytopenia, respectively. Based on the SNVs/INDELs we identified the toxicity module, consisting of 215 unique overlapping genes inferred from MCODE-generated gene network modules of 350, 345, and 313 genes, respectively. These module genes showed enrichment for differentially expressed genes in rat bone marrow, human bone marrow, and human cell lines exposed to carboplatin and gemcitabine (p < 0.05). Then using 80% of the patients as training data, random LASSO reduced the number of SNVs/INDELs in the toxicity module into a feasible prediction model consisting of 62 SNVs/INDELs that accurately predict both the training and the test (remaining 20%) data with high (CTCAE 3-4) and low (CTCAE 0-1) maximal myelosuppressive toxicity completely, with the receiver-operating characteristic (ROC) area under the curve (AUC) of 100%. The present study shows how WGS, gene network modules, and random LASSO can be used to develop a feasible and tested model for predicting myelosuppressive toxicity. Although the proposed model predicts myelosuppression in this study, further evaluation in other studies is required to determine its reproducibility, usability, and clinical effect.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/32839457" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1038/s41540-020-00146-6" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/be6dfc07d42b41648f42432ddcda8787" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_24e0c101ab3f49ac830c669e75aa8dfa" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">Evolutionary History, Genomic Adaptation to Toxic Diet, and Extinction of the Carolina Parakeet.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Pere Gelabert">P Gelabert</span>, <span class="pub-author" title="Marcela Sandoval-Velasco">M Sandoval-Velasco</span>, <span class="pub-author" title="Aitor Serres">A Serres</span>, <span class="pub-author" title="Marc de Manuel">M de Manuel</span>, <span class="pub-author" title="Pere Renom">P Renom</span>, <span class="pub-author" title="Ashot Margaryan">A Margaryan</span>, <span class="pub-author" title="Josefin Stiller">J Stiller</span>, <span class="pub-author" title="Toni de-Dios">T de-Dios</span>, <span class="pub-author" title="Qi Fang">Q Fang</span>, <span class="pub-author" title="Shaohong Feng">S Feng</span>, <span class="pub-author" title="Santi Mañosa">S Mañosa</span>, <span class="pub-author" title="George Pacheco">G Pacheco</span>, <span class="pub-author" title="Manuel Ferrando-Bernal">M Ferrando-Bernal</span>, <span class="pub-author" title="Guolin Shi">G Shi</span>, <span class="pub-author" title="Fei Hao">F Hao</span>, <span class="pub-author" title="Xianqing Chen">X Chen</span>, <span class="pub-author" title="Bent Petersen">B Petersen</span>, <span class="pub-author" title="Remi-André Olsen">RA Olsen</span>, <span class="pub-author" title="Arcadi Navarro">A Navarro</span>, <span class="pub-author" title="Yuan Deng">Y Deng</span>, <span class="pub-author" title="Love Dalén">L Dalén</span>, <span class="pub-author" title="Tomàs Marquès-Bonet">T Marquès-Bonet</span>, <span class="pub-author" title="Guojie Zhang">G Zhang</span>, <span class="pub-author" title="Agostinho Antunes">A Antunes</span>, <span class="pub-author" title="M Thomas P Gilbert">MTP Gilbert</span>, <span class="pub-author" title="Carles Lalueza-Fox">C Lalueza-Fox</span></div>
                        <p class="mt-2 mb-0"><span class="float-right badge badge-primary" title="A publication where a facility member is in the authors list" data-toggle="tooltip">NGI Collaboration</span></p>
                    </div>
                    <div class="modal-body small">As the only endemic neotropical parrot to have recently lived in the northern hemisphere, the Carolina parakeet (Conuropsis carolinensis) was an iconic North American bird. The last surviving specimen died in the Cincinnati Zoo in 1918 [1]. The cause of its extinction remains contentious: besides excessive mortality associated to habitat destruction and active hunting, their survival could have been negatively affected by its range having become increasingly patchy [2] or by the exposure to poultry pathogens [3, 4]. In addition, the Carolina parakeet showed a predilection for cockleburs, an herbaceous plant that contains a powerful toxin, carboxyatractyloside, or CAT [5], which did not seem to affect them but made the birds notoriously toxic to most predators [3]. To explore the demographic history of this bird, we generated the complete genomic sequence of a preserved specimen held in a private collection in Espinelves (Girona, Spain), as well as of a close extant relative, Aratinga solstitialis. We identified two non-synonymous genetic changes in two highly conserved proteins known to interact with CAT that could underlie a specific dietary adaptation to this toxin. Our genomic analyses did not reveal evidence of a dramatic past demographic decline in the Carolina parakeet; also, its genome did not exhibit the long runs of homozygosity that are signals of recent inbreeding and are typically found in endangered species. As such, our results suggest its extinction was an abrupt process and thus likely solely attributable to human causes.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/31839456" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1016/j.cub.2019.10.066" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/24e0c101ab3f49ac830c669e75aa8dfa" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_deff2c719d7a42779cbc1bc349308d01" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">How to Make a Rodent Giant: Genomic Basis and Tradeoffs of Gigantism in the Capybara, the World's Largest Rodent.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Santiago Herrera-Álvarez">S Herrera-Álvarez</span>, <span class="pub-author" title="Elinor Karlsson">E Karlsson</span>, <span class="pub-author" title="Oliver A Ryder">OA Ryder</span>, <span class="pub-author" title="Kerstin Lindblad-Toh">K Lindblad-Toh</span>, <span class="pub-author" title="Andrew J Crawford">AJ Crawford</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">Gigantism results when one lineage within a clade evolves extremely large body size relative to its small-bodied ancestors, a common phenomenon in animals. Theory predicts that the evolution of giants should be constrained by two tradeoffs. First, because body size is negatively correlated with population size, purifying selection is expected to be less efficient in species of large body size, leading to increased mutational load. Second, gigantism is achieved through generating a higher number of cells along with higher rates of cell proliferation, thus increasing the likelihood of cancer. To explore the genetic basis of gigantism in rodents and uncover genomic signatures of gigantism-related tradeoffs, we assembled a draft genome of the capybara (Hydrochoerus hydrochaeris), the world's largest living rodent. We found that the genome-wide ratio of nonsynonymous to synonymous mutations (ω) is elevated in the capybara relative to other rodents, likely caused by a generation-time effect and consistent with a nearly neutral model of molecular evolution. A genome-wide scan for adaptive protein evolution in the capybara highlighted several genes controlling postnatal bone growth regulation and musculoskeletal development, which are relevant to anatomical and developmental modifications for an increase in overall body size. Capybara-specific gene-family expansions included a putative novel anticancer adaptation that involves T-cell-mediated tumor suppression, offering a potential resolution to the increased cancer risk in this lineage. Our comparative genomic results uncovered the signature of an intragenomic conflict where the evolution of gigantism in the capybara involved selection on genes and pathways that are directly linked to cancer.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/33169792" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1093/molbev/msaa285" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/deff2c719d7a42779cbc1bc349308d01" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_ae4ff1411a31429f8d1c53a6fc16ed11" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">PRC2-mediated repression is essential to maintain identity and function of differentiated dopaminergic and serotonergic neurons.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="Konstantinos Toskas">K Toskas</span>, <span class="pub-author" title="Behzad Yaghmaeian-Salmani">B Yaghmaeian-Salmani</span>, <span class="pub-author" title="Olga Skiteva">O Skiteva</span>, <span class="pub-author" title="Wojciech Paslawski">W Paslawski</span>, <span class="pub-author" title="Linda Gillberg">L Gillberg</span>, <span class="pub-author" title="Vasiliki Skara">V Skara</span>, <span class="pub-author" title="Irene Antoniou">I Antoniou</span>, <span class="pub-author" title="Erik Södersten">E Södersten</span>, <span class="pub-author" title="Per Svenningsson">P Svenningsson</span>, <span class="pub-author" title="Karima Chergui">K Chergui</span>, <span class="pub-author" title="Markus Ringnér">M Ringnér</span>, <span class="pub-author" title="Thomas Perlmann">T Perlmann</span>, <span class="pub-author" title="Johan Holmberg">J Holmberg</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">How neurons can maintain cellular identity over an entire life span remains largely unknown. Here, we show that maintenance of identity in differentiated dopaminergic and serotonergic neurons is critically reliant on the Polycomb repressive complex 2 (PRC2). Deletion of the obligate PRC2 component, Eed, in these neurons resulted in global loss of H3K27me3, followed by a gradual activation of genes harboring both H3K27me3 and H3K9me3 modifications. Notably, H3K9me3 was lost at these PRC2 targets before gene activation. Neuronal survival was not compromised; instead, there was a reduction in subtype-specific gene expression and a progressive impairment of dopaminergic and serotonergic neuronal function, leading to behavioral deficits characteristic of Parkinson's disease and anxiety. Single-cell analysis revealed subtype-specific vulnerability to loss of PRC2 repression in dopamine neurons of the substantia nigra. Our study reveals that a PRC2-dependent nonpermissive chromatin state is essential to maintain the subtype identity and function of dopaminergic and serotonergic neurons.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/36026451" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/sciadv.abo1543" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/ae4ff1411a31429f8d1c53a6fc16ed11" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    

        <div class="modal ngisweden-publications-modal fade" id="pub_2a387a31c3c54f2f9d3abce4e7efdfe2" tabindex="-1" role="dialog" aria-hidden="true">
            <div class="modal-dialog modal-lg modal-dialog-centered modal-dialog-scrollable" role="document">
                <div class="modal-content">
                    <div class="modal-header">
                        <h5 class="modal-title">A Shh/Gli-driven three-node timer motif controls temporal identity and fate of neural stem cells.</h5>
                        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
                            <span aria-hidden="true">&times;</span>
                        </button>
                    </div>
                    <div class="modal-sub-header">
                        <div class="font-weight-light pub-authors"><span class="pub-author" title="José M Dias">JM Dias</span>, <span class="pub-author" title="Zhanna Alekseenko">Z Alekseenko</span>, <span class="pub-author" title="Ashwini Jeggari">A Jeggari</span>, <span class="pub-author" title="Marcelo Boareto">M Boareto</span>, <span class="pub-author" title="Jannik Vollmer">J Vollmer</span>, <span class="pub-author" title="Mariya Kozhevnikova">M Kozhevnikova</span>, <span class="pub-author" title="Hui Wang">H Wang</span>, <span class="pub-author" title="Michael P Matise">MP Matise</span>, <span class="pub-author" title="Andrey Alexeyenko">A Alexeyenko</span>, <span class="pub-author" title="Dagmar Iber">D Iber</span>, <span class="pub-author" title="Johan Ericson">J Ericson</span></div>
                        <p class="mt-2 mb-0"></p>
                    </div>
                    <div class="modal-body small">How time is measured by neural stem cells during temporal neurogenesis has remained unresolved. By combining experiments and computational modeling, we define a Shh/Gli-driven three-node timer underlying the sequential generation of motor neurons (MNs) and serotonergic neurons in the brainstem. The timer is founded on temporal decline of Gli-activator and Gli-repressor activities established through down-regulation of Gli transcription. The circuitry conforms an incoherent feed-forward loop, whereby Gli proteins not only promote expression of Phox2b and thereby MN-fate but also account for a delayed activation of a self-promoting transforming growth factor-β (Tgfβ) node triggering a fate switch by repressing Phox2b. Hysteresis and spatial averaging by diffusion of Tgfβ counteract noise and increase temporal accuracy at the population level, providing a functional rationale for the intrinsically programmed activation of extrinsic switch signals in temporal patterning. Our study defines how time is reliably encoded during the sequential specification of neurons.</div>
                    <div class="modal-footer ">
                        <button type="button" class="btn btn-sm btn-secondary" data-dismiss="modal">Close</button>
                        <a href="https://www.ncbi.nlm.nih.gov/pubmed/32938678" target="_blank" class="btn btn-sm btn-info">
                            Pubmed <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://dx.doi.org/10.1126/sciadv.aba8196" target="_blank" class="btn btn-sm btn-primary">
                            DOI <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                        <a href="https://publications.scilifelab.se/publication/2a387a31c3c54f2f9d3abce4e7efdfe2" target="_blank" class="btn btn-sm btn-success">
                            SciLifeLab Pubs <i class="fas fa-external-link-alt fa-sm ml-2"></i>
                        </a>
                    </div>
                </div>
            </div>
        </div>
    